
    <!DOCTYPE html>
    <html lang="en">

    <head>
        <meta charset="UTF-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <title>Your Table Title</title>

        <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.11.5/css/jquery.dataTables.css">
    </head>

    <body>

        <!-- Placeholder for the Generated Table -->
        <table border="1" class="dataframe my_table">
  <thead>
    <tr style="text-align: right;">
      <th>module</th>
      <th>name</th>
      <th>category</th>
      <th>cluster</th>
      <th>license</th>
      <th>keywords</th>
      <th>versions</th>
      <th>id</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>454-dataanalysis</td>
      <td>454-dataanalysis (Roche Newbler assembler)</td>
      <td>Bioinformatics SW Collections</td>
      <td>Rackham, Bianca, Snowy</td>
      <td></td>
      <td>assembler</td>
      <td>2.3, 2.53, 2.6, 2.9</td>
      <td>454-dataanalysis</td>
    </tr>
    <tr>
      <td>AAT</td>
      <td>AAT</td>
      <td>Bioinformatics Annotation</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Custom "AS IS"</td>
      <td>annotation</td>
      <td>r03052011</td>
      <td>AAT</td>
    </tr>
    <tr>
      <td>AGE</td>
      <td>AGE</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td></td>
      <td>alignment</td>
      <td>0.4</td>
      <td>AGE</td>
    </tr>
    <tr>
      <td>AMOS</td>
      <td>AMOS</td>
      <td>Bioinformatics Assembly</td>
      <td>Milou, Rackham, Bianca</td>
      <td>Artistic</td>
      <td>assembly</td>
      <td>3.0.0, 3.1.0</td>
      <td>AMOS</td>
    </tr>
    <tr>
      <td>ANGSD</td>
      <td>ANGSD</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Open source</td>
      <td>sequencing, gene</td>
      <td>0.917, 0.917-g6522d3e, 0.921, 0.933</td>
      <td>ANGSD</td>
    </tr>
    <tr>
      <td>ARCS</td>
      <td>ARCS</td>
      <td>Bioinformatics Assembly</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>assembly</td>
      <td>1.0.6, 1.1.1</td>
      <td>ARCS</td>
    </tr>
    <tr>
      <td>ARPACK-NG</td>
      <td>ARPACK-NG</td>
      <td>Libraries</td>
      <td>Milou, Rackham, Bianca</td>
      <td>BSD Software License</td>
      <td>eigenvalue</td>
      <td>3.8.0</td>
      <td>ARPACK-NG</td>
    </tr>
    <tr>
      <td>ArrowGrid</td>
      <td>ArrowGrid</td>
      <td>Bioinformatics Assembly</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Public Domain</td>
      <td>parallel, wrapper, consensus, assembly</td>
      <td>20191022-9759eb1</td>
      <td>ArrowGrid</td>
    </tr>
    <tr>
      <td>BOLT-LMM</td>
      <td>BOLT-LMM</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>model, genetic, gene, association, variance, heritability, correlation</td>
      <td>2.3.5</td>
      <td>BOLT-LMM</td>
    </tr>
    <tr>
      <td>Bamsurgeon</td>
      <td>Bamsurgeon</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>MIT License</td>
      <td>geo, mutation</td>
      <td>1.3</td>
      <td>Bamsurgeon</td>
    </tr>
    <tr>
      <td>BioBakery</td>
      <td>BioBakery\n            (Metapackage that includes MetaPhlAn, PhyloPhlan, KneadData and HUMAnN)</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>MIT</td>
      <td>microbial, profiling, community, micro</td>
      <td>3.0</td>
      <td>BioBakery</td>
    </tr>
    <tr>
      <td>CDO</td>
      <td>CDO</td>
      <td>Geospatial</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v2</td>
      <td>spatial</td>
      <td>1.9.5, 1.9.7.1, 1.9.7.1-intel18.3</td>
      <td>CDO</td>
    </tr>
    <tr>
      <td>COIN-OR-OptimizationSuite</td>
      <td>COIN-OR Optimization Suite</td>
      <td>Misc Applications</td>
      <td>Rackham, Bianca, Snowy</td>
      <td></td>
      <td>solver</td>
      <td>1.8.0</td>
      <td>COIN-OR-OptimizationSuite</td>
    </tr>
    <tr>
      <td>CONSEL</td>
      <td>CONSEL</td>
      <td>Bioinformatics Phylogeny</td>
      <td>Milou, Rackham, Bianca</td>
      <td></td>
      <td>phylogeny</td>
      <td>0.20</td>
      <td>CONSEL</td>
    </tr>
    <tr>
      <td>CheckM</td>
      <td>CheckM</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>microbial, cell, metagenome, micro, genome, quality</td>
      <td>1.0.11, 1.0.12, 1.1.3</td>
      <td>CheckM</td>
    </tr>
    <tr>
      <td>Conterminator</td>
      <td>Conterminator</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td></td>
      <td>kingdom</td>
      <td>20200601-570993b</td>
      <td>Conterminator</td>
    </tr>
    <tr>
      <td>Cromwell</td>
      <td>Cromwell</td>
      <td>Misc Applications</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>BSD 3-Clause</td>
      <td>bioinformatics</td>
      <td>71</td>
      <td>Cromwell</td>
    </tr>
    <tr>
      <td>DeepVariant</td>
      <td>DeepVariant</td>
      <td>Bioinformatics Pipelines</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>BSD-3</td>
      <td>pipeline</td>
      <td>0.8.0</td>
      <td>DeepVariant</td>
    </tr>
    <tr>
      <td>DelGet</td>
      <td>DelGet</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td></td>
      <td>orthologous, micro, region, species, gaps</td>
      <td>5.4</td>
      <td>DelGet</td>
    </tr>
    <tr>
      <td>EDirect</td>
      <td>EDirect</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Public Domain</td>
      <td>variation, structure, database, terminal, gene</td>
      <td>15.1</td>
      <td>EDirect</td>
    </tr>
    <tr>
      <td>ExaBayes</td>
      <td>ExaBayes</td>
      <td>Bioinformatics Phylogeny</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>phylogeny</td>
      <td>1.5, 1.5-mpi</td>
      <td>ExaBayes</td>
    </tr>
    <tr>
      <td>FALCON</td>
      <td>FALCON</td>
      <td>Bioinformatics Assembly</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Clear BSD</td>
      <td>assembly</td>
      <td>0.3.0, 0.4.1, 2018.31.08-03.06</td>
      <td>FALCON</td>
    </tr>
    <tr>
      <td>FALCON-integrate</td>
      <td>FALCON-integrate</td>
      <td>Bioinformatics Assembly</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Custom "as is"</td>
      <td>assembly</td>
      <td>20161113</td>
      <td>FALCON-integrate</td>
    </tr>
    <tr>
      <td>FFmpeg</td>
      <td>FFmpeg</td>
      <td>Misc Applications</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>LGPL v2.1+ with non-free features</td>
      <td>cross-platform, audio</td>
      <td>4.4</td>
      <td>FFmpeg</td>
    </tr>
    <tr>
      <td>FYBA</td>
      <td>OpenFYBA / SOSI</td>
      <td>Geospatial</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Custom open-source "as is"</td>
      <td>spatial</td>
      <td>4.1.1</td>
      <td>FYBA</td>
    </tr>
    <tr>
      <td>FastME</td>
      <td>FastME</td>
      <td>Bioinformatics Phylogeny</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>distance-based, inference, phylogeny</td>
      <td>2.1.6.2</td>
      <td>FastME</td>
    </tr>
    <tr>
      <td>FastML</td>
      <td>FastML</td>
      <td>Bioinformatics Phylogeny</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v2+</td>
      <td>phylogeny</td>
      <td>3.1, 3.11</td>
      <td>FastML</td>
    </tr>
    <tr>
      <td>FastQC</td>
      <td>FastQC</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL</td>
      <td>quality</td>
      <td>0.11.2, 0.11.5, 0.11.8, 0.11.9</td>
      <td>FastQC</td>
    </tr>
    <tr>
      <td>FastTree</td>
      <td>FastTree</td>
      <td>Bioinformatics Phylogeny</td>
      <td>Milou, Rackham, Bianca</td>
      <td>Open source "as is"</td>
      <td>phylogeny</td>
      <td>2.1.8, 2.1.10</td>
      <td>FastTree</td>
    </tr>
    <tr>
      <td>Flye</td>
      <td>Flye</td>
      <td>Bioinformatics Assembly</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>BSD 3-clause</td>
      <td>assembly</td>
      <td>2.3.5, 2.4.2, 2.9</td>
      <td>Flye</td>
    </tr>
    <tr>
      <td>GAA</td>
      <td>GAA</td>
      <td>Bioinformatics Assembly</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v2+</td>
      <td>assembly</td>
      <td>1.1</td>
      <td>GAA</td>
    </tr>
    <tr>
      <td>GARM</td>
      <td>GARM</td>
      <td>Bioinformatics Assembly</td>
      <td>Rackham, Bianca</td>
      <td></td>
      <td>assembly</td>
      <td>0.7.3</td>
      <td>GARM</td>
    </tr>
    <tr>
      <td>GDAL</td>
      <td>GDAL</td>
      <td>Geospatial</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>X/MIT style Open Source</td>
      <td>spatial</td>
      <td>2.1.0, 3.1.0</td>
      <td>GDAL</td>
    </tr>
    <tr>
      <td>GEM-Tools</td>
      <td>GEM-Tools</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL (GEM-Tools) and custom non-commercial (GEM library)</td>
      <td>alignment</td>
      <td>1.7.1</td>
      <td>GEM-Tools</td>
    </tr>
    <tr>
      <td>GEMINI</td>
      <td>GEMINI</td>
      <td>Bioinformatics Annotation</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>MIT</td>
      <td>annotation</td>
      <td>0.18.3, 0.19.0, 0.20.0, 0.20.1</td>
      <td>GEMINI</td>
    </tr>
    <tr>
      <td>GEOS</td>
      <td>GEOS</td>
      <td>Geospatial</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>LGPL v2.1</td>
      <td>spatial</td>
      <td>3.5.0, 3.9.1-gcc9.3.0</td>
      <td>GEOS</td>
    </tr>
    <tr>
      <td>GMP</td>
      <td>GMP</td>
      <td>Libraries</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>LGPL v3; GPL v2</td>
      <td>library, precision, floating-point, integer, rational</td>
      <td>6.2.1</td>
      <td>GMP</td>
    </tr>
    <tr>
      <td>GOTM</td>
      <td>GOTM</td>
      <td>Geospatial</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v2</td>
      <td>spatial</td>
      <td>5.3-221-gac7ec88d</td>
      <td>GOTM</td>
    </tr>
    <tr>
      <td>GUSHR</td>
      <td>GUSHR</td>
      <td>Bioinformatics Annotation</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>annotation, coding</td>
      <td>1.0.0</td>
      <td>GUSHR</td>
    </tr>
    <tr>
      <td>GeneMark</td>
      <td>GeneMark</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GeneMark licence</td>
      <td>annotation</td>
      <td>2.3-es, 4.32-es, 4.33-es, 4.33-es_Perl5.24.1, 4.38-es, 4.57-es, 4.62-es, 4.68-es</td>
      <td>GeneMark</td>
    </tr>
    <tr>
      <td>GenomeThreader</td>
      <td>GenomeThreader</td>
      <td>Bioinformatics Annotation</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Custom "AS IS"</td>
      <td>annotation</td>
      <td>1.7.0, 1.7.1</td>
      <td>GenomeThreader</td>
    </tr>
    <tr>
      <td>GhostPDL</td>
      <td>Ghostscript, GhostPCL, GhostXPS, GhostPDL</td>
      <td>Misc Applications</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Gnu GPL Affero v3</td>
      <td>language, interpreter</td>
      <td>9.53.3</td>
      <td>GhostPDL</td>
    </tr>
    <tr>
      <td>HESS</td>
      <td>HESS</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>genetic, local, covariance, gene, association, variance, heritability, correlation</td>
      <td>0.5.4-beta</td>
      <td>HESS</td>
    </tr>
    <tr>
      <td>HISAT2</td>
      <td>HISAT2</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>alignment</td>
      <td>2.1.0, 2.2.1</td>
      <td>HISAT2</td>
    </tr>
    <tr>
      <td>HOMER</td>
      <td>HOMER</td>
      <td>Bioinformatics Annotation</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>annotation</td>
      <td>4.10, 4.11</td>
      <td>HOMER</td>
    </tr>
    <tr>
      <td>HUMAnN</td>
      <td>HUMAnN</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>MIT</td>
      <td>microbial, sequencing, transcript, micro, metatranscriptomic, profiling, molecular, metagenomic, metabolic, genomic, abundance, pathway</td>
      <td>3.0</td>
      <td>HUMAnN</td>
    </tr>
    <tr>
      <td>HiC-Pro</td>
      <td>HiC-Pro</td>
      <td>Bioinformatics Pipelines</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>BSD-3</td>
      <td>pipeline</td>
      <td>2.11.1</td>
      <td>HiC-Pro</td>
    </tr>
    <tr>
      <td>HyPhy</td>
      <td>HyPhy</td>
      <td>Bioinformatics Phylogeny</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Custum open-source "as is"</td>
      <td>phylogeny</td>
      <td>2.5.0, 2.5.0-mpi</td>
      <td>HyPhy</td>
    </tr>
    <tr>
      <td>ImageMagick</td>
      <td>ImageMagick</td>
      <td>Misc Applications</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Derived Apache 2.0</td>
      <td>images</td>
      <td>6.9.9-35, 7.0.11-3</td>
      <td>ImageMagick</td>
    </tr>
    <tr>
      <td>InterProScan</td>
      <td>InterProScan</td>
      <td>Bioinformatics Annotation</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Misc</td>
      <td>annotation</td>
      <td>5.30-69.0, 5.52-86.0</td>
      <td>InterProScan</td>
    </tr>
    <tr>
      <td>Kalign</td>
      <td>Kalign</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v2</td>
      <td>alignment, multi</td>
      <td>1.04, 2.04</td>
      <td>Kalign</td>
    </tr>
    <tr>
      <td>KneadData</td>
      <td>KneadData</td>
      <td>Bioinformatics Misc</td>
      <td>Snowy, Rackham, Bianca</td>
      <td>MIT</td>
      <td>microbiome, sequencing, transcript, micro, metatranscriptomic, quality, metagenomic, genomic</td>
      <td>0.10.0</td>
      <td>KneadData</td>
    </tr>
    <tr>
      <td>LEON-BIS</td>
      <td>LEON-BIS</td>
      <td>Bioinformatics Alignment</td>
      <td>Milou, Rackham, Bianca</td>
      <td></td>
      <td>alignment</td>
      <td>20130322</td>
      <td>LEON-BIS</td>
    </tr>
    <tr>
      <td>LINKS</td>
      <td>LINKS</td>
      <td>Bioinformatics Assembly</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>assembly</td>
      <td>1.8.7</td>
      <td>LINKS</td>
    </tr>
    <tr>
      <td>LoRDEC</td>
      <td>LoRDEC</td>
      <td>Bioinformatics Assembly</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>CeCILL 2.1</td>
      <td>assembly</td>
      <td>0.9</td>
      <td>LoRDEC</td>
    </tr>
    <tr>
      <td>MAFFT</td>
      <td>MAFFT</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>BSD (main), mixed open-source (extensions)</td>
      <td>alignment</td>
      <td>7.205, 7.245, 7.310, 7.407</td>
      <td>MAFFT</td>
    </tr>
    <tr>
      <td>MEGAN</td>
      <td>MEGAN</td>
      <td>Bioinformatics Phylogeny</td>
      <td>Milou, Rackham, Bianca</td>
      <td>Personal academic</td>
      <td>phylogeny</td>
      <td>4.70.4, 5.1.5, 5.11.3, 6.3.5, 6.10.5, 6.20.17</td>
      <td>MEGAN</td>
    </tr>
    <tr>
      <td>MPC</td>
      <td>MPC</td>
      <td>Libraries</td>
      <td>Rackham, Snowy, Bianca</td>
      <td>LGPL v3+</td>
      <td>library, complex, precision</td>
      <td>1.2.1</td>
      <td>MPC</td>
    </tr>
    <tr>
      <td>MPFR</td>
      <td>MPFR</td>
      <td>Libraries</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>LGPL v3+</td>
      <td>library, multi, precision, floating-point</td>
      <td>4.1.0</td>
      <td>MPFR</td>
    </tr>
    <tr>
      <td>MUMmer</td>
      <td>MUMmer</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Artistic License 2.0</td>
      <td>alignment</td>
      <td>3.22, 3.23, 3.9.4alpha, 4.0.0beta2, 4.0.0rc1</td>
      <td>MUMmer</td>
    </tr>
    <tr>
      <td>MaSuRCA</td>
      <td>MaSuRCA</td>
      <td>Bioinformatics Assembly</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>assembly</td>
      <td>3.2.2, 3.2.3, 3.3.5, 3.4.2, 4.0.2</td>
      <td>MaSuRCA</td>
    </tr>
    <tr>
      <td>MafFilter</td>
      <td>MafFilter</td>
      <td>Bioinformatics Alignment</td>
      <td>Milou, Rackham, Bianca</td>
      <td></td>
      <td>alignment</td>
      <td>1.1.2</td>
      <td>MafFilter</td>
    </tr>
    <tr>
      <td>Magics</td>
      <td>Magics</td>
      <td>Geospatial</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Apache 2.0</td>
      <td>spatial</td>
      <td>3.3.1, 3.3.1-intel18.3</td>
      <td>Magics</td>
    </tr>
    <tr>
      <td>MetAMOS</td>
      <td>MetAMOS</td>
      <td>Bioinformatics Assembly</td>
      <td>Milou</td>
      <td>GPLv2 and other open source</td>
      <td>assembly</td>
      <td>1.5rc3</td>
      <td>MetAMOS</td>
    </tr>
    <tr>
      <td>MetaPhlAn3</td>
      <td>MetaPhlAn3</td>
      <td>Bioinformatics Phylogeny</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>MIT license</td>
      <td>microbial, composition, sequencing, communities, micro, phylogeny, profiling, species, metagenomic, shotgun, genomic, computational</td>
      <td>3.0.8</td>
      <td>MetaPhlAn3</td>
    </tr>
    <tr>
      <td>MultiQC</td>
      <td>MultiQC</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>MIT</td>
      <td>bioinformatics</td>
      <td>0.6, 0.7, 0.8, 0.9, 1.0, 1.2, 1.3, 1.5, 1.6, 1.7, 1.8, 1.9, 1.10, 1.10.1, 1.11</td>
      <td>MultiQC</td>
    </tr>
    <tr>
      <td>NCO</td>
      <td>NCO</td>
      <td>Geospatial</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>BSD</td>
      <td>spatial</td>
      <td>4.8.1, 4.9.2, 4.9.3</td>
      <td>NCO</td>
    </tr>
    <tr>
      <td>NPStat</td>
      <td>NPStat</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL version 3</td>
      <td>genetic, gene, pool</td>
      <td>1</td>
      <td>NPStat</td>
    </tr>
    <tr>
      <td>NanoPlot</td>
      <td>NanoPlot</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>alignment, sequencing</td>
      <td>1.33.1</td>
      <td>NanoPlot</td>
    </tr>
    <tr>
      <td>NeEstimator</td>
      <td>NeEstimator</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>AS IS</td>
      <td>genetic, genotype, gene, population</td>
      <td>2.1</td>
      <td>NeEstimator</td>
    </tr>
    <tr>
      <td>NeuSomatic</td>
      <td>NeuSomatic</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Creative Commons Public License</td>
      <td>convolutional, neural, networks, network, somatic, mutation</td>
      <td>0.2.1</td>
      <td>NeuSomatic</td>
    </tr>
    <tr>
      <td>NextDenovo</td>
      <td>NextDenovo</td>
      <td>Bioinformatics Assembly</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GNU</td>
      <td>string, assembler, graph, assembly, reads</td>
      <td>2.4.0, 2.5.0</td>
      <td>NextDenovo</td>
    </tr>
    <tr>
      <td>NextGenMap</td>
      <td>NextGenMap</td>
      <td>Bioinformatics Alignment</td>
      <td>Milou, Rackham, Bianca</td>
      <td>Artistic License 1.0</td>
      <td>alignment</td>
      <td>0.4.12, 0.5.4</td>
      <td>NextGenMap</td>
    </tr>
    <tr>
      <td>Nextflow</td>
      <td>Nextflow</td>
      <td>Bioinformatics Alignment</td>
      <td>Milou, Rackham, Bianca</td>
      <td>GPLv3</td>
      <td>pipeline</td>
      <td>0.17.3, 0.22.2, 0.26.0</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>Nextflow</td>
      <td>Nextflow</td>
      <td>Bioinformatics Pipelines</td>
      <td>Rackham, Bianca, Snowy</td>
      <td></td>
      <td>pipeline</td>
      <td>18.10.1</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>NgsRelate</td>
      <td>NgsRelate</td>
      <td>Bioinformatics Misc</td>
      <td>Snowy, Rackham, Bianca</td>
      <td>GPL v2</td>
      <td>statistics, genotype, coverage, relatedness, inbreeding, likelihood</td>
      <td>2.0</td>
      <td>NgsRelate</td>
    </tr>
    <tr>
      <td>OMBlast</td>
      <td>OMBlast</td>
      <td>Bioinformatics Alignment</td>
      <td>Milou, Rackham, Bianca</td>
      <td>unknown</td>
      <td>alignment</td>
      <td>1.0</td>
      <td>OMBlast</td>
    </tr>
    <tr>
      <td>OpenBUGS</td>
      <td>OpenBUGS</td>
      <td>Misc Applications</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v2</td>
      <td>inference</td>
      <td>3.2.3</td>
      <td>OpenBUGS</td>
    </tr>
    <tr>
      <td>PBSuite</td>
      <td>PBSuite (contains PBJelly and PBHoney)</td>
      <td>Bioinformatics Assembly</td>
      <td>Milou, Rackham, Bianca</td>
      <td>Custom redistributable "as is"</td>
      <td>assembly</td>
      <td>15.8.24</td>
      <td>PBSuite</td>
    </tr>
    <tr>
      <td>PLplot</td>
      <td>PLplot</td>
      <td>Libraries</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>LGPL v2</td>
      <td>plot</td>
      <td>5.15.0, 5.15.0-old_wxWidgets</td>
      <td>PLplot</td>
    </tr>
    <tr>
      <td>PRISMS-PF</td>
      <td>PRISMS-PS</td>
      <td>Chemistry/Physics</td>
      <td>Rackham, Snowy</td>
      <td>Lesser GPL</td>
      <td>microstructural, simulation, parallel, structural, micro, phase, evolution, finite</td>
      <td>2.1.1</td>
      <td>PRISMS-PF</td>
    </tr>
    <tr>
      <td>PROJ</td>
      <td>PROJ</td>
      <td>Geospatial</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>MIT</td>
      <td>spatial</td>
      <td>6.3.2</td>
      <td>PROJ</td>
    </tr>
    <tr>
      <td>PROJ.4</td>
      <td>PROJ.4</td>
      <td>Geospatial</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>MIT</td>
      <td>spatial</td>
      <td>4.9.2, 4.9.3, 4.9.3-intel18.3</td>
      <td>PROJ.4</td>
    </tr>
    <tr>
      <td>Phobius</td>
      <td>Phobius</td>
      <td>Bioinformatics Annotation</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Private study, education and non-profit research only</td>
      <td>annotation</td>
      <td>1.01</td>
      <td>Phobius</td>
    </tr>
    <tr>
      <td>PhyML</td>
      <td>PhyML</td>
      <td>Bioinformatics Phylogeny</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>phylogeny</td>
      <td>3.1, 3.3.20190321</td>
      <td>PhyML</td>
    </tr>
    <tr>
      <td>Pilon</td>
      <td>Pilon</td>
      <td>Bioinformatics Assembly</td>
      <td>Milou, Snowy, Rackham, Bianca</td>
      <td>GPL v2</td>
      <td>assembly</td>
      <td>1.17, 1.22, 1.24</td>
      <td>Pilon</td>
    </tr>
    <tr>
      <td>Platanus-allee</td>
      <td>Platanus-allee</td>
      <td>Bioinformatics Assembly</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>assembly</td>
      <td>2.0.2</td>
      <td>Platanus-allee</td>
    </tr>
    <tr>
      <td>ProbCons</td>
      <td>ProbCons</td>
      <td>Bioinformatics Alignment</td>
      <td>Milou, Rackham, Bianca</td>
      <td>Public domain</td>
      <td>alignment</td>
      <td>1.12</td>
      <td>ProbCons</td>
    </tr>
    <tr>
      <td>ProtHint</td>
      <td>ProtHint</td>
      <td>Bioinformatics Annotation</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GeneMark license</td>
      <td>annotation</td>
      <td>2.4.0, 2.5.0, 2.6.0</td>
      <td>ProtHint</td>
    </tr>
    <tr>
      <td>PsN</td>
      <td>Perl-speaks-NONMEM</td>
      <td>Statistics</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v2+</td>
      <td>peak</td>
      <td>5.0.0</td>
      <td>PsN</td>
    </tr>
    <tr>
      <td>QGIS</td>
      <td>QGIS</td>
      <td>Geospatial</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL</td>
      <td>geospatial, geo, spatial</td>
      <td>3.4.12</td>
      <td>QGIS</td>
    </tr>
    <tr>
      <td>QualiMap</td>
      <td>QualiMap</td>
      <td>Bioinformatics Alignment</td>
      <td>Milou, Rackham, Bianca</td>
      <td>GPL</td>
      <td>alignment</td>
      <td>2.0.2, 2.2</td>
      <td>QualiMap</td>
    </tr>
    <tr>
      <td>R</td>
      <td>R</td>
      <td>Statistics</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL</td>
      <td>graphics, statistical, graph, environment</td>
      <td>3.0.2, 3.2.3, 3.3.2, 3.4.0, 3.4.3, 3.5.0, 3.5.2, 3.6.0, 3.6.1, 4.0.0, 4.0.4, 4.1.1</td>
      <td>R</td>
    </tr>
    <tr>
      <td>RAisD</td>
      <td>RAisD</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v2</td>
      <td>polymorphism, nucleotide, multi, sweep</td>
      <td>2.9</td>
      <td>RAisD</td>
    </tr>
    <tr>
      <td>RAxML-NG</td>
      <td>RAxML-NG</td>
      <td>Bioinformatics Phylogeny</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GNU Affero GPL v3.0</td>
      <td>phylogeny</td>
      <td>0.2.0b, 0.2.0b-mpi, 0.7.0b, 0.7.0b-mpi, 0.9.0, 0.9.0-mpi</td>
      <td>RAxML-NG</td>
    </tr>
    <tr>
      <td>REAPR</td>
      <td>REAPR</td>
      <td>Bioinformatics Assembly</td>
      <td>Milou, Rackham, Bianca</td>
      <td>GPL</td>
      <td>assembly</td>
      <td>1.0.17, 1.0.18</td>
      <td>REAPR</td>
    </tr>
    <tr>
      <td>RStudio</td>
      <td>RStudio</td>
      <td>Statistics</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>AGPL</td>
      <td>language, development, environment</td>
      <td>1.0.136, 1.0.143, 1.0.153, 1.1.423, 1.1.463, 1.4.1106</td>
      <td>RStudio</td>
    </tr>
    <tr>
      <td>Racon</td>
      <td>Racon</td>
      <td>Bioinformatics Assembly</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>MIT</td>
      <td>genome, reads, consensus, assembly</td>
      <td>1.4.21</td>
      <td>Racon</td>
    </tr>
    <tr>
      <td>Ragout</td>
      <td>Ragout</td>
      <td>Bioinformatics Assembly</td>
      <td>Milou, Rackham, Bianca</td>
      <td>GPL v3.0</td>
      <td>assembly</td>
      <td>2.0-20170312</td>
      <td>Ragout</td>
    </tr>
    <tr>
      <td>Ray</td>
      <td>Ray</td>
      <td>Bioinformatics Assembly</td>
      <td>Milou, Rackham, Bianca</td>
      <td>GPL</td>
      <td>assembly</td>
      <td>0.0.4, 0.0.7, 1.6.1, 2.3.0, 2.3.1, 2.3.1-mpiio (with support for MPI I/O, but this seems not to work very well)</td>
      <td>Ray</td>
    </tr>
    <tr>
      <td>Rcorrector</td>
      <td>Rcorrector</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>kmer, correction</td>
      <td>1.0.4</td>
      <td>Rcorrector</td>
    </tr>
    <tr>
      <td>Redundans</td>
      <td>Redundans</td>
      <td>Bioinformatics Assembly</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v2</td>
      <td>assembly</td>
      <td>0.13a-20170321, 0.14a-20190314-de_novo-42e8edf (development only), 0.14a-20190313-d11d32b, 0.14a-20190509-ffae69e</td>
      <td>Redundans</td>
    </tr>
    <tr>
      <td>RepeatMasker</td>
      <td>RepeatMasker</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>open source</td>
      <td>annotation</td>
      <td>4.0.7, 4.0.7_Perl5.24.1, 4.0.8, 4.1.0</td>
      <td>RepeatMasker</td>
    </tr>
    <tr>
      <td>Roary</td>
      <td>Roary</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>assemblies, pipeline, genome</td>
      <td>3.6.2, 3.12.0, 3.13.0</td>
      <td>Roary</td>
    </tr>
    <tr>
      <td>SACRA</td>
      <td>SACRA</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>MIT 3</td>
      <td>virome, reads, chimeric</td>
      <td>1.0.0</td>
      <td>SACRA</td>
    </tr>
    <tr>
      <td>SAIGE</td>
      <td>SAIGE</td>
      <td>Misc Applications</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>biobanks, model, genetic, relatedness, gene, association, cohort</td>
      <td>0.42.1, 0.44.6.5</td>
      <td>SAIGE</td>
    </tr>
    <tr>
      <td>SGA</td>
      <td>SGA</td>
      <td>Bioinformatics Assembly</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>assembly</td>
      <td>0.10.14</td>
      <td>SGA</td>
    </tr>
    <tr>
      <td>SHAPEIT</td>
      <td>SHAPEIT</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Custom non-commercial</td>
      <td>phasing, plot, multi, haplotype, sequencing</td>
      <td>v2.r837, v2.r904, v3.r884.2, v4.1.3</td>
      <td>SHAPEIT</td>
    </tr>
    <tr>
      <td>SHAPELIB</td>
      <td>SHAPELIB</td>
      <td>Geospatial</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>LGPL v2</td>
      <td>spatial</td>
      <td>1.5.0</td>
      <td>SHAPELIB</td>
    </tr>
    <tr>
      <td>SLiM</td>
      <td>SLiM</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>model, simulation, genetic, modeling, evolution, gene, evolutionary, population, complex</td>
      <td>2.6, 3.2.1, 3.3.2, 3.6</td>
      <td>SLiM</td>
    </tr>
    <tr>
      <td>SMARTdenovo</td>
      <td>SMARTdenovo</td>
      <td>Bioinformatics Assembly</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Misc open source</td>
      <td>assembly</td>
      <td>20180219-5cc1356, 20210224-8488de9</td>
      <td>SMARTdenovo</td>
    </tr>
    <tr>
      <td>SMC++</td>
      <td>SMC++</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>genome, population</td>
      <td>1.15.2</td>
      <td>SMC++</td>
    </tr>
    <tr>
      <td>SMRT</td>
      <td>SMRT Link, SMRT Analysis</td>
      <td>Bioinformatics Pipelines</td>
      <td>Rackham, Bianca, Snowy</td>
      <td></td>
      <td>pipeline</td>
      <td>2.2.0, 2.3.0, 5.0.1, 7.0.1</td>
      <td>SMRT</td>
    </tr>
    <tr>
      <td>SNeP</td>
      <td>SNeP</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Snowy, Bianca</td>
      <td>AS IS</td>
      <td>phasing, recombination, genome, mutation, population, genome-wide</td>
      <td>1.1</td>
      <td>SNeP</td>
    </tr>
    <tr>
      <td>Salmon</td>
      <td>Salmon</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3+</td>
      <td>transcript, quantification</td>
      <td>0.8.2, 0.9.1, 0.11.2, 1.1.0, 1.4.0</td>
      <td>Salmon</td>
    </tr>
    <tr>
      <td>SalmonTE</td>
      <td>SalmonTE</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td></td>
      <td>pipeline</td>
      <td>20180926</td>
      <td>SalmonTE</td>
    </tr>
    <tr>
      <td>Scipio</td>
      <td>Scipio</td>
      <td>Bioinformatics Annotation</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Custom non-commercial</td>
      <td>annotation</td>
      <td>1.4.1</td>
      <td>Scipio</td>
    </tr>
    <tr>
      <td>SeqKit</td>
      <td>SeqKit</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>MIT License</td>
      <td>cross-platform</td>
      <td>0.15.0</td>
      <td>SeqKit</td>
    </tr>
    <tr>
      <td>Shasta</td>
      <td>Shasta</td>
      <td>Bioinformatics Assembly</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>MIT</td>
      <td>cell, assembler, assembly, gene, reads</td>
      <td>0.7.0</td>
      <td>Shasta</td>
    </tr>
    <tr>
      <td>SiBELia</td>
      <td>SiBELia</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td></td>
      <td>alignment</td>
      <td>3.0.6, 3.0.7</td>
      <td>SiBELia</td>
    </tr>
    <tr>
      <td>SignalP</td>
      <td>SignalP</td>
      <td>Bioinformatics Annotation</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Custom academic user licence</td>
      <td>annotation</td>
      <td>4.1c, 4.1f, 5.0b</td>
      <td>SignalP</td>
    </tr>
    <tr>
      <td>Sniffles</td>
      <td>Sniffles</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Snowy, Bianca</td>
      <td>MIT license</td>
      <td>variation, structural, sequencing, gene</td>
      <td>1.0.10, 1.0.12-201218-4ff6ecb</td>
      <td>Sniffles</td>
    </tr>
    <tr>
      <td>SortMeRNA</td>
      <td>SortMeRNA (original site)</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>alignment, site</td>
      <td>2.0, 2.1b, 3.0.3</td>
      <td>SortMeRNA</td>
    </tr>
    <tr>
      <td>Stampy</td>
      <td>Stampy</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Custom (non-commercial)</td>
      <td>alignment</td>
      <td>1.0.27, 1.0.32</td>
      <td>Stampy</td>
    </tr>
    <tr>
      <td>StringTie</td>
      <td>StringTie</td>
      <td>Bioinformatics Assembly</td>
      <td>Rackham, BIanca, Snowy</td>
      <td>MIT</td>
      <td>assembly</td>
      <td>1.2.0, 1.3.3, 2.1.4</td>
      <td>StringTie</td>
    </tr>
    <tr>
      <td>SuperLU</td>
      <td>SuperLU</td>
      <td>Libraries</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Custom</td>
      <td>sparse, linear</td>
      <td>5.2.1</td>
      <td>SuperLU</td>
    </tr>
    <tr>
      <td>T-Coffee</td>
      <td>T-Coffee</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v2</td>
      <td>alignment</td>
      <td>11.00.8cbe486, 13.39.0.d675aed</td>
      <td>T-Coffee</td>
    </tr>
    <tr>
      <td>TSEBRA</td>
      <td>TSEBRA</td>
      <td>Bioinformatics Annotation</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Artistic License 2.0</td>
      <td>annotation, transcript, codons, prediction, gene, introns</td>
      <td>1.0.2-20210716-1f167ad</td>
      <td>TSEBRA</td>
    </tr>
    <tr>
      <td>TargetP</td>
      <td>TargetP</td>
      <td>Bioinformatics Annotation</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Custom academic 'AS IS'</td>
      <td>cleavage, annotation, chloroplast, mitochondrial, thylakoid, site, terminal, luminal, signal, peptide</td>
      <td>2.0</td>
      <td>TargetP</td>
    </tr>
    <tr>
      <td>Tcl-Tk</td>
      <td>Tcl-Tk</td>
      <td>Misc Applications</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>BSD-like</td>
      <td>graphical, table, programming, graph, language, network</td>
      <td>8.6.11</td>
      <td>Tcl-Tk</td>
    </tr>
    <tr>
      <td>TransDecoder</td>
      <td>TransDecoder</td>
      <td>Bioinformatics Annotation</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Custom open-source 'AS IS'</td>
      <td>annotation</td>
      <td>5.0.1, 5.3.0, 5.5.0</td>
      <td>TransDecoder</td>
    </tr>
    <tr>
      <td>Unicycler</td>
      <td>Unicycler</td>
      <td>Bioinformatics Assembly</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>assembly</td>
      <td>0.4.8</td>
      <td>Unicycler</td>
    </tr>
    <tr>
      <td>VIBRANT</td>
      <td>VIBRANT</td>
      <td>Bioinformatics Annotation</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GNU General Public License</td>
      <td>annotation</td>
      <td>1.2.1</td>
      <td>VIBRANT</td>
    </tr>
    <tr>
      <td>Vapor</td>
      <td>Vapor</td>
      <td>Geospatial</td>
      <td>Rackham, Snowy</td>
      <td>Apache v2.0 license</td>
      <td>visualization, spatial, images, search, environment</td>
      <td>3.2.0, 3.3.0</td>
      <td>Vapor</td>
    </tr>
    <tr>
      <td>Vmatch</td>
      <td>Vmatch</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Custom non-commercial</td>
      <td>alignment</td>
      <td>2.2.5, 2.3.0</td>
      <td>Vmatch</td>
    </tr>
    <tr>
      <td>Yleaf</td>
      <td>Yleaf</td>
      <td>Bioinformatics Phylogeny</td>
      <td>Rackham, Bianca, Snowy</td>
      <td></td>
      <td>phylogeny</td>
      <td>20180326, 2.0</td>
      <td>Yleaf</td>
    </tr>
    <tr>
      <td>abyss</td>
      <td>ABySS</td>
      <td>Bioinformatics Assembly</td>
      <td>Milou, Rackham, Bianca</td>
      <td>GPL v3</td>
      <td>assembly</td>
      <td>1.2.3, 1.2.4, 1.2.5, 1.2.7 1.2.7-maxk96, 1.3.0, 1.3.2, 1.3.3, 1.3.3-k96, 1.3.4, 1.3.5, 1.3.5-max, 1.3.7, 1.3.7-max, 1.3.7-k128, 1.5.2, 1.9.0, 1.9.0-k128, 2.0.2, 2.0.2-k128</td>
      <td>abyss</td>
    </tr>
    <tr>
      <td>allpathslg</td>
      <td>AllPathsLG</td>
      <td>Bioinformatics Assembly</td>
      <td>Milou, </td>
      <td></td>
      <td>assembly</td>
      <td>47300, 49618, 52485</td>
      <td>allpathslg</td>
    </tr>
    <tr>
      <td>alphafold</td>
      <td>Alphafold</td>
      <td>Misc Applications</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Apache License</td>
      <td>pipeline, inference</td>
      <td>2.0.0</td>
      <td>alphafold</td>
    </tr>
    <tr>
      <td>anfo</td>
      <td>ANFO</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL</td>
      <td>alignment</td>
      <td>0.98</td>
      <td>anfo</td>
    </tr>
    <tr>
      <td>annovar</td>
      <td>ANNOVAR</td>
      <td>Bioinformatics SW Collections</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Strictly non-commercial usage</td>
      <td>annotation</td>
      <td>2014.11.12, 2015.06.17, 2016.05.11, 2017.07.16, 2018.04.16</td>
      <td>annovar</td>
    </tr>
    <tr>
      <td>aragorn</td>
      <td>ARAGORN</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td></td>
      <td>alignment</td>
      <td>1.2.36, 1.2.38</td>
      <td>aragorn</td>
    </tr>
    <tr>
      <td>augustus</td>
      <td>Augustus</td>
      <td>Bioinformatics Annotation</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Artistic License</td>
      <td>annotation</td>
      <td>3.2.2, 3.2.3, 3.2.3_Perl5.24.1, 3.3.3, 3.3.3-CGP</td>
      <td>augustus</td>
    </tr>
    <tr>
      <td>barrnap</td>
      <td>Barrnap</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>alignment, rrna, rna</td>
      <td>0.2, 0.6, 0.8, 0.9</td>
      <td>barrnap</td>
    </tr>
    <tr>
      <td>bcftools</td>
      <td>BCFtools</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>MIT/Expat or GPL v3</td>
      <td>variant</td>
      <td>1.2, 1.3.1, 1.5, 1.6, 1.8, 1.9, 1.9-develop, 1.10, 1.12, 1.14</td>
      <td>bcftools</td>
    </tr>
    <tr>
      <td>bcl2fastq</td>
      <td>bcl2fastq</td>
      <td>Bioinformatics SW Collections</td>
      <td>Rackham, Bianca, Snowy</td>
      <td></td>
      <td>fastq</td>
      <td>1.8.4, 2.15.0, 2.16.0, 2.17.1, 2.20.0</td>
      <td>bcl2fastq</td>
    </tr>
    <tr>
      <td>beagle</td>
      <td>beagle-lib</td>
      <td>Bioinformatics Phylogeny</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL</td>
      <td>phylogeny</td>
      <td>2.1.2, 3.1.2</td>
      <td>beagle</td>
    </tr>
    <tr>
      <td>beast</td>
      <td>BEAST</td>
      <td>Bioinformatics Phylogeny</td>
      <td>Milou, Rackham, Bianca</td>
      <td>GPL</td>
      <td>phylogeny</td>
      <td>1.7.5, 1.8.1, 1.8.4</td>
      <td>beast</td>
    </tr>
    <tr>
      <td>beast2</td>
      <td>BEAST2</td>
      <td>Bioinformatics Phylogeny</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>LGPL v2.1</td>
      <td>phylogeny</td>
      <td>2.1.2, 2.3.1, 2.4.0, 2.4.8, 2.6.2, 2.6.3</td>
      <td>beast2</td>
    </tr>
    <tr>
      <td>binutils</td>
      <td>Gnu binutils</td>
      <td>Compilers and build tools</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v2, v3; LGPL v2, v3</td>
      <td>compiler</td>
      <td>2.26, 2.28, 2.30</td>
      <td>binutils</td>
    </tr>
    <tr>
      <td>biopython</td>
      <td>BioPython</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Biopython</td>
      <td>python</td>
      <td>1.68 (loads python 2.7.6), 1.68-py3 (loads python/3.4.3); 1.73 (python/2.7.15), 1.73-py3 (python/3.6.0), 1.76-py3 (python/3.7.2)</td>
      <td>biopython</td>
    </tr>
    <tr>
      <td>bison</td>
      <td>bison</td>
      <td>Compilers and build tools</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3+</td>
      <td>grammar, compiler, table, context-free, gene, deterministic, parser</td>
      <td>3.7.6</td>
      <td>bison</td>
    </tr>
    <tr>
      <td>blasr</td>
      <td>Blasr and associated utilities</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Open source redistributable "as is"</td>
      <td>alignment</td>
      <td>20150922, 5.3-20161121-e901e48, 5.3-20171117-f72428d, 5.3.2</td>
      <td>blasr</td>
    </tr>
    <tr>
      <td>blast</td>
      <td>BLAST</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Public Domain</td>
      <td>alignment</td>
      <td>2.2.23+, 2.2.24+, 2.2.25+, 2.2.27+, 2.2.28+, 2.2.29+, 2.2.31+, 2.4.0+, 2.5.0+, 2.6.0+, 2.7.1+, 2.9.0+, 2.10.1+, 2.11.0+</td>
      <td>blast</td>
    </tr>
    <tr>
      <td>blast</td>
      <td>BLAST (legacy)</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Public Domain</td>
      <td>alignment</td>
      <td>2.2.15, 2.2.18, 2.2.22, 2.2.23, 2.2.24, 2.2.25, 2.2.26</td>
      <td>blast</td>
    </tr>
    <tr>
      <td>blat</td>
      <td>BLAT</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Open-source and strictly non-commercial</td>
      <td>alignment</td>
      <td>34, 35, 36</td>
      <td>blat</td>
    </tr>
    <tr>
      <td>bowtie</td>
      <td>Bowtie</td>
      <td>Bioinformatics Pipelines</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Artistic</td>
      <td>alignment</td>
      <td>0.12.9, 1.1.2, 1.2.0, 1.2.2, 1.2.3</td>
      <td>bowtie</td>
    </tr>
    <tr>
      <td>bowtie2</td>
      <td>Bowtie2</td>
      <td>Bioinformatics Pipelines</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>alignment</td>
      <td>2.1.0, 2.2.9, 2.3.2, 2.3.3.1, 2.3.4.1, 2.3.4.3, 2.3.5.1</td>
      <td>bowtie2</td>
    </tr>
    <tr>
      <td>bpipe</td>
      <td>bpipe</td>
      <td>Bioinformatics Pipelines</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>BSD</td>
      <td>pipeline</td>
      <td>0.9.8.6, 0.9.8.7</td>
      <td>bpipe</td>
    </tr>
    <tr>
      <td>braker</td>
      <td>BRAKER</td>
      <td>Bioinformatics Annotation</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Artistic License</td>
      <td>annotation</td>
      <td>2.1.1_Perl5.24.1, 2.1.5, 2.1.5-20200826-cf958ec, 2.1.5-20210115-e98b812, 2.1.6</td>
      <td>braker</td>
    </tr>
    <tr>
      <td>bwa</td>
      <td>BWA</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL</td>
      <td>alignment</td>
      <td>0.5.8a, 0.5.9, 0.6.1, 0.6.2, 0.7.4, 0.7.8, 0.7.10, 0.7.12, 0.7.13, 0.7.15, 0.7.17</td>
      <td>bwa</td>
    </tr>
    <tr>
      <td>bwa-meth</td>
      <td>bwa-meth</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>MIT</td>
      <td>alignment</td>
      <td>0.2.2</td>
      <td>bwa-meth</td>
    </tr>
    <tr>
      <td>bwakit</td>
      <td>bwa.kit</td>
      <td>Bioinformatics Alignment</td>
      <td>Milou, Rackham, Bianca</td>
      <td>GPL</td>
      <td>alignment</td>
      <td>0.7.12</td>
      <td>bwakit</td>
    </tr>
    <tr>
      <td>cactus</td>
      <td>cactus</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Custom open-source AS IS</td>
      <td>alignment</td>
      <td>1.0.0, 1.2.3</td>
      <td>cactus</td>
    </tr>
    <tr>
      <td>cairo</td>
      <td>Cairo</td>
      <td>Libraries</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GNU LGPL v2.1 or Mozilla Public Licence 1.1</td>
      <td>library, multi, graphics, graph</td>
      <td>1.14.8, 1.14.12, 1.17.2, 1.17.4</td>
      <td>cairo</td>
    </tr>
    <tr>
      <td>canu</td>
      <td>Canu</td>
      <td>Bioinformatics Assembly</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v2 and others for subcomponents</td>
      <td>assembly</td>
      <td>1.5, 1.6, 1.7, 1.7-86da76b, 1.8, 2.0, 2.2</td>
      <td>canu</td>
    </tr>
    <tr>
      <td>cdbfasta</td>
      <td>cdbfasta</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Artistic 2.0</td>
      <td>fasta</td>
      <td>0.99, 1.00</td>
      <td>cdbfasta</td>
    </tr>
    <tr>
      <td>cegma</td>
      <td>CEGMA</td>
      <td>Bioinformatics Annotation</td>
      <td>Rackham, Bianca, Snowy</td>
      <td></td>
      <td>annotation</td>
      <td>2.5</td>
      <td>cegma</td>
    </tr>
    <tr>
      <td>cellranger</td>
      <td>Cell Ranger (Chromium 10X)</td>
      <td>Bioinformatics SW Collections</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>other</td>
      <td>cell</td>
      <td>1.1.0, 1.3.0, 2.0.2, 3.0.1, 4.0.0, 5.0.1</td>
      <td>cellranger</td>
    </tr>
    <tr>
      <td>cellranger-ARC</td>
      <td>Cell Ranger ARC (Chromium 10X)</td>
      <td>Bioinformatics SW Collections</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>other</td>
      <td>cell</td>
      <td>1.0.0</td>
      <td>cellranger-ARC</td>
    </tr>
    <tr>
      <td>cellranger-ATAC</td>
      <td>Cell Ranger ATAC</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Misc non-commercial</td>
      <td>cell</td>
      <td>1.2.0</td>
      <td>cellranger-ATAC</td>
    </tr>
    <tr>
      <td>cellranger-DNA</td>
      <td>Cell Ranger DNA (Chromium 10X)</td>
      <td>Bioinformatics SW Collections</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>other</td>
      <td>cell</td>
      <td>1.1.0</td>
      <td>cellranger-DNA</td>
    </tr>
    <tr>
      <td>clustalo</td>
      <td>ClustalO</td>
      <td>Bioinformatics Alignment</td>
      <td>Milou, Rackham, Bianca</td>
      <td></td>
      <td>alignment</td>
      <td>1.2, 1.2.4</td>
      <td>clustalo</td>
    </tr>
    <tr>
      <td>clustalw</td>
      <td>ClustalW</td>
      <td>Bioinformatics Alignment</td>
      <td>Milou, Rackham, Bianca</td>
      <td>LGPL</td>
      <td>alignment</td>
      <td>2.1</td>
      <td>clustalw</td>
    </tr>
    <tr>
      <td>clusterflow</td>
      <td>Cluster Flow</td>
      <td>Bioinformatics Pipelines</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>cluster, pipeline</td>
      <td>0.3, 0.4devel, 0.4</td>
      <td>clusterflow</td>
    </tr>
    <tr>
      <td>cufflinks</td>
      <td>Cufflinks</td>
      <td>Bioinformatics Pipelines</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Boost License v1.0</td>
      <td>pipeline, link</td>
      <td>2.1.1, 2.2.1, 2.2.1-b55bb21</td>
      <td>cufflinks</td>
    </tr>
    <tr>
      <td>cutadapt</td>
      <td>cutadapt</td>
      <td>Bioinformatics Pipelines</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>MIT</td>
      <td>pipeline</td>
      <td>1.9.1, 1.13, 1.16, 2.0, 2.3, 3.1</td>
      <td>cutadapt</td>
    </tr>
    <tr>
      <td>ddt</td>
      <td>Allinea DDT</td>
      <td>Compilers and build tools</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Commercial</td>
      <td>compiler</td>
      <td>7.0</td>
      <td>ddt</td>
    </tr>
    <tr>
      <td>delly</td>
      <td>Delly</td>
      <td>Bioinformatics Annotation</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>annotation</td>
      <td>0.7.6, 0.7.7</td>
      <td>delly</td>
    </tr>
    <tr>
      <td>diamond</td>
      <td>DIAMOND</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>alignment</td>
      <td>0.7.9, 0.7.12, 0.8.26, 0.9.10, 0.9.24, 0.9.26, 0.9.29, 0.9.31, 2.0.4</td>
      <td>diamond</td>
    </tr>
    <tr>
      <td>ecCodes</td>
      <td>ecCodes</td>
      <td>Geospatial</td>
      <td>Rackham, Bianca, Snowy</td>
      <td></td>
      <td>spatial</td>
      <td>2.13.1</td>
      <td>ecCodes</td>
    </tr>
    <tr>
      <td>eggNOG-mapper</td>
      <td>EggNOG mapper and EggNOG databases 4.5.1</td>
      <td>Bioinformatics Annotation</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v2</td>
      <td>annotation, database</td>
      <td>1.0.3</td>
      <td>eggNOG-mapper</td>
    </tr>
    <tr>
      <td>exonerate</td>
      <td>Exonerate</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>alignment</td>
      <td>2.2.0, 2.4.0</td>
      <td>exonerate</td>
    </tr>
    <tr>
      <td>fasta</td>
      <td>FASTA</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Apache 2.0</td>
      <td>alignment, fasta</td>
      <td>36.3.6d, 36.3.6f, 36.3.8, 36.3.8h</td>
      <td>fasta</td>
    </tr>
    <tr>
      <td>fastp</td>
      <td>fastp</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>MIT</td>
      <td>preprocess</td>
      <td>0.20.0, 0.23.1</td>
      <td>fastp</td>
    </tr>
    <tr>
      <td>fastq_screen</td>
      <td>fastq_screen</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL</td>
      <td>fastq</td>
      <td>0.5.1, 0.9.2, 0.11.1</td>
      <td>fastq_screen</td>
    </tr>
    <tr>
      <td>fermi</td>
      <td>fermi</td>
      <td>Bioinformatics Assembly</td>
      <td>Milou, Rackham, Bianca</td>
      <td></td>
      <td>assembly</td>
      <td>1.1-r751-beta</td>
      <td>fermi</td>
    </tr>
    <tr>
      <td>fermikit</td>
      <td>fermikit</td>
      <td>Bioinformatics Pipelines</td>
      <td>Rackham, Bianca, Snowy</td>
      <td></td>
      <td>pipeline</td>
      <td>r178, 0.14-prerelease-96f7820</td>
      <td>fermikit</td>
    </tr>
    <tr>
      <td>flex</td>
      <td>flex</td>
      <td>Compilers and build tools</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>modified BSD</td>
      <td>gene, compiler</td>
      <td>2.6.4</td>
      <td>flex</td>
    </tr>
    <tr>
      <td>fpc</td>
      <td>Free Pascal Compiler</td>
      <td>Compilers and build tools</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL and LGPL</td>
      <td>compiler</td>
      <td>3.0.4</td>
      <td>fpc</td>
    </tr>
    <tr>
      <td>garli</td>
      <td>GARLI</td>
      <td>Bioinformatics Phylogeny</td>
      <td>Milou, Rackham, Bianca</td>
      <td>GPL v3</td>
      <td>phylogeny</td>
      <td>2.1</td>
      <td>garli</td>
    </tr>
    <tr>
      <td>gcta</td>
      <td>GCTA</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>MIT</td>
      <td>architecture, genetic, gene, trait, complex</td>
      <td>1.26.0, 1.93.2beta</td>
      <td>gcta</td>
    </tr>
    <tr>
      <td>geneid</td>
      <td>GeneID</td>
      <td>Bioinformatics Alignment</td>
      <td>Milou, Rackham, Bianca</td>
      <td>GPL</td>
      <td>alignment, gene</td>
      <td>1.4.4</td>
      <td>geneid</td>
    </tr>
    <tr>
      <td>gffread</td>
      <td>gffread</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca,Snowy</td>
      <td>MIT</td>
      <td>conversion, filter, tract</td>
      <td>0.12.6</td>
      <td>gffread</td>
    </tr>
    <tr>
      <td>git</td>
      <td>Git</td>
      <td>Compilers and build tools</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>LGPL 2.1</td>
      <td>compiler</td>
      <td>2.5.0, 2.10.2, 2.16.1, 2.21.0, 2.24.0, 2.28.0</td>
      <td>git</td>
    </tr>
    <tr>
      <td>git-lfs</td>
      <td>Git Large File Support</td>
      <td>Compilers and build tools</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>MIT</td>
      <td>compiler</td>
      <td>2.9.1</td>
      <td>git-lfs</td>
    </tr>
    <tr>
      <td>gmap-gsnap</td>
      <td>GMAP-GSNAP</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Custom as-is</td>
      <td>alignment</td>
      <td>2014-01-21, 2015-09-28, 2016-05-01, 2017-09-11, 2018-02-12, 2018-07-04, 2021-03-08</td>
      <td>gmap-gsnap</td>
    </tr>
    <tr>
      <td>gnuparallel</td>
      <td>GNU parallel</td>
      <td>Parallel</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL</td>
      <td>parallel</td>
      <td>20170122, 20180822</td>
      <td>gnuparallel</td>
    </tr>
    <tr>
      <td>gnuplot</td>
      <td>Gnuplot</td>
      <td>Misc Applications</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Custom open-source "as-is"</td>
      <td>plot</td>
      <td>5.0.7, 5.2.7</td>
      <td>gnuplot</td>
    </tr>
    <tr>
      <td>gubbins</td>
      <td>Gubbins</td>
      <td>Bioinformatics Phylogeny</td>
      <td>Milou, Rackham, Bianca</td>
      <td>GPL v2</td>
      <td>phylogeny</td>
      <td>1.4.2, 2.3.1</td>
      <td>gubbins</td>
    </tr>
    <tr>
      <td>guile</td>
      <td>Guile</td>
      <td>Compilers and build tools</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>LGPL 2.1</td>
      <td>compiler</td>
      <td>1.8.8</td>
      <td>guile</td>
    </tr>
    <tr>
      <td>gvcftools</td>
      <td>gvcftools</td>
      <td>Bioinformatics Misc</td>
      <td>Milou, Rackham, Bianca</td>
      <td>Various "As Is" open source</td>
      <td>gvcf</td>
      <td>0.16.2, 0.17.0</td>
      <td>gvcftools</td>
    </tr>
    <tr>
      <td>hdf4</td>
      <td>HDF4</td>
      <td>Libraries</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>BSD-style open source</td>
      <td>hdf</td>
      <td>4.2.11_gcc4.9.2, 4.2.14-gcc6.3.0</td>
      <td>hdf4</td>
    </tr>
    <tr>
      <td>hdf5</td>
      <td>HDF5</td>
      <td>Libraries</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>BSD-style open source</td>
      <td>hdf</td>
      <td>1.8.18, 1.8.18_gcc6.3.0, 1.10.5, 1.10.5-intel18.3</td>
      <td>hdf5</td>
    </tr>
    <tr>
      <td>hichipper</td>
      <td>hichipper</td>
      <td>Bioinformatics Pipelines</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>MIT</td>
      <td>pipeline</td>
      <td>0.7.3</td>
      <td>hichipper</td>
    </tr>
    <tr>
      <td>hmmer</td>
      <td>HMMER</td>
      <td>Bioinformatics Alignment</td>
      <td>Milou, Rackham, Bianca</td>
      <td>GPL</td>
      <td>alignment, hmm</td>
      <td>2.3.2-gcc, 2.3.2-intel, 3.0, 3.1b1, 3.1b2, 3.1b2-intel (MPI support, compiled with intel/17.2 and openmpi/2.1.0)</td>
      <td>hmmer</td>
    </tr>
    <tr>
      <td>htslib</td>
      <td>HTSlib</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>MIT/Expat and modified 3-clause BSD</td>
      <td>variant</td>
      <td>1.3, 1.3.2, 1.4, 1.5, 1.6, 1.8, 1.9, 1.10, 1.12, 1.14</td>
      <td>htslib</td>
    </tr>
    <tr>
      <td>infernal</td>
      <td>Infernal</td>
      <td>Bioinformatics Alignment</td>
      <td>Milou, Rackham, Bianca</td>
      <td>BSD</td>
      <td>alignment, rna</td>
      <td>1.1.1, 1.1.2</td>
      <td>infernal</td>
    </tr>
    <tr>
      <td>iqtree</td>
      <td>IQ-TREE</td>
      <td>Bioinformatics Phylogeny</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v2</td>
      <td>phylogeny, tree</td>
      <td>1.5.3-omp, 1.5.4-omp-mpi, 1.6.5-omp-mpi, 1.6.10-omp-mpi, 1.6.12-omp-mpi, 2.0-rc2-omp-mpi</td>
      <td>iqtree</td>
    </tr>
    <tr>
      <td>java</td>
      <td>Java JDK</td>
      <td>Compilers and build tools</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Other</td>
      <td>compiler</td>
      <td>sun_jdk1.6.0_04, sun_jdk1.6.0_18, sun_jdk1.7.0_25, sun_jdk1.8.0_40, sun_jdk1.8.0_92, sun_jdk1.8.0_151</td>
      <td>java</td>
    </tr>
    <tr>
      <td>julia</td>
      <td>Julia</td>
      <td>Compilers and build tools</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>MIT</td>
      <td>julia, compiler</td>
      <td>1.1.1, 1.4.2, 1.6.1</td>
      <td>julia</td>
    </tr>
    <tr>
      <td>k8</td>
      <td>k8</td>
      <td>Compilers and build tools</td>
      <td>Rackham, Bianca, Snowy</td>
      <td></td>
      <td>server-side, compiler, language, gene, shell, server</td>
      <td>0.2.5</td>
      <td>k8</td>
    </tr>
    <tr>
      <td>kallisto</td>
      <td>kallisto</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Custom "as is" open source</td>
      <td>sequencing, transcript, target, gene, reads, abundance</td>
      <td>0.42.4, 0.43.0, 0.43.1, 0.45.1, 0.46.2</td>
      <td>kallisto</td>
    </tr>
    <tr>
      <td>kourami</td>
      <td>Kourami</td>
      <td>Bioinformatics Assembly</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>BSD 3</td>
      <td>assembly</td>
      <td>0.9.6</td>
      <td>kourami</td>
    </tr>
    <tr>
      <td>last</td>
      <td>LAST</td>
      <td>Bioinformatics Alignment</td>
      <td>Milou, Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>alignment</td>
      <td>847, 963</td>
      <td>last</td>
    </tr>
    <tr>
      <td>lastz</td>
      <td>LASTZ</td>
      <td>Bioinformatics Alignment</td>
      <td>Milou, Rackham, Bianca</td>
      <td></td>
      <td>alignment</td>
      <td>1.03.54, 1.04.00</td>
      <td>lastz</td>
    </tr>
    <tr>
      <td>libBigWig</td>
      <td>libBigWig</td>
      <td>Bioinformatics Annotation</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>MIT</td>
      <td>annotation</td>
      <td>0.4.4</td>
      <td>libBigWig</td>
    </tr>
    <tr>
      <td>libgeotiff</td>
      <td>libgeotiff</td>
      <td>Geospatial</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>X-style and custom open-source "as is"</td>
      <td>geo, spatial</td>
      <td>1.4.1, 1.4.3</td>
      <td>libgeotiff</td>
    </tr>
    <tr>
      <td>libsodium</td>
      <td>Sodium</td>
      <td>Libraries</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>ISC</td>
      <td>table, library, decryption, hash, encryption</td>
      <td>1.0.18-stable</td>
      <td>libsodium</td>
    </tr>
    <tr>
      <td>libwebp</td>
      <td>libwebp</td>
      <td>Libraries</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Google "as is" open source</td>
      <td>library, decode, images</td>
      <td>1.2.0</td>
      <td>libwebp</td>
    </tr>
    <tr>
      <td>libzmq</td>
      <td>ZeroMQ</td>
      <td>Libraries</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>transport, queue, library, kernel, asynchronous, multi, tract, pattern, protocol, filter</td>
      <td>4.3.4</td>
      <td>libzmq</td>
    </tr>
    <tr>
      <td>lpsolve</td>
      <td>lp_solve</td>
      <td>Libraries</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>LGPL v2</td>
      <td>solver</td>
      <td>5.5.2.9</td>
      <td>lpsolve</td>
    </tr>
    <tr>
      <td>mafTools</td>
      <td>mafTools</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Custom "As is"</td>
      <td>alignment</td>
      <td>20170914-82077ac</td>
      <td>mafTools</td>
    </tr>
    <tr>
      <td>maker</td>
      <td>MAKER</td>
      <td>Bioinformatics Annotation</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL</td>
      <td>annotation</td>
      <td>2.31.9, 2.31.10, 3.01.1-beta, 3.01.2-beta, 3.01.2-beta-mpi, 3.01.04</td>
      <td>maker</td>
    </tr>
    <tr>
      <td>maq</td>
      <td>Maq</td>
      <td>Bioinformatics Alignment</td>
      <td>Milou, Rackham, Bianca</td>
      <td>GPL</td>
      <td>alignment</td>
      <td>0.7.1</td>
      <td>maq</td>
    </tr>
    <tr>
      <td>medaka</td>
      <td>medaka</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>MPL 2.0</td>
      <td>variant, nanopore, consensus, sequencing</td>
      <td>0.7.1, 1.4.3</td>
      <td>medaka</td>
    </tr>
    <tr>
      <td>megahit</td>
      <td>MEGAHIT</td>
      <td>Bioinformatics Assembly</td>
      <td>Milou, Rackham, Bianca</td>
      <td>GPL v3</td>
      <td>assembly</td>
      <td>1.0.3, 1.1.2, 2.0.4</td>
      <td>megahit</td>
    </tr>
    <tr>
      <td>metaWRAP</td>
      <td>metaWRAP</td>
      <td>Misc Applications</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>MIT</td>
      <td>genome, metagenomic, pipeline, genomic</td>
      <td>1.3.2</td>
      <td>metaWRAP</td>
    </tr>
    <tr>
      <td>metaphlan2</td>
      <td>MetaPhlAn2</td>
      <td>Bioinformatics Phylogeny</td>
      <td>Rackham, Bianca, Snowy</td>
      <td></td>
      <td>phylogeny</td>
      <td>2.0</td>
      <td>metaphlan2</td>
    </tr>
    <tr>
      <td>minced</td>
      <td>MinCED</td>
      <td>Bioinformatics Alignment</td>
      <td>Milou, Rackham, Bianca</td>
      <td>GPL v3</td>
      <td>alignment</td>
      <td>0.2.0</td>
      <td>minced</td>
    </tr>
    <tr>
      <td>minimap</td>
      <td>Minimap</td>
      <td>Bioinformatics Assembly</td>
      <td>Milou, Rackham, Bianca</td>
      <td>MIT</td>
      <td>assembly</td>
      <td>20160114</td>
      <td>minimap</td>
    </tr>
    <tr>
      <td>minimap2</td>
      <td>Minimap2</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>MIT</td>
      <td>alignment</td>
      <td>2.4, 2.14, 2.16, 2.17-r941, 2.18-r1015</td>
      <td>minimap2</td>
    </tr>
    <tr>
      <td>mip</td>
      <td>MIP</td>
      <td>Bioinformatics Pipelines</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>MIT</td>
      <td>pipeline</td>
      <td>6.0.0</td>
      <td>mip</td>
    </tr>
    <tr>
      <td>mono</td>
      <td>Mono</td>
      <td>Compilers and build tools</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Custom open source</td>
      <td>compiler</td>
      <td>3.12.0, 5.8.1.0, 5.10.1.27</td>
      <td>mono</td>
    </tr>
    <tr>
      <td>mosaik-aligner</td>
      <td>MOSAIK</td>
      <td>Bioinformatics Alignment</td>
      <td>Milou, Rackham, Bianca</td>
      <td>GPL</td>
      <td>alignment, aligner</td>
      <td>1.0.1388, 1.1.0021, 2.2.3, 2.2.26, 2.2.30</td>
      <td>mosaik-aligner</td>
    </tr>
    <tr>
      <td>mrbayes</td>
      <td>MrBayes</td>
      <td>Bioinformatics Phylogeny</td>
      <td>Milou, Rackham, Bianca, Snowy</td>
      <td>Public domain</td>
      <td>phylogeny</td>
      <td>3.1.2-mpi, 3.2.20 (mpi), 3.2.1 (mpi), 3.2.3a (mpi), 3.2.4 (mpi), 3.2.6 (mpi), 3.2.6 with gcc 7.1.0 (mpi), 3.2.7a with gcc 8.3.0 (mpi, gpu)</td>
      <td>mrbayes</td>
    </tr>
    <tr>
      <td>multiz</td>
      <td>Multiz</td>
      <td>Bioinformatics Alignment</td>
      <td>Milou, Rackham, Bianca</td>
      <td></td>
      <td>alignment, multi</td>
      <td>012109</td>
      <td>multiz</td>
    </tr>
    <tr>
      <td>muscle</td>
      <td>MUSCLE</td>
      <td>Bioinformatics Alignment</td>
      <td>Milou, Rackham, Bianca</td>
      <td>Public domain</td>
      <td>alignment</td>
      <td>3.8.31, 3.8.1551</td>
      <td>muscle</td>
    </tr>
    <tr>
      <td>nasm</td>
      <td>NASM</td>
      <td>Compilers and build tools</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>BSD 2-clause</td>
      <td>table, compiler, gene, architecture</td>
      <td>2.15.05</td>
      <td>nasm</td>
    </tr>
    <tr>
      <td>ncl</td>
      <td>NEXUS Class Library</td>
      <td>Bioinformatics Phylogeny</td>
      <td>Milou, Rackham, Bianca</td>
      <td>Simplified BSD</td>
      <td>phylogeny</td>
      <td>2.1.21-dev</td>
      <td>ncl</td>
    </tr>
    <tr>
      <td>ncview</td>
      <td>ncview</td>
      <td>Geospatial</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>browser</td>
      <td>2.1.7</td>
      <td>ncview</td>
    </tr>
    <tr>
      <td>netcdf</td>
      <td>NetCDF</td>
      <td>Libraries</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Custom open source "as is"</td>
      <td>netcdf</td>
      <td>4.7.1, 4.7.1-intel18.3</td>
      <td>netcdf</td>
    </tr>
    <tr>
      <td>nf-core</td>
      <td>nf-core</td>
      <td>Bioinformatics Pipelines</td>
      <td>Rackham, Bianca, Snowy</td>
      <td></td>
      <td>pipeline</td>
      <td>1.12.1, 1.14, 2.1</td>
      <td>nf-core</td>
    </tr>
    <tr>
      <td>nf-core-pipelines</td>
      <td>nf-core-pipelines</td>
      <td>Bioinformatics Pipelines</td>
      <td>Rackham, Bianca, Snowy</td>
      <td></td>
      <td>pipeline</td>
      <td>latest</td>
      <td>nf-core-pipelines</td>
    </tr>
    <tr>
      <td>ngsplot</td>
      <td>ngsplot</td>
      <td>Bioinformatics Misc</td>
      <td>Milou, Rackham, Bianca</td>
      <td>GPL v2</td>
      <td>plot</td>
      <td>2.61, 2.63</td>
      <td>ngsplot</td>
    </tr>
    <tr>
      <td>oncotator</td>
      <td>Oncotator</td>
      <td>Bioinformatics Annotation</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>BROAD INSTITUTE SOFTWARE LICENSE AGREEMENT FOR ACADEMIC NON-COMMERCIAL RESEARCH PURPOSES ONLY</td>
      <td>annotation</td>
      <td>1.9.9.0</td>
      <td>oncotator</td>
    </tr>
    <tr>
      <td>openbabel</td>
      <td>Open Babel</td>
      <td>Misc Applications</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v2</td>
      <td>language, chemical, peak</td>
      <td>3.1.1-gcc9.3.0</td>
      <td>openbabel</td>
    </tr>
    <tr>
      <td>openblas</td>
      <td>OpenBLAS</td>
      <td>Libraries</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>BSD 3-clause</td>
      <td>library, openmp</td>
      <td>0.2.14a, 0.2.19, 0.2.19-singlethread, 0.2.20, 0.2.20-singlethread, 0.2.20-openmp, 0.3.15, 0.3.15-singlethread, 0.3.15-openmp</td>
      <td>openblas</td>
    </tr>
    <tr>
      <td>optima</td>
      <td>OPTIMA</td>
      <td>Bioinformatics Alignment</td>
      <td>Milou, Rackham, Bianca</td>
      <td>GPL v2</td>
      <td>alignment</td>
      <td>f-1.3</td>
      <td>optima</td>
    </tr>
    <tr>
      <td>paml</td>
      <td>PAML</td>
      <td>Bioinformatics Phylogeny</td>
      <td>Milou, Rackham, Bianca</td>
      <td>Other</td>
      <td>phylogeny</td>
      <td>4.6, 4.9e</td>
      <td>paml</td>
    </tr>
    <tr>
      <td>patchelf</td>
      <td>patchelf</td>
      <td>Compilers and build tools</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>compiler</td>
      <td>0.8, 0.10</td>
      <td>patchelf</td>
    </tr>
    <tr>
      <td>pbbam</td>
      <td>pbbam</td>
      <td>Bioinformatics Assembly</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>PacBio open source license</td>
      <td>language, library, assembly</td>
      <td>1.6.0</td>
      <td>pbbam</td>
    </tr>
    <tr>
      <td>pblat</td>
      <td>pblat</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Same as blat</td>
      <td>alignment</td>
      <td>2.0, 2.1</td>
      <td>pblat</td>
    </tr>
    <tr>
      <td>pblat-cluster</td>
      <td>pblat-cluster</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Same as blat</td>
      <td>alignment, cluster</td>
      <td>0.1, 0.3, 1.0</td>
      <td>pblat-cluster</td>
    </tr>
    <tr>
      <td>perl</td>
      <td>Perl</td>
      <td>Compilers and build tools</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL, Perl</td>
      <td>perl, compiler</td>
      <td>5.18.2, 5.18.4, 5.22.2, 5.24.0, 5.24.1, 5.26.2 (5.24+ build with 64-bit ints and pointers)</td>
      <td>perl</td>
    </tr>
    <tr>
      <td>perl6</td>
      <td>Perl 6</td>
      <td>Compilers and build tools</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Artistic License 2.0</td>
      <td>perl, compiler</td>
      <td>rakudo-star-2017.04, rakudo-star-2019.03</td>
      <td>perl6</td>
    </tr>
    <tr>
      <td>perl_modules</td>
      <td>Perl Modules/Packages (large collection)</td>
      <td>Compilers and build tools</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL, Perl</td>
      <td>perl, compiler</td>
      <td>5.18.4, 5.24.1, 5.26.2</td>
      <td>perl_modules</td>
    </tr>
    <tr>
      <td>pfam_scan</td>
      <td>pfam_scan</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>alignment</td>
      <td>1.3, 1.5, 1.6</td>
      <td>pfam_scan</td>
    </tr>
    <tr>
      <td>phantompeakqualtools</td>
      <td>phantompeakqualtools</td>
      <td>Bioinformatics Misc</td>
      <td>Milou, Rackham, Bianca</td>
      <td>MIT</td>
      <td>peak</td>
      <td>1.1</td>
      <td>phantompeakqualtools</td>
    </tr>
    <tr>
      <td>phaser</td>
      <td>phASER</td>
      <td>Bioinformatics Misc</td>
      <td>Snowy, Rackham, Bianca</td>
      <td>GNU</td>
      <td>phasing, phase</td>
      <td>20210423-5d4926d</td>
      <td>phaser</td>
    </tr>
    <tr>
      <td>phast</td>
      <td>Phast</td>
      <td>Bioinformatics Phylogeny</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>BSD</td>
      <td>phylogeny</td>
      <td>1.4, 1.5</td>
      <td>phast</td>
    </tr>
    <tr>
      <td>phylobayes</td>
      <td>PhyloBayes</td>
      <td>Bioinformatics Phylogeny</td>
      <td>Rackham, Bianca, Snowy</td>
      <td></td>
      <td>phylogeny</td>
      <td>4.1c</td>
      <td>phylobayes</td>
    </tr>
    <tr>
      <td>phylobayesmpi</td>
      <td>PhyloBayes MPI</td>
      <td>Bioinformatics Phylogeny</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v2</td>
      <td>phylogeny</td>
      <td>1.8, 1.8b</td>
      <td>phylobayesmpi</td>
    </tr>
    <tr>
      <td>phyutility</td>
      <td>Phyutility</td>
      <td>Bioinformatics Phylogeny</td>
      <td>Rackham, Bianca, Snowy</td>
      <td></td>
      <td>phylogeny</td>
      <td>2.7.3</td>
      <td>phyutility</td>
    </tr>
    <tr>
      <td>pindel</td>
      <td>Pindel</td>
      <td>Bioinformatics Misc</td>
      <td>Milou</td>
      <td>GPL 3</td>
      <td>indel</td>
      <td>0.2.4w</td>
      <td>pindel</td>
    </tr>
    <tr>
      <td>pixy</td>
      <td>pixy</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>MIT License</td>
      <td>nucleotide, diversity, population</td>
      <td>1.2.5.beta1</td>
      <td>pixy</td>
    </tr>
    <tr>
      <td>pizzly</td>
      <td>pizzly</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>BSD 2-clause</td>
      <td>fusion</td>
      <td>0.37.3</td>
      <td>pizzly</td>
    </tr>
    <tr>
      <td>plink</td>
      <td>PLINK</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>link</td>
      <td>1.07, 1.90b3n, 1.90b4.9</td>
      <td>plink</td>
    </tr>
    <tr>
      <td>plink2</td>
      <td>PLINK2</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>link</td>
      <td>2.00-alpha-2-20180704, 2.00-alpha-2-20190429, 2.00-alpha-2.3-20200124</td>
      <td>plink2</td>
    </tr>
    <tr>
      <td>plinkseq</td>
      <td>PLINK/SEQ</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Other</td>
      <td>link</td>
      <td>0.08, 0.10</td>
      <td>plinkseq</td>
    </tr>
    <tr>
      <td>pplacer</td>
      <td>pplacer</td>
      <td>Bioinformatics Alignment</td>
      <td>Milou, Rackham, Bianca</td>
      <td>GPL v3</td>
      <td>alignment</td>
      <td>1.1.alpha19</td>
      <td>pplacer</td>
    </tr>
    <tr>
      <td>prank</td>
      <td>PRANK</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v2+</td>
      <td>alignment</td>
      <td>150803, 170427</td>
      <td>prank</td>
    </tr>
    <tr>
      <td>protobuf</td>
      <td>Protocol Buffers</td>
      <td>Libraries</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Google custom AS IS</td>
      <td>language, structure, utr</td>
      <td>3.11.4, 3.15.5, 3.15.5-gcc9.3.0</td>
      <td>protobuf</td>
    </tr>
    <tr>
      <td>pysam</td>
      <td>PySam</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>MIT</td>
      <td>python</td>
      <td>0.7.7 (python 2.7.6), 0.7.7-py3 (python 3.3.1), 0.8.3 (python 2.7.6), 0.8.3-py3 (python 3.4.3), 0.9.1.4, 0.10.0, 0.13-python2.7.11, 0.13-python3.6.0, 0.15.3-python3.6.8, 0.15.3-python3.7.2, 0.17.0-python3.9.5</td>
      <td>pysam</td>
    </tr>
    <tr>
      <td>python</td>
      <td>Python</td>
      <td>Compilers and build tools</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>PSF License Agreement</td>
      <td>python, compiler</td>
      <td>2.7.6, 2.7.9, 2.7.11, 2.7.15, 3.3, 3.3.1, 3.4.3, 3.5.0, 3.6.0, 3.6.8, 3.7.2, 3.8.7, 3.9.5</td>
      <td>python</td>
    </tr>
    <tr>
      <td>python3</td>
      <td>Python 3 (does not conflict with python module)</td>
      <td>Compilers and build tools</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>PSF License Agreement</td>
      <td>python, conflict, compiler</td>
      <td>3.6.0, 3.6.8, 3.7.2, 3.8.7, 3.9.5</td>
      <td>python3</td>
    </tr>
    <tr>
      <td>qiime2</td>
      <td>qiime2</td>
      <td>Bioinformatics Pipelines</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>BSD 3</td>
      <td>pipeline</td>
      <td>2018.11.0</td>
      <td>qiime2</td>
    </tr>
    <tr>
      <td>quickmerge</td>
      <td>quickmerge</td>
      <td>Bioinformatics Assembly</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>kmer, assembly</td>
      <td>0.3-9233726</td>
      <td>quickmerge</td>
    </tr>
    <tr>
      <td>raxml</td>
      <td>RAxML</td>
      <td>Bioinformatics Phylogeny</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>phylogeny</td>
      <td>7.3.0, 8.2.10-gcc, 8.2.10-gcc-mpi, 8.2.12-gcc, 8.2.12-gcc-mpi</td>
      <td>raxml</td>
    </tr>
    <tr>
      <td>rclone</td>
      <td>rclone</td>
      <td>Misc Applications</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>MIT</td>
      <td>storage, cloud</td>
      <td>1.56.2</td>
      <td>rclone</td>
    </tr>
    <tr>
      <td>rnammer</td>
      <td>RNAmmer</td>
      <td>Bioinformatics Alignment</td>
      <td>Milou, Rackham, Bianca</td>
      <td></td>
      <td>alignment, rna</td>
      <td>1.2</td>
      <td>rnammer</td>
    </tr>
    <tr>
      <td>rtgcore</td>
      <td>RTG Core non-commercial (aka rtg-core)</td>
      <td>Bioinformatics Misc</td>
      <td>Milou, Rackham, Bianca</td>
      <td>Custom non-commercial use only</td>
      <td>commercial</td>
      <td>3.8.3</td>
      <td>rtgcore</td>
    </tr>
    <tr>
      <td>ruby</td>
      <td>Ruby</td>
      <td>Compilers and build tools</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Ruby license</td>
      <td>compiler</td>
      <td>2.4.1, 2.5.0, 2.6.2</td>
      <td>ruby</td>
    </tr>
    <tr>
      <td>rust</td>
      <td>Rust</td>
      <td>Compilers and build tools</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Apache 2.0, MIT</td>
      <td>compiler</td>
      <td>1.43.1</td>
      <td>rust</td>
    </tr>
    <tr>
      <td>samtools</td>
      <td>SAMtools</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>MIT</td>
      <td>variant</td>
      <td>0.1.19, 1.1, 1.2, 1.3, 1.4, 1.5, 1.5_debug, 1.6, 1.8, 1.9, 1.10, 1.12, 1.14</td>
      <td>samtools</td>
    </tr>
    <tr>
      <td>satsuma2</td>
      <td>satsuma2</td>
      <td>Bioinformatics Assembly</td>
      <td>Milou, Rackham, Bianca</td>
      <td>LGPL v3+</td>
      <td>alignment</td>
      <td>2016-12-07</td>
      <td>satsuma2</td>
    </tr>
    <tr>
      <td>schmutzi</td>
      <td>Schmutzi</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>contamination, ancient</td>
      <td>20160424, 20200706-597c6bc</td>
      <td>schmutzi</td>
    </tr>
    <tr>
      <td>selscan</td>
      <td>Selscan</td>
      <td>Bioinformatics Misc</td>
      <td>Milou, Rackham, Bianca</td>
      <td>GPL v3</td>
      <td>selection, plot, haplotype</td>
      <td>1.1.0, 1.3.0</td>
      <td>selscan</td>
    </tr>
    <tr>
      <td>sissrs</td>
      <td>sissrs</td>
      <td>Bioinformatics Annotation</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Open source</td>
      <td>annotation</td>
      <td>1.4</td>
      <td>sissrs</td>
    </tr>
    <tr>
      <td>slurm-drmaa</td>
      <td>slurm-drmaa</td>
      <td>Libraries</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>slurm, drmaa</td>
      <td>1.1.2-slurm19.05.8</td>
      <td>slurm-drmaa</td>
    </tr>
    <tr>
      <td>snakemake</td>
      <td>snakemake</td>
      <td>Bioinformatics Pipelines</td>
      <td>Rackham, Bianca, Snowy</td>
      <td></td>
      <td>pipeline, snakemake</td>
      <td>3.10.1, 4.5.0, 5.2.3, 5.4.5, 5.10.0, 5.13.0, 5.17.0, 5.26.1, 5.30.1, 6.9.1</td>
      <td>snakemake</td>
    </tr>
    <tr>
      <td>snap</td>
      <td>snap</td>
      <td>Bioinformatics Misc</td>
      <td>Milou, Rackham, Bianca</td>
      <td>GPL</td>
      <td>annotation</td>
      <td>2013-02-16, 2013-11-29</td>
      <td>snap</td>
    </tr>
    <tr>
      <td>snippy</td>
      <td>snippy</td>
      <td>Bioinformatics Misc</td>
      <td>Snowy, Rackham, Bianca</td>
      <td>GPL 2.0</td>
      <td>genome, haploid, reads, reference</td>
      <td>4.0, 4.0.5, 4.6.0</td>
      <td>snippy</td>
    </tr>
    <tr>
      <td>snpAD</td>
      <td>snpAD</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>open source</td>
      <td>snp</td>
      <td>0.3.0, 0.3.4</td>
      <td>snpAD</td>
    </tr>
    <tr>
      <td>snpEff</td>
      <td>snpEff</td>
      <td>Bioinformatics Annotation</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>LGPL</td>
      <td>annotation, snp</td>
      <td>4.1, 4.2, 4.3t</td>
      <td>snpEff</td>
    </tr>
    <tr>
      <td>soapdenovo</td>
      <td>SOAPdenovo</td>
      <td>Bioinformatics Assembly</td>
      <td>Milou, Rackham, Bianca</td>
      <td></td>
      <td>assembly</td>
      <td>1.05, 2.04-r240</td>
      <td>soapdenovo</td>
    </tr>
    <tr>
      <td>soapdenovo-trans</td>
      <td>SOAPdenovo-Trans</td>
      <td>Bioinformatics Assembly</td>
      <td>Milou, Rackham, Bianca</td>
      <td>GPL v3</td>
      <td>assembly</td>
      <td>1.03, 1.04</td>
      <td>soapdenovo-trans</td>
    </tr>
    <tr>
      <td>spades</td>
      <td>SPAdes</td>
      <td>Bioinformatics Assembly</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3,GPLv2(3.14.1)</td>
      <td>assembly</td>
      <td>3.6.0, 3.8.0, 3.8.1, 3.9.0, 3.10.0, 3.10.1, 3.11.1, 3.12.0, 3.13.1, 3.14.1, 3.15.3</td>
      <td>spades</td>
    </tr>
    <tr>
      <td>spaln</td>
      <td>Spaln</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v2</td>
      <td>alignment</td>
      <td>2.1.4, 2.3.1, 2.4.0</td>
      <td>spaln</td>
    </tr>
    <tr>
      <td>sqlite</td>
      <td>SQLite</td>
      <td>Misc Applications</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Public domain</td>
      <td>language, library, database</td>
      <td>(system), 3.8.5, 3.11.1, 3.16.2, 3.24.0, 3.34.0</td>
      <td>sqlite</td>
    </tr>
    <tr>
      <td>star</td>
      <td>STAR</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>MIT, GPLv3 (prior to 2.7.2b)</td>
      <td>alignment</td>
      <td>2.3.1o, 2.5.1b, 2.5.2b, 2.5.3a, 2.7.0e, 2.7.0f, 2.7.1a, 2.7.2b, 2.7.9a</td>
      <td>star</td>
    </tr>
    <tr>
      <td>star-fusion</td>
      <td>STAR-Fusion together with Genome Resource Libraries</td>
      <td>Bioinformatics Annotation</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Broad Institute redistributable as-is</td>
      <td>pipeline, fusion</td>
      <td>1.0.0, 1.2.0, 1.5.0, 1.7.0</td>
      <td>star-fusion</td>
    </tr>
    <tr>
      <td>subversion</td>
      <td>Subversion</td>
      <td>Compilers and build tools</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Apache 2.0</td>
      <td>compiler</td>
      <td>1.9.3</td>
      <td>subversion</td>
    </tr>
    <tr>
      <td>tRNAscan-SE</td>
      <td>tRNAscan-SE</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, snowy</td>
      <td>GPL</td>
      <td>alignment</td>
      <td>1.3.1</td>
      <td>tRNAscan-SE</td>
    </tr>
    <tr>
      <td>tigmint</td>
      <td>Tigmint</td>
      <td>Bioinformatics Assembly</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>assembly</td>
      <td>1.1.2</td>
      <td>tigmint</td>
    </tr>
    <tr>
      <td>tmhmm</td>
      <td>TMHMM</td>
      <td>Bioinformatics Annotation</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Custom academic as-is</td>
      <td>annotation, protein, helices, hmm, transmembrane</td>
      <td>2.0c</td>
      <td>tmhmm</td>
    </tr>
    <tr>
      <td>tophat</td>
      <td>Tophat</td>
      <td>Bioinformatics Pipelines</td>
      <td>Rackham, Bianca, Snowy</td>
      <td></td>
      <td>pipeline</td>
      <td>1.0.14, 1.1.0, 1.1.3, 1.2.0, 1.3.3, 1.4.0, 2.0.0, 2.0.3, 2.0.4, 2.0.8b, 2.0.10, 2.0.11, 2.0.12, 2.1.1</td>
      <td>tophat</td>
    </tr>
    <tr>
      <td>trf</td>
      <td>TRF</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Gnu Affero GPL v3</td>
      <td>alignment</td>
      <td>4.07b, 4.09, 4.09.1</td>
      <td>trf</td>
    </tr>
    <tr>
      <td>trimmomatic</td>
      <td>Trimmomatic</td>
      <td>Bioinformatics Misc</td>
      <td>Milou, Rackham, Bianca</td>
      <td>GPL</td>
      <td>trim</td>
      <td>0.32, 0.36, 0.39</td>
      <td>trimmomatic</td>
    </tr>
    <tr>
      <td>trinity</td>
      <td>Trinity</td>
      <td>Bioinformatics Assembly</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Custom 'AS IS'</td>
      <td>transcript, assembly</td>
      <td>2014-07-17, 2.4.0, 2.9.1, 2.11.0, 2.13.2</td>
      <td>trinity</td>
    </tr>
    <tr>
      <td>trinotate</td>
      <td>Trinotate</td>
      <td>Bioinformatics Annotation</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Custom open-source 'AS IS'</td>
      <td>annotation</td>
      <td>3.0.2, 3.1.1, 3.2.2</td>
      <td>trinotate</td>
    </tr>
    <tr>
      <td>ucsc-utilities</td>
      <td>UCSC Bioinformatics Utilities</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Non-commercial</td>
      <td>bioinformatics</td>
      <td>v287, v334, v345, v398, v421</td>
      <td>ucsc-utilities</td>
    </tr>
    <tr>
      <td>unimap</td>
      <td>unimap</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>MIT</td>
      <td>reference, alignment, assembly-to-reference, assembly</td>
      <td>0.1-r46-dirty</td>
      <td>unimap</td>
    </tr>
    <tr>
      <td>usearch</td>
      <td>usearch (32-bit version); see vsearch for similar 64-bit capable tool</td>
      <td>Bioinformatics Misc</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Other</td>
      <td>search</td>
      <td>5.2.236, 5.2.32, 6.1.544, 7.0.1090, 8.1.1861, 9.2.64, 11.0.667</td>
      <td>usearch</td>
    </tr>
    <tr>
      <td>velvet</td>
      <td>Velvet</td>
      <td>Bioinformatics Assembly</td>
      <td>Milou, Rackham, Bianca</td>
      <td>GPL</td>
      <td>assembly</td>
      <td>1.0.03, 1.1.04, 1.1.04_K101, 1.1.07, 1.2.10, 1.2.10_K128</td>
      <td>velvet</td>
    </tr>
    <tr>
      <td>vmtouch</td>
      <td>vmtouch</td>
      <td>Misc Applications</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>BSD</td>
      <td>learning, cache</td>
      <td>1.3.1-20200309-4b70a9f</td>
      <td>vmtouch</td>
    </tr>
    <tr>
      <td>vsearch</td>
      <td>Vsearch</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3 or BSD 2-clause</td>
      <td>alignment, search</td>
      <td>2.3.2, 2.4.3, 2.5.2, 2.14.1, 2.18.0</td>
      <td>vsearch</td>
    </tr>
    <tr>
      <td>wise2</td>
      <td>Wise2</td>
      <td>Bioinformatics Alignment</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>Other</td>
      <td>alignment</td>
      <td>2.2.0, 2.4.1</td>
      <td>wise2</td>
    </tr>
    <tr>
      <td>wrf-python</td>
      <td>wrf-python</td>
      <td>Geospatial</td>
      <td>Rackham, Snowy</td>
      <td>GPL v2</td>
      <td>python, spatial, interpolation, search</td>
      <td>1.3.1</td>
      <td>wrf-python</td>
    </tr>
    <tr>
      <td>wtdbg2</td>
      <td>wtdbg2</td>
      <td>Bioinformatics Assembly</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>GPL v3</td>
      <td>assembly</td>
      <td>2.4</td>
      <td>wtdbg2</td>
    </tr>
    <tr>
      <td>yasm</td>
      <td>Yasm</td>
      <td>Compilers and build tools</td>
      <td>Rackham, Bianca, Snowy</td>
      <td>'new' BSD</td>
      <td>assembler, compiler</td>
      <td>1.3.0</td>
      <td>yasm</td>
    </tr>
  </tbody>
</table>

        <!-- Include jQuery and DataTables libraries -->
        <script type="text/javascript" charset="utf8" src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
        <script type="text/javascript" charset="utf8" src="https://cdn.datatables.net/1.11.5/js/jquery.dataTables.js"></script>

        <!-- Initialize DataTables -->
        <script>
            $(document).ready( function () {
                $('.my_table').DataTable({
                    "stripeClasses": ['', 'table-striped'], // For stripes
                    "hover": true,  // For hover effect
            });
            });
        </script>

    </body>

    </html>
    