
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../overview/">
      
      
        <link rel="next" href="../text_editors/">
      
      
        
      
      
      <link rel="icon" href="../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.7.3">
    
    
      
        <title>Software table - UPPMAX Documentation</title>
      
    
    
      <link rel="stylesheet" href="../../assets/stylesheets/main.484c7ddc.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.ab4e12ef.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato:300,300i,400,400i,700,700i%7CJetBrains+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Lato";--md-code-font:"JetBrains Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../../stylesheets/extra.css">
    
      <link rel="stylesheet" href="../../stylesheets/jquery.dataTables.min.css">
    
      <link rel="stylesheet" href="../../stylesheets/jquery.dataTables.overrides.css">
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="indigo" data-md-color-accent="indigo">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#bianca-software" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow md-header--lifted" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="UPPMAX Documentation" class="md-header__button md-logo" aria-label="UPPMAX Documentation" data-md-component="logo">
      
  <img src="../../assets/logo_color.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            UPPMAX Documentation
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Software table
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="indigo" data-md-color-accent="indigo"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 7a5 5 0 0 1 5 5 5 5 0 0 1-5 5 5 5 0 0 1-5-5 5 5 0 0 1 5-5m0 2a3 3 0 0 0-3 3 3 3 0 0 0 3 3 3 3 0 0 0 3-3 3 3 0 0 0-3-3m0-7 2.39 3.42C13.65 5.15 12.84 5 12 5s-1.65.15-2.39.42zM3.34 7l4.16-.35A7.2 7.2 0 0 0 5.94 8.5c-.44.74-.69 1.5-.83 2.29zm.02 10 1.76-3.77a7.131 7.131 0 0 0 2.38 4.14zM20.65 7l-1.77 3.79a7.02 7.02 0 0 0-2.38-4.15zm-.01 10-4.14.36c.59-.51 1.12-1.14 1.54-1.86.42-.73.69-1.5.83-2.29zM12 22l-2.41-3.44c.74.27 1.55.44 2.41.44.82 0 1.63-.17 2.37-.44z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="indigo" data-md-color-accent="indigo"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="m17.75 4.09-2.53 1.94.91 3.06-2.63-1.81-2.63 1.81.91-3.06-2.53-1.94L12.44 4l1.06-3 1.06 3zm3.5 6.91-1.64 1.25.59 1.98-1.7-1.17-1.7 1.17.59-1.98L15.75 11l2.06-.05L18.5 9l.69 1.95zm-2.28 4.95c.83-.08 1.72 1.1 1.19 1.85-.32.45-.66.87-1.08 1.27C15.17 23 8.84 23 4.94 19.07c-3.91-3.9-3.91-10.24 0-14.14.4-.4.82-.76 1.27-1.08.75-.53 1.93.36 1.85 1.19-.27 2.86.69 5.83 2.89 8.02a9.96 9.96 0 0 0 8.02 2.89m-1.64 2.02a12.08 12.08 0 0 1-7.8-3.47c-2.17-2.19-3.33-5-3.49-7.82-2.81 3.14-2.7 7.96.31 10.98 3.02 3.01 7.84 3.12 10.98.31"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var palette=__md_get("__palette");if(palette&&palette.color){if("(prefers-color-scheme)"===palette.color.media){var media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']");palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent")}for(var[key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      
      
        <label class="md-header__button md-icon" for="__search">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        </label>
        <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
        <div class="md-search__suggest" data-md-component="search-suggest"></div>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
      
    
    
      <div class="md-header__source">
        <a href="https://github.com/UPPMAX/UPPMAX-documentation" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
    
      
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
  
    <li class="md-tabs__item">
      <a href="../.." class="md-tabs__link">
        
  
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../../getting_started/get_started/" class="md-tabs__link">
          
  
  
  Getting Started

        </a>
      </li>
    
  

      
        
  
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../../cluster_guides/uppmax_cluster/" class="md-tabs__link">
          
  
  
  Cluster user guide

        </a>
      </li>
    
  

      
        
  
  
  
    
  
  
    
    
      <li class="md-tabs__item md-tabs__item--active">
        <a href="../overview/" class="md-tabs__link">
          
  
  
  Software

        </a>
      </li>
    
  

      
        
  
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../../databases/overview/" class="md-tabs__link">
          
  
  
  Databases

        </a>
      </li>
    
  

      
        
  
  
  
  
    <li class="md-tabs__item">
      <a href="../../support/" class="md-tabs__link">
        
  
  
    
  
  Support

      </a>
    </li>
  

      
        
  
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../../courses_workshops/courses_workshops/" class="md-tabs__link">
          
  
  
  Courses and workshops

        </a>
      </li>
    
  

      
        
  
  
  
  
    <li class="md-tabs__item">
      <a href="https://www.uu.se/en/centre/uppmax" class="md-tabs__link">
        
  
  
    
  
  UPPMAX main page

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
    
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="UPPMAX Documentation" class="md-nav__button md-logo" aria-label="UPPMAX Documentation" data-md-component="logo">
      
  <img src="../../assets/logo_color.png" alt="logo">

    </a>
    UPPMAX Documentation
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/UPPMAX/UPPMAX-documentation" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Home
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" >
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Getting Started
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            
  
    Getting Started
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../getting_started/get_started/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Get started
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../getting_started/project_apply/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Applying for a Project
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2_3" >
        
          
          <label class="md-nav__link" for="__nav_2_3" id="__nav_2_3_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Accessing UPPMAX
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_2_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2_3">
            <span class="md-nav__icon md-icon"></span>
            
  
    Accessing UPPMAX
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../getting_started/get_inside_sunet/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Get inside the university networks
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../getting_started/login_pelle/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Log in to Pelle
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../getting_started/login_bianca/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Log in to Bianca
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../getting_started/login_dardel/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Log in to Dardel (at PDC)
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2_4" >
        
          
          <label class="md-nav__link" for="__nav_2_4" id="__nav_2_4_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Linux tutorials
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_2_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2_4">
            <span class="md-nav__icon md-icon"></span>
            
  
    Linux tutorials
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../getting_started/linux/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Working in Linux
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../getting_started/linux_basics/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Basic Linux commands
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" >
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Cluster user guide
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            
  
    Cluster user guide
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster_guides/uppmax_cluster/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    The UPPMAX clusters
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster_guides/system_usage/system_usage.html" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    System usage
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3_3" >
        
          
          <label class="md-nav__link" for="__nav_3_3" id="__nav_3_3_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Clusters
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_3_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3_3">
            <span class="md-nav__icon md-icon"></span>
            
  
    Clusters
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster_guides/pelle/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Pelle
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster_guides/bianca/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Bianca
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster_guides/dardel/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Dardel (at PDC)
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3_4" >
        
          
          <label class="md-nav__link" for="__nav_3_4" id="__nav_3_4_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Storage systems
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_3_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3_4">
            <span class="md-nav__icon md-icon"></span>
            
  
    Storage systems
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster_guides/uppmax_storage_system/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    UPPMAX storage systems
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster_guides/spirula/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Object storage Spirula
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3_5" >
        
          
          <label class="md-nav__link" for="__nav_3_5" id="__nav_3_5_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    General cluster tools
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_3_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3_5">
            <span class="md-nav__icon md-icon"></span>
            
  
    General cluster tools
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster_guides/start_interactive_session/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Start an interactive session
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster_guides/modules/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Modules
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster_guides/slurm/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Slurm
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster_guides/project_management/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Managing your project
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3_6" >
        
          
          <label class="md-nav__link" for="__nav_3_6" id="__nav_3_6_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Running jobs
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_3_6_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3_6">
            <span class="md-nav__icon md-icon"></span>
            
  
    Running jobs
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster_guides/optimizing_jobs.md" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Checking and optimizing jobs
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster_guides/runtime_tips/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Runtime tips
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster_guides/storage_compression/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Storage and compression
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../screen/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Screen
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3_7" >
        
          
          <label class="md-nav__link" for="__nav_3_7" id="__nav_3_7_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Data transfer
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_3_7_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3_7">
            <span class="md-nav__icon md-icon"></span>
            
  
    Data transfer
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster_guides/transfer_pelle/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Transfer to/from Pelle
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster_guides/transfer_bianca/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Transfer to/from Bianca
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster_guides/webexport/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Webexport
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
    
    
      
        
        
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" checked>
        
          
          <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="">
            
  
  
  <span class="md-ellipsis">
    
  
    Software
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            
  
    Software
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../overview/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Overview
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
        
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  
  <span class="md-ellipsis">
    
  
    Software table
  

    
  </span>
  
  

      </a>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../text_editors/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Text editors
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
      
      
        
          
          
        
      
    
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4_4" >
        
          
          <label class="md-nav__link" for="__nav_4_4" id="__nav_4_4_label" tabindex="">
            
  
  
  <span class="md-ellipsis">
    
  
    Software-specific documentation
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_4_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4_4">
            <span class="md-nav__icon md-icon"></span>
            
  
    Software-specific documentation
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4_4_1" >
        
          
          <label class="md-nav__link" for="__nav_4_4_1" id="__nav_4_4_1_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Chemistry/physics/atmosphere
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="3" aria-labelledby="__nav_4_4_1_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4_4_1">
            <span class="md-nav__icon md-icon"></span>
            
  
    Chemistry/physics/atmosphere
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../games_us/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    GAMESS_US
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../gaussian/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Gaussian
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../gromacs/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    GROMACS
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../wrf/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    WRF
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4_4_2" >
        
          
          <label class="md-nav__link" for="__nav_4_4_2" id="__nav_4_4_2_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Integrated development environments (IDE)
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="3" aria-labelledby="__nav_4_4_2_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4_4_2">
            <span class="md-nav__icon md-icon"></span>
            
  
    Integrated development environments (IDE)
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../jupyter/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Jupyter
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../rstudio/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    RStudio
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../vscode/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    VSCode
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../vscodium/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    VSCodium
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4_4_3" >
        
          
          <label class="md-nav__link" for="__nav_4_4_3" id="__nav_4_4_3_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Life Science
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="3" aria-labelledby="__nav_4_4_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4_4_3">
            <span class="md-nav__icon md-icon"></span>
            
  
    Life Science
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../beast/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    BEAST
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../beast2/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    BEAST2
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../cram/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Cram
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../igv/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    IGV
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../metontiime/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    MetONTIIME
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../tracer/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Tracer
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4_4_4" >
        
          
          <label class="md-nav__link" for="__nav_4_4_4" id="__nav_4_4_4_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Machine Learning and AI
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="3" aria-labelledby="__nav_4_4_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4_4_4">
            <span class="md-nav__icon md-icon"></span>
            
  
    Machine Learning and AI
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../nvidia-deep-learning-frameworks/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    NVIDIA DLF
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../tensorflow/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    TensorFlow
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../whisper/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Whisper
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4_4_5" >
        
          
          <label class="md-nav__link" for="__nav_4_4_5" id="__nav_4_4_5_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Programming languages
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="3" aria-labelledby="__nav_4_4_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4_4_5">
            <span class="md-nav__icon md-icon"></span>
            
  
    Programming languages
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../julia/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Julia
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../matlab/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    MATLAB
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../python/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Python
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../perl/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Perl
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../r/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    R
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4_4_6" >
        
          
          <label class="md-nav__link" for="__nav_4_4_6" id="__nav_4_4_6_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Misc tools
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="3" aria-labelledby="__nav_4_4_6_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4_4_6">
            <span class="md-nav__icon md-icon"></span>
            
  
    Misc tools
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../allinea-ddt/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Allinea DDT
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../darsync/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    darsync
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../jobstats/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    jobstats
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../mariadb/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    MariaDB
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../projplot/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    projplot
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../rclone/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Rclone
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../ssh/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    SSH
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../sftp/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    SFTP
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../thinlinc/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    ThinLinc
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
      
      
        
          
          
        
      
    
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4_5" >
        
          
          <label class="md-nav__link" for="__nav_4_5" id="__nav_4_5_label" tabindex="">
            
  
  
  <span class="md-ellipsis">
    
  
    Installing and development
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_4_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4_5">
            <span class="md-nav__icon md-icon"></span>
            
  
    Installing and development
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../install/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Installing
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../conda/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Installing with conda
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../compilers/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Compilers
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../compiling_serial/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Compiling serial code
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../compiling_parallel/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Compiling parallel code
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../debuggers/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Debuggers
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../nextflow/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Nextflow & nf-core
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../profilers/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Profilers
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../singularity/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Singularity/Apptainer
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../spack/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Spack
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5" >
        
          
          <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Databases
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_5">
            <span class="md-nav__icon md-icon"></span>
            
  
    Databases
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../databases/overview/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Overview
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../databases/blast/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    BLAST databases available locally
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../databases/ncbi/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    NCBI taxonomy databases
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../databases/diamond/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    DIAMOND protein alignment databases
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../databases/reference_genomes/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Reference genomes
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../databases/1000-genome_project/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    1000-genome project
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../databases/simons_genome/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Simons Genome Diversity Project datasets
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../databases/other_local/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Other bioinformatics-oriented local data resources
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../support/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Support
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_7" >
        
          
          <label class="md-nav__link" for="__nav_7" id="__nav_7_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Courses and workshops
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_7_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_7">
            <span class="md-nav__icon md-icon"></span>
            
  
    Courses and workshops
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../courses_workshops/courses_workshops/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Overview
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../courses_workshops/awk/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    AWK
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../courses_workshops/bianca_intro.md" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Bianca intro
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../courses_workshops/bianca_intermediate/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Bianca intermediate
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../courses_workshops/naiss_transfer.md" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    File transfer
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../courses_workshops/hpc_python/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    HPC Python
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../courses_workshops/intro_to_python/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Intro to Python
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../courses_workshops/programming_formalisms/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Programming formalisms
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../courses_workshops/R_matlab_julia/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    R-MATLAB-Julia
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../courses_workshops/singularity/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Singularity
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../courses_workshops/uppmax_intro_course/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    UPPMAX intro
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="https://www.uu.se/en/centre/uppmax" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    UPPMAX main page
  

    
  </span>
  
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              
                



  


  <nav class="md-path" aria-label="Navigation" >
    <ol class="md-path__list">
      
        
  
  
    <li class="md-path__item">
      <a href="../.." class="md-path__link">
        
  <span class="md-ellipsis">
    Home
  </span>

      </a>
    </li>
  

      
      
        
  
  
    
    
      <li class="md-path__item">
        <a href="../overview/" class="md-path__link">
          
  <span class="md-ellipsis">
    Software
  </span>

        </a>
      </li>
    
  

      
    </ol>
  </nav>

              
              <article class="md-content__inner md-typeset">
                
                  


  
    <a href="https://github.com/UPPMAX/UPPMAX-documentation/blob/main/docs/software/software-table.md" title="Edit this page" class="md-content__button md-icon" rel="edit">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M10 20H6V4h7v5h5v3.1l2-2V8l-6-6H6c-1.1 0-2 .9-2 2v16c0 1.1.9 2 2 2h4zm10.2-7c.1 0 .3.1.4.2l1.3 1.3c.2.2.2.6 0 .8l-1 1-2.1-2.1 1-1c.1-.1.2-.2.4-.2m0 3.9L14.1 23H12v-2.1l6.1-6.1z"/></svg>
    </a>
  
  
    
      
    
    <a href="https://github.com/UPPMAX/UPPMAX-documentation/raw/main/docs/software/software-table.md" title="View source of this page" class="md-content__button md-icon">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 18c.56 0 1 .44 1 1s-.44 1-1 1-1-.44-1-1 .44-1 1-1m0-3c-2.73 0-5.06 1.66-6 4 .94 2.34 3.27 4 6 4s5.06-1.66 6-4c-.94-2.34-3.27-4-6-4m0 6.5a2.5 2.5 0 0 1-2.5-2.5 2.5 2.5 0 0 1 2.5-2.5 2.5 2.5 0 0 1 2.5 2.5 2.5 2.5 0 0 1-2.5 2.5M9.27 20H6V4h7v5h5v4.07c.7.08 1.36.25 2 .49V8l-6-6H6a2 2 0 0 0-2 2v16a2 2 0 0 0 2 2h4.5a8.2 8.2 0 0 1-1.23-2"/></svg>
    </a>
  


<div><h1 id="bianca-software">Bianca software<a class="headerlink" href="#bianca-software" title="Anchor link to this section for reference"></a></h1>
<div class="sortable_searchable_table">
<table>
<thead>
<tr>
<th>Category</th>
<th>Name</th>
<th>Module</th>
<th>Versions</th>
</tr>
</thead>
<tbody>
<tr>
<td>Alignment</td>
<td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi">blast</a></td>
<td>blast</td>
<td>2.14.1+, 2.15.0+, 2.16.0+</td>
</tr>
<tr>
<td>Alignment</td>
<td><a href="https://bowtie-bio.sourceforge.net/bowtie2/index.shtml">bowtie2</a></td>
<td>bowtie2</td>
<td>2.5.2</td>
</tr>
<tr>
<td>Alignment</td>
<td><a href="https://github.com/ComparativeGenomicsToolkit/cactus">cactus</a></td>
<td>cactus</td>
<td>2.7.2, 2.8.2</td>
</tr>
<tr>
<td>Alignment</td>
<td><a href="https://github.com/JLSteenwyk/ClipKIT">ClipKIT</a></td>
<td>ClipKIT</td>
<td>2.2.4</td>
</tr>
<tr>
<td>Alignment</td>
<td><a href="https://crossmap.sourceforge.net/">CrossMap</a></td>
<td>CrossMap</td>
<td>0.6.6</td>
</tr>
<tr>
<td>Alignment</td>
<td><a href="https://github.com/bbuchfink/diamond">diamond</a></td>
<td>diamond</td>
<td>2.0.6, 2.1.9</td>
</tr>
<tr>
<td>Alignment</td>
<td><a href="https://fasta.bioch.virginia.edu/wrpearson/fasta/">fasta</a></td>
<td>fasta</td>
<td>36.3.8i</td>
</tr>
<tr>
<td>Alignment</td>
<td><a href="http://hmmer.org/">hmmer</a></td>
<td>hmmer</td>
<td>3.4</td>
</tr>
<tr>
<td>Alignment</td>
<td>last</td>
<td>last</td>
<td>1505</td>
</tr>
<tr>
<td>Alignment</td>
<td><a href="https://github.com/lastz/lastz/">lastz</a></td>
<td>lastz</td>
<td>1.04.22</td>
</tr>
<tr>
<td>Alignment</td>
<td><a href="https://github.com/mpieva/mapAD">mapAD</a></td>
<td>mapAD</td>
<td>0.42.1</td>
</tr>
<tr>
<td>Alignment</td>
<td><a href="https://github.com/hsinnan75/MapCaller">mapcaller</a></td>
<td>mapcaller</td>
<td>0.9.9.41</td>
</tr>
<tr>
<td>Alignment</td>
<td><a href="https://github.com/marbl/MashMap">MashMap</a></td>
<td>MashMap</td>
<td>3.0.4</td>
</tr>
<tr>
<td>Alignment</td>
<td>minimap2</td>
<td>minimap2</td>
<td>2.26-r1175</td>
</tr>
<tr>
<td>Alignment</td>
<td><a href="https://github.com/lh3/miniprot">miniprot</a></td>
<td>miniprot</td>
<td>0.12, 0.13</td>
</tr>
<tr>
<td>Alignment</td>
<td><a href="https://github.com/soedinglab/mmseqs2">MMseqs2</a></td>
<td>MMseqs2</td>
<td>14-7e284, 15-6f452</td>
</tr>
<tr>
<td>Alignment</td>
<td><a href="https://github.com/philres/ngmlr">ngmlr</a></td>
<td>ngmlr</td>
<td>0.2.7-20210816-a2a31fb</td>
</tr>
<tr>
<td>Alignment</td>
<td><a href="https://github.com/PacificBiosciences/pbmm2">pbmm2</a></td>
<td>pbmm2</td>
<td>1.13.0</td>
</tr>
<tr>
<td>Alignment</td>
<td><a href="http://bioinformatics.psb.ugent.be/supplementary_data/erbon/nov2003/">randfold</a></td>
<td>randfold</td>
<td>2.0.1</td>
</tr>
<tr>
<td>Alignment</td>
<td><a href="https://www.repeatmasker.org/rmblast/">RMBlast</a></td>
<td>RMBlast</td>
<td>2.2.28, 2.14.1+</td>
</tr>
<tr>
<td>Alignment</td>
<td><a href="https://github.com/bluenote-1577/skani/">skani</a></td>
<td>skani</td>
<td>0.2.1</td>
</tr>
<tr>
<td>Alignment</td>
<td><a href="https://github.com/amplab/snap">SNAP-aligner</a></td>
<td>SNAP-aligner</td>
<td>2.0.3</td>
</tr>
<tr>
<td>Alignment</td>
<td><a href="https://github.com/ogotoh/spaln">spaln</a></td>
<td>spaln</td>
<td>3.0.3</td>
</tr>
<tr>
<td>Alignment</td>
<td><a href="https://github.com/alexdobin/STAR">star</a></td>
<td>star</td>
<td>2.7.8a, 2.7.11a</td>
</tr>
<tr>
<td>Alignment</td>
<td><a href="https://github.com/Benson-Genomics-Lab/TRF">trf</a></td>
<td>trf</td>
<td>4.10.0-rc.2</td>
</tr>
<tr>
<td>Alignment</td>
<td><a href="http://lowelab.ucsc.edu/tRNAscan-SE/">tRNAscan-SE</a></td>
<td>tRNAscan-SE</td>
<td>2.0.12</td>
</tr>
<tr>
<td>Alignment</td>
<td><a href="https://zhanggroup.org/US-align/">US-align</a></td>
<td>US-align</td>
<td>20230727-e5c6270</td>
</tr>
<tr>
<td>Alignment</td>
<td><a href="https://github.com/waveygang/wfmash">wfmash</a></td>
<td>wfmash</td>
<td>0.12.5</td>
</tr>
<tr>
<td>Alignment</td>
<td><a href="https://github.com/nloyfer/wgbs_tools">wgbs_tools</a></td>
<td>wgbs_tools</td>
<td>0.2.2-20250518-1b067ae</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://github.com/FreshAirTonight/af2complex">af2complex</a></td>
<td>af2complex</td>
<td>1.4.0</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://github.com/KosinskiLab/AlphaPulldown">alphapulldown</a></td>
<td>alphapulldown</td>
<td>1.0.4</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="http://bioinf.uni-greifswald.de/webaugustus/">augustus</a></td>
<td>augustus</td>
<td>3.5.0, 3.5.0-20231223-33fc04d</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://www.bali-phy.org/">bali-phy</a></td>
<td>bali-phy</td>
<td>4.0-beta15</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://github.com/freeseek/score">bcftools-score</a></td>
<td>bcftools-score</td>
<td>1.18-20231207-9c8e21e</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://github.com/Gaius-Augustus/BRAKER">braker</a></td>
<td>braker</td>
<td>3.0.7, 3.0.8</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://github.com/YoshitakaMo/localcolabfold">colabfold-local</a></td>
<td>colabfold-local</td>
<td>1.5.5</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://github.com/douglasgscofield/compleasm">compleasm</a></td>
<td>compleasm</td>
<td>0.2.2, 0.2.5, 0.2.6</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://github.com/WansonChoi/CookHLA">CookHLA</a></td>
<td>CookHLA</td>
<td>1.0.0</td>
</tr>
<tr>
<td>Annotation</td>
<td>copykat</td>
<td>copykat</td>
<td>1.1.0-b7a4763</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://services.healthtech.dtu.dk/service.php?DeepLoc-2.0">DeepLoc</a></td>
<td>DeepLoc</td>
<td>2.0</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://github.com/nanoporetech/dorado">dorado</a></td>
<td>dorado</td>
<td>0.6.1, 0.8.2, 0.9.1, 1.2.0</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://github.com/nanoporetech/duplex-tools">duplex-tools</a></td>
<td>duplex-tools</td>
<td>0.3.3</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://github.com/nextgenusfs/funannotate">funannotate</a></td>
<td>funannotate</td>
<td>1.8.1, 1.8.17</td>
</tr>
<tr>
<td>Annotation</td>
<td>GATE</td>
<td>GATE</td>
<td>0.42, 0.45</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="http://exon.gatech.edu/GeneMark">GeneMark</a></td>
<td>GeneMark</td>
<td>4.69-es, 4.71-es, 4.72-es</td>
</tr>
<tr>
<td>Annotation</td>
<td>GeneMark-ETP</td>
<td>GeneMark-ETP</td>
<td>1.02-20231213-dd8b37b</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="http://genomethreader.org/">GenomeThreader</a></td>
<td>GenomeThreader</td>
<td>1.7.4</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://github.com/vanheeringen-lab/gimmemotifs">gimmemotifs</a></td>
<td>gimmemotifs</td>
<td>0.18.0</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://github.com/raphael-group/hatchet">HATCHet</a></td>
<td>HATCHet</td>
<td>2.0.1</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://github.com/WansonChoi/HATK">HATK</a></td>
<td>HATK</td>
<td>2.0beta</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://github.com/mossmatters/HybPiper">hybpiper</a></td>
<td>hybpiper</td>
<td>2.1.6</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://github.com/igvteam/igv-reports">igv-reports</a></td>
<td>igv-reports</td>
<td>1.12.0-python3.9.5</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://www.ebi.ac.uk/interpro/download/InterProScan/">InterProScan</a></td>
<td>None</td>
<td>5.67-99.0, .z</td>
</tr>
<tr>
<td>Annotation</td>
<td>IsoQuant</td>
<td>IsoQuant</td>
<td>3.3.1</td>
</tr>
<tr>
<td>Annotation</td>
<td>jcvi</td>
<td>jcvi</td>
<td>1.3.8</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://github.com/voichek/kmersGWAS">kmersGWAS</a></td>
<td>kmersGWAS</td>
<td>20221010-a706bb7</td>
</tr>
<tr>
<td>Annotation</td>
<td>LDAK</td>
<td>LDAK</td>
<td>5.2, 6.1</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://bitbucket.org/mcgranahanlab/lohhla/src/master/">LOHHLA</a></td>
<td>LOHHLA</td>
<td>20210129-00744c5</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://github.com/slagtermaarten/LOHHLA">LOHHLA-slagtermaarten</a></td>
<td>LOHHLA-slagtermaarten</td>
<td>20200219-b38c477</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://www.agap-ge2pop.org/macsee-pipelines/">macse</a></td>
<td>macse</td>
<td>2.07</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://github.com/Gaius-Augustus/MakeHub">MakeHub</a></td>
<td>MakeHub</td>
<td>1.0.5-20200210-1ecd6bb, 1.0.8-20240217-31cc299</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://github.com/soedinglab/metaeuk">metaeuk</a></td>
<td>metaeuk</td>
<td>6-a5d39d9</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://msprime.readthedocs.io/en/latest/">msprime</a></td>
<td>msprime</td>
<td>1.2.0</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://pythonhosted.org/OBITools/welcome.html#installing-the-obitools">OBITools</a></td>
<td>OBITools</td>
<td>1.2.13</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://git.metabarcoding.org/obitools/obitools3">OBITools3</a></td>
<td>OBITools3</td>
<td>3.0.1b24</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://www.agap-ge2pop.org/macsee-pipelines/">omm-macse</a></td>
<td>omm-macse</td>
<td>12.01</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://www.ncbi.nlm.nih.gov/orffinder">ORFfinder</a></td>
<td>ORFfinder</td>
<td>0.4.3</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://davidemms.github.io/">OrthoFinder</a></td>
<td>OrthoFinder</td>
<td>2.5.2, 2.5.5</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://hub.docker.com/r/sachet/polysolver/tags/">polysolver</a></td>
<td>polysolver</td>
<td>v4</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://github.com/gatech-genemark/ProtHint">ProtHint</a></td>
<td>ProtHint</td>
<td>2.6.0-20231027-103304c</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://github.com/bcthomas/pullseq">pullseq</a></td>
<td>pullseq</td>
<td>20230518-7381691</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://github.com/griffithlab/pVACtools">pVACtools</a></td>
<td>pVACtools</td>
<td>3.1.1</td>
</tr>
<tr>
<td>Annotation</td>
<td>pycoQC</td>
<td>pycoQC</td>
<td>2.5.2</td>
</tr>
<tr>
<td>Annotation</td>
<td>RepeatMasker</td>
<td>RepeatMasker</td>
<td>4.1.5</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://github.com/smirarab/sepp">sepp</a></td>
<td>sepp</td>
<td>4.3.10_python_3.7.2, 4.5.1, 4.5.2</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://bitbucket.org/sequenzatools/sequenza-utils/src/master/">sequenza-utils</a></td>
<td>sequenza-utils</td>
<td>3.0.0</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="http://www.cbs.dtu.dk/services/SignalP/">SignalP</a></td>
<td>SignalP</td>
<td>6.0h</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://pcingola.github.io/SnpEff">snpEff</a></td>
<td>snpEff</td>
<td>5.2</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://github.com/instituteofcancerresearch/SOPRANO/">soprano</a></td>
<td>soprano</td>
<td>20240418-938604e</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://github.com/bdsc-tds/SPLIT">SPLIT</a></td>
<td>SPLIT</td>
<td>0.1.2.1-26f9b3a</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://github.com/hall-lab/svtools">svtools</a></td>
<td>svtools</td>
<td>0.5.1</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://github.com/vcflib/tabixpp">tabixpp</a></td>
<td>tabixpp</td>
<td>1.1.2</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://github.com/fansalon/TEspeX">TEspeX</a></td>
<td>TEspeX</td>
<td>2.0.1</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://github.com/loosolab/TOBIAS">tobias</a></td>
<td>tobias</td>
<td>0.17.3-86e7729</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://github.com/TransDecoder/TransDecoder/wiki">TransDecoder</a></td>
<td>TransDecoder</td>
<td>5.7.1-20230913-8b926ac</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://github.com/DrosophilaGenomeEvolution/TrEMOLO">TrEMOLO</a></td>
<td>TrEMOLO</td>
<td>2.2-beta1</td>
</tr>
<tr>
<td>Annotation</td>
<td>trex</td>
<td>trex</td>
<td>20230904-df86afe, 20231120-d9c840a</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://github.com/Gaius-Augustus/TSEBRA">TSEBRA</a></td>
<td>TSEBRA</td>
<td>1.1.2.4</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://github.com/10XGenomics/vartrix">vartrix</a></td>
<td>vartrix</td>
<td>1.1.22</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://github.com/mskcc/vcf2maf">vcf2maf</a></td>
<td>vcf2maf</td>
<td>1.6.21</td>
</tr>
<tr>
<td>Annotation</td>
<td><a href="https://whatshap.readthedocs.io/en/latest/">WhatsHap</a></td>
<td>WhatsHap</td>
<td>2.3-20240529-be88057</td>
</tr>
<tr>
<td>Assembly</td>
<td><a href="https://github.com/sanger-pathogens/assembly-stats">assembly-stats</a></td>
<td>assembly-stats</td>
<td>1.0.1-20211102-c006b9c</td>
</tr>
<tr>
<td>Assembly</td>
<td><a href="https://github.com/edgardomortiz/Captus">Captus</a></td>
<td>Captus</td>
<td>1.1.0</td>
</tr>
<tr>
<td>Assembly</td>
<td><a href="https://github.com/mikolmogorov/Flye">Flye</a></td>
<td>Flye</td>
<td>2.9.5</td>
</tr>
<tr>
<td>Assembly</td>
<td><a href="https://github.com/Kinggerm/GetOrganelle">GetOrganelle</a></td>
<td>GetOrganelle</td>
<td>1.7.3.3, 1.7.7.0</td>
</tr>
<tr>
<td>Assembly</td>
<td><a href="https://github.com/chhylp123/hifiasm">hifiasm</a></td>
<td>hifiasm</td>
<td>0.16.1-r375, 0.19.8-r603, 0.20.0-r639</td>
</tr>
<tr>
<td>Assembly</td>
<td><a href="https://github.com/xfengnefx/hifiasm-meta">hifiasm-meta</a></td>
<td>hifiasm-meta</td>
<td>0.3.2-r74</td>
</tr>
<tr>
<td>Assembly</td>
<td><a href="https://github.com/PacificBiosciences/pbipa">IPA</a></td>
<td>IPA</td>
<td>1.8.0</td>
</tr>
<tr>
<td>Assembly</td>
<td><a href="https://github.com/CAFS-bioinformatics/L_RNA_scaffolder">L_RNA_scaffolder</a></td>
<td>L_RNA_scaffolder</td>
<td>20190530-98f19e3</td>
</tr>
<tr>
<td>Assembly</td>
<td><a href="https://github.com/maickrau/MBG">MBG</a></td>
<td>MBG</td>
<td>1.0.14</td>
</tr>
<tr>
<td>Assembly</td>
<td><a href="https://github.com/GaetanBenoitDev/metaMDBG">metaMDBG</a></td>
<td>metaMDBG</td>
<td>0.3-20240117-57f4493</td>
</tr>
<tr>
<td>Assembly</td>
<td><a href="https://github.com/lh3/miniasm">miniasm</a></td>
<td>miniasm</td>
<td>0.3-r179-20191007-ce615d1</td>
</tr>
<tr>
<td>Assembly</td>
<td><a href="https://myloasm-docs.github.io/">myloasm</a></td>
<td>myloasm</td>
<td>0.2.0</td>
</tr>
<tr>
<td>Assembly</td>
<td><a href="https://github.com/rrwick/Polypolish">Polypolish</a></td>
<td>Polypolish</td>
<td>5.0</td>
</tr>
<tr>
<td>Assembly</td>
<td><a href="https://github.com/dfguan/purge_dups">purge_dups</a></td>
<td>purge_dups</td>
<td>1.2.5, 1.2.6</td>
</tr>
<tr>
<td>Assembly</td>
<td><a href="https://github.com/Gabaldonlab/redundans">Redundans</a></td>
<td>Redundans</td>
<td>2.0.1</td>
</tr>
<tr>
<td>Assembly</td>
<td><a href="https://github.com/tseemann/shovill">shovill</a></td>
<td>shovill</td>
<td>1.0.0, 1.1.0</td>
</tr>
<tr>
<td>Assembly</td>
<td><a href="https://github.com/ncbi/SKESA">SKESA</a></td>
<td>SKESA</td>
<td>2.4.0</td>
</tr>
<tr>
<td>Assembly</td>
<td><a href="http://cab.spbu.ru/software/spades/">spades</a></td>
<td>spades</td>
<td>4.0.0</td>
</tr>
<tr>
<td>Assembly</td>
<td><a href="https://ccb.jhu.edu/software/stringtie/">StringTie</a></td>
<td>StringTie</td>
<td>2.2.1</td>
</tr>
<tr>
<td>Assembly</td>
<td><a href="https://github.com/liulab-dfci/TRUST4">TRUST4</a></td>
<td>TRUST4</td>
<td>1.1.7</td>
</tr>
<tr>
<td>Assembly</td>
<td><a href="https://github.com/rrwick/Trycycler">Trycycler</a></td>
<td>Trycycler</td>
<td>0.5.4</td>
</tr>
<tr>
<td>Assembly</td>
<td><a href="https://biostars.org/tag/vg/">vg</a></td>
<td>vg</td>
<td>1.29.0, 1.48.0</td>
</tr>
<tr>
<td>Bioinformatics alignment</td>
<td><a href="https://github.com/abyzovlab/AGE">AGE</a></td>
<td>AGE</td>
<td>0.4</td>
</tr>
<tr>
<td>Bioinformatics alignment</td>
<td><a href="https://gemtools.github.io/">GEM-Tools</a></td>
<td>GEM-Tools</td>
<td>1.7.1</td>
</tr>
<tr>
<td>Bioinformatics alignment</td>
<td><a href="http://daehwankimlab.github.io/hisat2/">HISAT2</a></td>
<td>HISAT2</td>
<td>2.0.1-beta, 2.0.5, 2.1.0, 2.2.1</td>
</tr>
<tr>
<td>Bioinformatics alignment</td>
<td><a href="http://eddylab.org/infernal/">infernal</a></td>
<td>infernal</td>
<td>1.0.2, 1.1.1, 1.1.2</td>
</tr>
<tr>
<td>Bioinformatics alignment</td>
<td><a href="https://msa.sbc.su.se/cgi-bin/msa.cgi">Kalign</a></td>
<td>Kalign</td>
<td>1.04, 2.04</td>
</tr>
<tr>
<td>Bioinformatics alignment</td>
<td><a href="http://www.lbgi.fr/~julie/LEON-BIS/">LEON-BIS</a></td>
<td>LEON-BIS</td>
<td>20130322</td>
</tr>
<tr>
<td>Bioinformatics alignment</td>
<td><a href="http://biopp.univ-montp2.fr/forge/maffilter">MafFilter</a></td>
<td>MafFilter</td>
<td>1.1.2</td>
</tr>
<tr>
<td>Bioinformatics alignment</td>
<td><a href="http://mafft.cbrc.jp/alignment/software/">MAFFT</a></td>
<td>MAFFT</td>
<td>7.205, 7.245, 7.310, 7.407</td>
</tr>
<tr>
<td>Bioinformatics alignment</td>
<td><a href="https://github.com/mummer4/mumme">MUMmer</a></td>
<td>MUMmer</td>
<td>3.9.4alpha, 3.22, 3.23, 4.0.0beta2, 4.0.0rc1</td>
</tr>
<tr>
<td>Bioinformatics alignment</td>
<td><a href="https://github.com/lh3/unimap">unimap</a></td>
<td>unimap</td>
<td>0.1-r46-dirty</td>
</tr>
<tr>
<td>Bioinformatics annotation</td>
<td>AAT</td>
<td>AAT</td>
<td>r03052011</td>
</tr>
<tr>
<td>Bioinformatics annotation</td>
<td><a href="http://gemini.readthedocs.org/en/latest/index.html">GEMINI</a></td>
<td>None</td>
<td>0.16.3, 0.18.3, 0.19.0, 0.20.0, 0.20.1, .gemini_0.18.3</td>
</tr>
<tr>
<td>Bioinformatics annotation</td>
<td><a href="https://github.com/BIONF/HaMStR">HaMStR</a></td>
<td>None</td>
<td>13.2.3, .HaMStR, .HaMStR-old</td>
</tr>
<tr>
<td>Bioinformatics annotation</td>
<td><a href="http://www.ebi.ac.uk/interpro/">InterProScan</a></td>
<td>InterProScan</td>
<td>5.52-86.0</td>
</tr>
<tr>
<td>Bioinformatics annotation</td>
<td><a href="https://github.com/dpryan79/libBigWig">libBigWig</a></td>
<td>libBigWig</td>
<td>0.4.4</td>
</tr>
<tr>
<td>Bioinformatics annotation</td>
<td><a href="https://github.com/gatech-genemark/ProtHint">ProtHint</a></td>
<td>ProtHint</td>
<td>2.4.0</td>
</tr>
<tr>
<td>Bioinformatics annotation</td>
<td><a href="https://services.healthtech.dtu.dk/cgi-bin/sw_request">tmhmm</a></td>
<td>tmhmm</td>
<td>2.0c</td>
</tr>
<tr>
<td>Bioinformatics annotation</td>
<td><a href="https://github.com/AnantharamanLab/VIBRANT">VIBRANT</a></td>
<td>VIBRANT</td>
<td>1.2.1</td>
</tr>
<tr>
<td>Bioinformatics assembly</td>
<td><a href="https://sourceforge.net/projects/ngopt/">A5-miseq</a></td>
<td>A5-miseq</td>
<td>20140113, 20140604, 20160825</td>
</tr>
<tr>
<td>Bioinformatics assembly</td>
<td><a href="http://www.bcgsc.ca/platform/bioinfo/software/abyss">abyss</a></td>
<td>abyss</td>
<td>1.3.5, 1.3.5-max, 1.3.7, 1.3.7-k128, 1.3.7-max, 1.5.2, 1.9.0, 1.9.0-k128, 2.0.2, 2.0.2-k128</td>
</tr>
<tr>
<td>Bioinformatics assembly</td>
<td>allpathslg</td>
<td>None</td>
<td>47300, 49618, 52485, 52488</td>
</tr>
<tr>
<td>Bioinformatics assembly</td>
<td><a href="http://amos.sourceforge.net/wiki/index.php/AMOS">AMOS</a></td>
<td>AMOS</td>
<td>3.0.0, 3.1.0</td>
</tr>
<tr>
<td>Bioinformatics assembly</td>
<td><a href="http://ibest.github.io/ARC/">ARC_assembler</a></td>
<td>ARC_assembler</td>
<td>1.1.3</td>
</tr>
<tr>
<td>Bioinformatics assembly</td>
<td><a href="https://github.com/bcgsc/arcs">ARCS</a></td>
<td>ARCS</td>
<td>1.0.6, 1.1.1</td>
</tr>
<tr>
<td>Bioinformatics assembly</td>
<td>BESST</td>
<td>None</td>
<td>1.0.4.3, 1.0.4.4, .besst</td>
</tr>
<tr>
<td>Bioinformatics assembly</td>
<td><a href="https://sites.google.com/site/dbg2olc/">DBG2OLC</a></td>
<td>DBG2OLC</td>
<td>20151208</td>
</tr>
<tr>
<td>Bioinformatics assembly</td>
<td><a href="http://www.broadinstitute.org/software/discovar/blog/">DISCOVARdenovo</a></td>
<td>None</td>
<td>51885, 52488</td>
</tr>
<tr>
<td>Bioinformatics assembly</td>
<td><a href="https://github.com/PacificBiosciences/FALCON_unzip/wiki/Binaries">FALCON</a></td>
<td>FALCON</td>
<td>0.3.0, 0.4.1, 2018.31.08-03.06</td>
</tr>
<tr>
<td>Bioinformatics assembly</td>
<td><a href="https://github.com/PacificBiosciences/FALCON-integrate">FALCON-integrate</a></td>
<td>FALCON-integrate</td>
<td>20161113</td>
</tr>
<tr>
<td>Bioinformatics assembly</td>
<td>Flye</td>
<td>Flye</td>
<td>2.3.5, 2.4.2, 2.8.1</td>
</tr>
<tr>
<td>Bioinformatics assembly</td>
<td><a href="https://sourceforge.net/projects/gaa-wugi">GAA</a></td>
<td>GAA</td>
<td>1.1</td>
</tr>
<tr>
<td>Bioinformatics assembly</td>
<td><a href="http://garm-meta-assem.sourceforge.net/">GARM</a></td>
<td>GARM</td>
<td>0.7, 0.7.3</td>
</tr>
<tr>
<td>Bioinformatics assembly</td>
<td><a href="http://i.cs.hku.hk/%7Ealse/hkubrg/projects/idba_ud/index.html">IDBA</a></td>
<td>None</td>
<td>1.1.1, 1.1.1-384, 1.1.3</td>
</tr>
<tr>
<td>Bioinformatics assembly</td>
<td><a href="https://github.com/bcgsc/links">LINKS</a></td>
<td>LINKS</td>
<td>1.8.7</td>
</tr>
<tr>
<td>Bioinformatics assembly</td>
<td><a href="https://github.com/alekseyzimin/masurca">MaSuRCA</a></td>
<td>MaSuRCA</td>
<td>2.0.3.1, 2.1.0, 2.2.1, 2.3.2, 3.1.3, 3.2.1, 3.2.2, 3.2.3, 3.3.5, 3.4.2</td>
</tr>
<tr>
<td>Bioinformatics assembly</td>
<td>MetAMOS</td>
<td>MetAMOS</td>
<td>1.5rc3</td>
</tr>
<tr>
<td>Bioinformatics assembly</td>
<td><a href="http://www.genomebiology.com/2015/16/1/207">Metassembler</a></td>
<td>Metassembler</td>
<td>1.5</td>
</tr>
<tr>
<td>Bioinformatics assembly</td>
<td><a href="http://mhap.readthedocs.org/en/stable/">MHAP</a></td>
<td>MHAP</td>
<td>1.6</td>
</tr>
<tr>
<td>Bioinformatics assembly</td>
<td><a href="http://platanus.bio.titech.ac.jp/platanus2">Platanus-allee</a></td>
<td>Platanus-allee</td>
<td>2.0.2</td>
</tr>
<tr>
<td>Bioinformatics assembly</td>
<td><a href="https://github.com/mahulchak/quickmerge">quickmerge</a></td>
<td>quickmerge</td>
<td>0.3-9233726</td>
</tr>
<tr>
<td>Bioinformatics assembly</td>
<td><a href="https://github.com/rrwick/Unicycler">Unicycler</a></td>
<td>Unicycler</td>
<td>0.4.8</td>
</tr>
<tr>
<td>Bioinformatics assembly</td>
<td><a href="https://github.com/ruanjue/wtdbg2">wtdbg2</a></td>
<td>wtdbg2</td>
<td>2.4</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/MikkelSchubert/adapterremoval">AdapterRemoval</a></td>
<td>AdapterRemoval</td>
<td>2.1.7, 2.2.2</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/DReichLab/AdmixTools">AdmixTools</a></td>
<td>AdmixTools</td>
<td>5.0-20170312, 5.0-20171024, 7.0.1, 20160803</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://www.genetics.ucla.edu/software/admixture/index.html">ADMIXTURE</a></td>
<td>ADMIXTURE</td>
<td>1.3.0</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://research.pasteur.fr/en/software/alientrimmer/">AlienTrimmer</a></td>
<td>AlienTrimmer</td>
<td>0.4.0</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/ANGSD/angsd">ANGSD</a></td>
<td>ANGSD</td>
<td>0.917, 0.917-g6522d3e, 0.921, 0.933</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://www.broadinstitute.org/scientific-community/science/projects/viral-genomics/athlates">Athlates</a></td>
<td>Athlates</td>
<td>20140426</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://bitbucket.org/phaentu/atlas/wiki/Home">ATLAS_aDNA</a></td>
<td>ATLAS_aDNA</td>
<td>20170510</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://www.embl-hamburg.de/biosaxs/software.html">ATSAS</a></td>
<td>ATSAS</td>
<td>2.8.3-1</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/PacificBiosciences/bam2fastx">bam2fastx</a></td>
<td>bam2fastx</td>
<td>1.3.0-80dbf79</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://sourceforge.net/projects/bambam/">bambam</a></td>
<td>bambam</td>
<td>1.4</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/adamewing/bamsurgeon">Bamsurgeon</a></td>
<td>Bamsurgeon</td>
<td>1.3</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/samtools/bcftools">bcftools</a></td>
<td>bcftools</td>
<td>1.8, 1.10</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/BEETL/BEETL">BEETL</a></td>
<td>BEETL</td>
<td>1.0.2, 1.1.0</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://bitbucket.org/gavinband/bgen">bgen</a></td>
<td>bgen</td>
<td>1.1.4</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="http://bioperl.org">BioPerl</a></td>
<td>BioPerl</td>
<td>1.6.924_Perl5.18.4, 1.7.1_Perl5.24.1, 1.7.2_Perl5.24.1, 1.7.2_Perl5.26.2</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/nanoporetech/bonito">bonito</a></td>
<td>None</td>
<td>0.3.7-cpu, 0.3.8, 0.4.0</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/teichlab/bracer">BraCeR</a></td>
<td>BraCeR</td>
<td>2019_10_03_22e49cb</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/gpertea/cdbfasta">cdbfasta</a></td>
<td>cdbfasta</td>
<td>1.00</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://support.10xgenomics.com/single-cell-atac/software/pipelines/latest/what-is-cell-ranger-atac">cellranger-ATAC</a></td>
<td>None</td>
<td>1.2.0, 2.0.0</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="http://ecogenomics.github.io/CheckM/">CheckM</a></td>
<td>CheckM</td>
<td>1.0.11, 1.0.12, 1.1.3</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://hoohm.github.io/CITE-seq-Count/">CITE-seq-Count</a></td>
<td>CITE-seq-Count</td>
<td>1.4.3</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="http://sites.psu.edu/axtell/software/cleaveland4/">CleaveLand4</a></td>
<td>CleaveLand4</td>
<td>4.3</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="http://tiger.dbs.nus.edu.sg/cnv-seq/">CNV-seq</a></td>
<td>CNV-seq</td>
<td>20140812</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/abyzovlab/CNVnator">CNVnator</a></td>
<td>CNVnator</td>
<td>0.3.2, 0.3.3</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/BinPro/CONCOCT">CONCOCT</a></td>
<td>CONCOCT</td>
<td>0.4.0, 0.4.0~, 0.5.0, 1.1.0</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/Oshlack/Corset/wiki">Corset</a></td>
<td>Corset</td>
<td>1.04, 1.07</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/priyamoorjani/DATES">DATES</a></td>
<td>DATES</td>
<td>753</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/Santy-8128/DosageConvertor">DosageConvertor</a></td>
<td>DosageConvertor</td>
<td>1.0.4</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/nh13/DWGSIM">DWGSIM</a></td>
<td>DWGSIM</td>
<td>0.1.11-6e9a361</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="http://www.ensembl.org/info/docs/api/core/">EnsEMBL-API</a></td>
<td>EnsEMBL-API</td>
<td>87, 94</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="http://genome.sph.umich.edu/wiki/EPACTS">EPACTS</a></td>
<td>EPACTS</td>
<td>3.2.6, 3.2.6_milou, 3.3.0-a5209db</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/Illumina/ExpansionHunter">ExpansionHunter</a></td>
<td>None</td>
<td>2.5.3, 2.5.5</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/ParBLiSS/FastANI">FastANI</a></td>
<td>FastANI</td>
<td>1.2</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/OpenGene/fastp">fastp</a></td>
<td>fastp</td>
<td>0.20.0, 0.23.1</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://sourceforge.net/projects/fcgene/">fcGENE</a></td>
<td>fcGENE</td>
<td>1.0.7</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/rrwick/Filtlong">Filtlong</a></td>
<td>Filtlong</td>
<td>0.2.0</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="http://ccb.jhu.edu/software/FLASH/">FLASH</a></td>
<td>FLASH</td>
<td>1.2.11</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/ndaniel/fusioncatcher">FusionCatcher</a></td>
<td>None</td>
<td>1.00, 1.10, 1.33</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://groups.csail.mit.edu/cgs/gem">GEM</a></td>
<td>GEM</td>
<td>2.7, 3.4</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="http://www.xzlab.org/software.html">GEMMA</a></td>
<td>GEMMA</td>
<td>0.98.1</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="http://exon.gatech.edu/GeneMark">GeneMark</a></td>
<td>GeneMark</td>
<td>2.3-es, 4.32-es, 4.33-es, 4.33-es_Perl5.24.1, 4.38-es, 4.57-es, 4.62-es, 4.68-es</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://kimura.univ-montp2.fr/~rousset/Genepop.htm">Genepop</a></td>
<td>Genepop</td>
<td>4.7</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="http://genometools.org/">GenomeTools</a></td>
<td>GenomeTools</td>
<td>1.5.8, 1.5.9, 1.6.1</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="http://mendel.stanford.edu/SidowLab/downloads/gerp/index.html">GERP++</a></td>
<td>GERP++</td>
<td>20110522</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="http://www.well.ox.ac.uk/%7Ecfreeman/software/gwas/gtool.html">GTOOL</a></td>
<td>GTOOL</td>
<td>0.7.5</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://www.bioinformatics.babraham.ac.uk/projects/hicup/">HiCUP</a></td>
<td>HiCUP</td>
<td>0.7.2</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/samtools/htslib">htslib</a></td>
<td>htslib</td>
<td>1.8, 1.10</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="http://www.broadinstitute.org/igv/">IGV</a></td>
<td>None</td>
<td>2.3.17, 2.3.40, 2.3.92, 2.4.2, 2.8.13</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="http://www.broadinstitute.org/igv/">IGVtools</a></td>
<td>IGVtools</td>
<td>2.3.17, 2.3.40, 2.3.91, 2.3.98</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td>IM</td>
<td>IM</td>
<td>20091217</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/arunsethuraman/ima2p">IMa2p</a></td>
<td>IMa2p</td>
<td>2015-08-09</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="http://mathgen.stats.ox.ac.uk/impute/impute_v2.html">IMPUTE2</a></td>
<td>IMPUTE2</td>
<td>2.3.2</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="http://microbiology.se/software/itsx/">ITSx</a></td>
<td>None</td>
<td>1.0.9, 1.0.11, 1.1-beta, .itsx</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="http://www.earlham.ac.uk/kat-tools">KAT</a></td>
<td>KAT</td>
<td>2.0.4, 2.0.6, 2.0.8, 2.1.1, 2.3.4, 2.4.2, 2.4.2_py3.5.0, 2.4.2_py3.7.2</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="http://kmergenie.bx.psu.edu/">KmerGenie</a></td>
<td>None</td>
<td>1.6741, 1.7039, .kmergenie</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/DerrickWood/kraken">Kraken</a></td>
<td>None</td>
<td>0.10.5-beta, 1.0, 1.1-352e780, 1.1.1, 1.1.1-20210927-375654f</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="http://ccb.jhu.edu/software/kraken/MANUAL.html">Kraken2</a></td>
<td>Kraken2</td>
<td>2.0.8-beta</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/marbl/Krona">Krona</a></td>
<td>Krona</td>
<td>2.7</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="http://csg.sph.umich.edu/chaolong/LASER/">LASER</a></td>
<td>LASER</td>
<td>2.01, 2.02, 2.04</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td>LatentStrainAnalysis</td>
<td>LatentStrainAnalysis</td>
<td>20160322</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://sourceforge.net/projects/ldhelmet/">LDhelmet</a></td>
<td>LDhelmet</td>
<td>1.7, 1.9, 1.10</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/xzhub/LTR_Finder">LTR_Finder</a></td>
<td>LTR_Finder</td>
<td>1.0.5, 1.0.7</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/arq5x/lumpy-sv">LUMPY</a></td>
<td>LUMPY</td>
<td>0.2.12, 0.2.13, 0.2.13-97cf18c, 0.2.13-213a417, 0.3.0</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="http://chipexo.sourceforge.net/">MACE</a></td>
<td>MACE</td>
<td>1.2</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/taoliu/MACS">MACS</a></td>
<td>MACS</td>
<td>2.1.0, 2.1.2, 2.2.6, 3.0.0a6</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://sourceforge.net/p/mageck/wiki/Home/">MAGeCK</a></td>
<td>MAGeCK</td>
<td>0.5.6, 0.5.9.4</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/nanoporetech/medaka">medaka</a></td>
<td>medaka</td>
<td>0.7.1</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="http://compbio.med.harvard.edu/Meerkat/">Meerkat</a></td>
<td>Meerkat</td>
<td>0.189</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://bitbucket.org/berkeleylab/metabat">MetaBat</a></td>
<td>MetaBat</td>
<td>0.26.3, 2.12.1</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="http://csg.sph.umich.edu/abecasis/Metal/">METAL</a></td>
<td>METAL</td>
<td>2011-03-25</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/bioinform/metasv">MetaSV</a></td>
<td>MetaSV</td>
<td>0.5.4</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://microbiology.se/software/metaxa2/">Metaxa2</a></td>
<td>Metaxa2</td>
<td>2.1.3, 2.2</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="http://smithlabresearch.org/software/methpipe/">MethPipe</a></td>
<td>MethPipe</td>
<td>3.4.3, 4.1.1</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/dpryan79/MethylDackel">MethylDackel</a></td>
<td>MethylDackel</td>
<td>0.2.1, 0.5.1</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="http://www.mothur.org/">Mothur</a></td>
<td>Mothur</td>
<td>1.40.5, 1.41.0</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="http://home.uchicago.edu/~rhudson1/source/mksamples.html">ms</a></td>
<td>ms</td>
<td>20071014</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="http://multiqc.info/">MultiQC</a></td>
<td>MultiQC</td>
<td>0.6, 0.7, 0.8, 1.8, 1.9, 1.10</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/wdecoster/nanocomp">NanoComp</a></td>
<td>NanoComp</td>
<td>1.9.2</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://pypi.org/project/NanoPlot/">NanoPlot</a></td>
<td>NanoPlot</td>
<td>1.33.1</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/lucaferretti/npstat">NPStat</a></td>
<td>NPStat</td>
<td>1</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="ftp://ftp.ncbi.nih.gov/pub/seg/nseg">nseg</a></td>
<td>nseg</td>
<td>1.0.1</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/nanoporetech/ont_fast5_api">ont_fast5_api</a></td>
<td>ont_fast5_api</td>
<td>3.1.6</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/Rosemeis/pcangsd">PCAngsd</a></td>
<td>PCAngsd</td>
<td>0.982</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td>PennCNV</td>
<td>PennCNV</td>
<td>20151014</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://code.google.com/p/phantompeakqualtools/">phantompeakqualtools</a></td>
<td>phantompeakqualtools</td>
<td>1.1</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/secastel/phaser">phaser</a></td>
<td>phaser</td>
<td>20210423-5d4926d</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/NationalGenomicsInfrastructure/piper/">piper</a></td>
<td>piper</td>
<td>1.5.1</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/pmelsted/pizzly">pizzly</a></td>
<td>pizzly</td>
<td>0.37.3</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://www.cog-genomics.org/plink/2.0/">plink2</a></td>
<td>plink2</td>
<td>2.00-alpha-2-20180704, 2.00-alpha-2-20190429, 2.00-alpha-2.3-20200124</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/rrwick/Porechop">Porechop</a></td>
<td>Porechop</td>
<td>0.2.4</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="http://prodigal.ornl.gov/">prodigal</a></td>
<td>prodigal</td>
<td>2.6.3</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://www.prsice.info/">PRSice</a></td>
<td>PRSice</td>
<td>2.2.11.b</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/lh3/psmc">psmc</a></td>
<td>psmc</td>
<td>0.6.5-r67-e5f7df5</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="http://eddylab.org/software/recon/">RECON</a></td>
<td>RECON</td>
<td>1.08</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://www.geenivaramu.ee/en/tools/regscan">RegScan</a></td>
<td>RegScan</td>
<td>0.5</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/mmcco/RepeatScout">RepeatScout</a></td>
<td>RepeatScout</td>
<td>1.0.5, 1.0.6</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/grenaud/schmutzi">schmutzi</a></td>
<td>schmutzi</td>
<td>20160424, 20200706-597c6bc</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://www.seqan.de/">SeqAn</a></td>
<td>None</td>
<td>1.4.2, 2.4.0</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://bioinf.shenwei.me/seqkit/">SeqKit</a></td>
<td>SeqKit</td>
<td>0.15.0</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="http://www.bioinformatics.babraham.ac.uk/projects/seqmonk/">seqmonk</a></td>
<td>seqmonk</td>
<td>0.21.0, 0.27.0, 0.32.1, 0.34.1, 1.36.0, 1.37.1</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/popgenmethods/smcpp">SMC++</a></td>
<td>SMC++</td>
<td>1.15.2</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/tseemann/snippy">snippy</a></td>
<td>snippy</td>
<td>4.0, 4.0.5, 4.6.0</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/what-is-space-ranger">spaceranger</a></td>
<td>spaceranger</td>
<td>1.0.0, 1.2.0</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="http://pop-gen.eu/wordpress/software/sweed">SweeD</a></td>
<td>SweeD</td>
<td>4.0.0</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/samtools/htslib">tabix</a></td>
<td>tabix</td>
<td>0.2.6</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/nanoporetech/tombo">Tombo</a></td>
<td>Tombo</td>
<td>1.5.1</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="https://github.com/cc2qe/vawk">vawk</a></td>
<td>vawk</td>
<td>0.0.1</td>
</tr>
<tr>
<td>Bioinformatics misc</td>
<td><a href="http://velocyto.org/velocyto.py/index.html">velocyto</a></td>
<td>velocyto</td>
<td>0.17.17</td>
</tr>
<tr>
<td>Bioinformatics phylogeny</td>
<td><a href="http://stat.sys.i.kyoto-u.ac.jp/prog/consel/">CONSEL</a></td>
<td>CONSEL</td>
<td>0.20</td>
</tr>
<tr>
<td>Bioinformatics phylogeny</td>
<td><a href="https://cme.h-its.org/exelixis/web/software/exabayes/">ExaBayes</a></td>
<td>ExaBayes</td>
<td>1.5, 1.5-mpi</td>
</tr>
<tr>
<td>Bioinformatics phylogeny</td>
<td><a href="http://fastml.tau.ac.il/source.php">FastML</a></td>
<td>FastML</td>
<td>3.1, 3.11</td>
</tr>
<tr>
<td>Bioinformatics phylogeny</td>
<td><a href="http://www.microbesonline.org/fasttree/">FastTree</a></td>
<td>FastTree</td>
<td>2.1.8, 2.1.10</td>
</tr>
<tr>
<td>Bioinformatics phylogeny</td>
<td><a href="http://www.hyphy.org">HyPhy</a></td>
<td>HyPhy</td>
<td>2.5.0, 2.5.0-mpi</td>
</tr>
<tr>
<td>Bioinformatics phylogeny</td>
<td><a href="https://software-ab.informatik.uni-tuebingen.de/download/megan6/welcome.html">MEGAN</a></td>
<td>None</td>
<td>4.70.4, 5.1.5, 5.11.3, 6.3.5, 6.10.5, 6.20.17, .MEGAN</td>
</tr>
<tr>
<td>Bioinformatics phylogeny</td>
<td><a href="http://sco.h-its.org/exelixis/web/software/raxml/index.html">raxml</a></td>
<td>None</td>
<td>7.0.4, 7.2.7, 7.2.8, 7.3.0, 7.4.7, 8.0.20, 8.0.20-mpi, 8.2.0-gcc, 8.2.0-gcc-mpi, 8.2.0-icc, 8.2.0-icc-mpi, 8.2.0_gcc, 8.2.0_icc, 8.2.4-gcc, 8.2.4-gcc-mpi, 8.2.10-gcc, 8.2.10-gcc-mpi, 8.2.12-gcc, 8.2.12-gcc-mpi, .raxml</td>
</tr>
<tr>
<td>Bioinformatics phylogeny</td>
<td><a href="https://github.com/amkozlov/raxml-ng/wiki">RAxML-NG</a></td>
<td>RAxML-NG</td>
<td>0.9.0-mpi</td>
</tr>
<tr>
<td>Bioinformatics pipelines</td>
<td><a href="https://github.com/lh3/fermikit">fermikit</a></td>
<td>fermikit</td>
<td>0.14-prerelease-96f7820, r178</td>
</tr>
<tr>
<td>Bioinformatics pipelines</td>
<td><a href="https://github.com/nf-core/tools">nf-core</a></td>
<td>nf-core</td>
<td>1.12.1, 1.14, 2.1</td>
</tr>
<tr>
<td>Bioinformatics pipelines</td>
<td><a href="https://nf-co.re/pipelines">nf-core-pipelines</a></td>
<td>nf-core-pipelines</td>
<td>latest</td>
</tr>
<tr>
<td>Bioinformatics sw collections</td>
<td>454-dataanalysis</td>
<td>454-dataanalysis</td>
<td>2.3, 2.5.3, 2.6, 2.9</td>
</tr>
<tr>
<td>Bioinformatics sw collections</td>
<td>454-dataprocessing</td>
<td>454-dataprocessing</td>
<td>2.3</td>
</tr>
<tr>
<td>Bioinformatics sw collections</td>
<td><a href="https://www.niehs.nih.gov/research/resources/software/biostatistics/art/">ART</a></td>
<td>ART</td>
<td>2016-06-05</td>
</tr>
<tr>
<td>Bioinformatics sw collections</td>
<td><a href="https://bedops.readthedocs.io">BEDOPS</a></td>
<td>BEDOPS</td>
<td>2.4.3, 2.4.28, 2.4.39</td>
</tr>
<tr>
<td>Bioinformatics sw collections</td>
<td><a href="https://bedtools.readthedocs.io">BEDTools</a></td>
<td>BEDTools</td>
<td>2.21.0, 2.23.0, 2.25.0, 2.26.0, 2.27.1, 2.29.2</td>
</tr>
<tr>
<td>Bioinformatics sw collections</td>
<td><a href="https://www.thermofisher.com/se/en/home/life-science/sequencing/next-generation-sequencing.html">BioScope</a></td>
<td>BioScope</td>
<td>1.3.1</td>
</tr>
<tr>
<td>Bioinformatics sw collections</td>
<td>CASAVA</td>
<td>CASAVA</td>
<td>1.7.0, 1.8.2</td>
</tr>
<tr>
<td>Bioinformatics sw collections</td>
<td><a href="https://support.10xgenomics.com/single-cell/software/pipelines/latest/what-is-cell-ranger">cellranger</a></td>
<td>None</td>
<td>1.1.0, 1.3.0, 2.0.2, 2.2.0, 3.0.1, 4.0.0, 5.0.1, 6.0.2</td>
</tr>
<tr>
<td>Bioinformatics sw collections</td>
<td><a href="https://support.10xgenomics.com/single-cell-multiome-atac-gex/software/downloads/latest">cellranger-ARC</a></td>
<td>cellranger-ARC</td>
<td>1.0.0</td>
</tr>
<tr>
<td>Bioinformatics sw collections</td>
<td><a href="https://support.10xgenomics.com/single-cell-dna/software/downloads/latest">cellranger-DNA</a></td>
<td>cellranger-DNA</td>
<td>1.1.0</td>
</tr>
<tr>
<td>Bioinformatics sw collections</td>
<td><a href="https://www.broadinstitute.org/gatk/">GATK</a></td>
<td>GATK</td>
<td>3.5.0, 3.6, 3.7, 3.8-0, 4.1.0.0, 4.1.1.0, 4.beta.5, build-20160727</td>
</tr>
<tr>
<td>Bioinformatics sw collections</td>
<td>GATK-Queue</td>
<td>GATK-Queue</td>
<td>3.2.2, 3.6, 3.7, 3.8-0</td>
</tr>
<tr>
<td>Bioinformatics sw collections</td>
<td><a href="http://software.broadinstitute.org/software/genomestrip/index.html">GenomeSTRiP</a></td>
<td>GenomeSTRiP</td>
<td>2.00.1650, 2.00.1685, 2.00.1710</td>
</tr>
<tr>
<td>Bioinformatics sw collections</td>
<td>HiSeq</td>
<td>HiSeq</td>
<td>0.9</td>
</tr>
<tr>
<td>Bioinformatics sw collections</td>
<td><a href="https://support.10xgenomics.com/genome-exome/software/pipelines/latest/what-is-long-ranger">longranger</a></td>
<td>longranger</td>
<td>2.0.1, 2.1.1, 2.1.2, 2.1.4, 2.2.2</td>
</tr>
<tr>
<td>Bioinformatics sw collections</td>
<td><a href="http://meme-suite.org">MEMEsuite</a></td>
<td>MEMEsuite</td>
<td>5.0.1</td>
</tr>
<tr>
<td>Bioinformatics sw collections</td>
<td><a href="https://support.10xgenomics.com/de-novo-assembly/software/overview/welcome">supernova</a></td>
<td>None</td>
<td>1.0.0, 1.1.1, 1.1.4, 2.0.0, 2.1.1</td>
</tr>
<tr>
<td>Chemistry/physics</td>
<td><a href="https://www.abinit.org/">ABINIT</a></td>
<td>ABINIT</td>
<td>8.10.3</td>
</tr>
<tr>
<td>Chemistry/physics</td>
<td><a href="http://alps.comp-phys.org/mediawiki/index.php/Main_Page">ALPS</a></td>
<td>ALPS</td>
<td>2.3.0</td>
</tr>
<tr>
<td>Chemistry/physics</td>
<td><a href="http://dock.compbio.ucsf.edu/DOCK3.7/">DOCK</a></td>
<td>DOCK</td>
<td>3.7</td>
</tr>
<tr>
<td>Chemistry/physics</td>
<td><a href="http://www.gromacs.org">gromacs</a></td>
<td>None</td>
<td>4.5.5, 4.5.5_intel, 4.5.7.th, 4.6.3, 4.6.3_rackham, 4.6.5, 4.6.5.th, 4.6.5_rackham, 4.6.7.th, 4.6.7.th.dp, 5.0.4, 5.0.4_rackham, 5.0.7, 5.1.1, 5.1.1_rackham, 5.1.5, 2016.1, 2016.6, 2018.6, 2018.6.th, 2019.1, 2019.1.th, 2019.6.th, 2020-GPU, 2021.1.th, .4.6</td>
</tr>
<tr>
<td>Chemistry/physics</td>
<td>molcas</td>
<td>molcas</td>
<td>7.8.082, 8.0.15-03-08</td>
</tr>
<tr>
<td>Chemistry/physics</td>
<td>molden</td>
<td>molden</td>
<td>5.1</td>
</tr>
<tr>
<td>Chemistry/physics</td>
<td>TmoleX</td>
<td>TmoleX</td>
<td>18</td>
</tr>
<tr>
<td>Compilers and build tools</td>
<td><a href="http://ant.apache.org">ant</a></td>
<td>ant</td>
<td>1.9.8, 1.10.0</td>
</tr>
<tr>
<td>Compilers and build tools</td>
<td><a href="https://www.gnu.org/software/autoconf/autoconf.html">autoconf</a></td>
<td>autoconf</td>
<td>2.68, 2.69</td>
</tr>
<tr>
<td>Compilers and build tools</td>
<td><a href="https://www.gnu.org/software/automake/">automake</a></td>
<td>automake</td>
<td>1.14.1, 1.16.1</td>
</tr>
<tr>
<td>Compilers and build tools</td>
<td><a href="https://www.gnu.org/software/binutils/">binutils</a></td>
<td>binutils</td>
<td>2.26, 2.28, 2.38, 2.39, 2.41</td>
</tr>
<tr>
<td>Compilers and build tools</td>
<td>ddt</td>
<td>ddt</td>
<td>3.2.1, 5.0.1, 6.0, 6.1, 7.0</td>
</tr>
<tr>
<td>Compilers and build tools</td>
<td><a href="https://github.com/westes/flex">flex</a></td>
<td>flex</td>
<td>2.6.4</td>
</tr>
<tr>
<td>Compilers and build tools</td>
<td><a href="https://www.freepascal.org/">fpc</a></td>
<td>fpc</td>
<td>3.0.4</td>
</tr>
<tr>
<td>Compilers and build tools</td>
<td><a href="https://git-scm.com/">git</a></td>
<td>git</td>
<td>2.5.0, 2.10.2, 2.16.1, 2.21.0, 2.24.0, 2.28.0, 2.34.1, 2.44.0</td>
</tr>
<tr>
<td>Compilers and build tools</td>
<td><a href="https://git-lfs.github.com/">git-lfs</a></td>
<td>git-lfs</td>
<td>2.9.1, 3.5.1</td>
</tr>
<tr>
<td>Compilers and build tools</td>
<td><a href="http://www.gnu.org/software/guile/">guile</a></td>
<td>guile</td>
<td>1.8.8</td>
</tr>
<tr>
<td>Compilers and build tools</td>
<td><a href="https://jdk.java.net/java-se-ri/24">java</a></td>
<td>java</td>
<td>OpenJDK, OpenJDK_11.0.2, OpenJDK_12+32, OpenJDK_17+35, OpenJDK_22+36, OpenJDK_24+36, OracleJDK, OracleJDK_11.0.9, sun_jdk1.6.0_04, sun_jdk1.6.0_18, sun_jdk1.6.0_45, sun_jdk1.7.0_25, sun_jdk1.8.0_40, sun_jdk1.8.0_92, sun_jdk1.8.0_151</td>
</tr>
<tr>
<td>Compilers and build tools</td>
<td><a href="https://julialang.org">julia</a></td>
<td>julia</td>
<td>0.3.0-prerelease+3043, 0.3.11, 0.4.6, 1.1.1, 1.4.2, 1.6.1, 1.9.3, 1.10.10_LTS, 1.11.6</td>
</tr>
<tr>
<td>Compilers and build tools</td>
<td><a href="https://www.gnu.org/software/libtool/">libtool</a></td>
<td>libtool</td>
<td>2.4.6</td>
</tr>
<tr>
<td>Compilers and build tools</td>
<td><a href="https://maven.apache.org/index.html">maven</a></td>
<td>maven</td>
<td>3.6.0</td>
</tr>
<tr>
<td>Compilers and build tools</td>
<td>meson</td>
<td>meson</td>
<td>0.49.2, 0.57.2, 1.1.0</td>
</tr>
<tr>
<td>Compilers and build tools</td>
<td><a href="http://www.mono-project.com/">mono</a></td>
<td>mono</td>
<td>3.12.0, 5.8.1.0, 5.10.1.27</td>
</tr>
<tr>
<td>Compilers and build tools</td>
<td><a href="https://ninja-build.org/">ninja</a></td>
<td>ninja</td>
<td>1.9.0, 1.10.0</td>
</tr>
<tr>
<td>Compilers and build tools</td>
<td><a href="https://nixos.org/patchelf.html">patchelf</a></td>
<td>patchelf</td>
<td>0.1, 0.8</td>
</tr>
<tr>
<td>Compilers and build tools</td>
<td><a href="https://www.perl.org/">perl</a></td>
<td>perl</td>
<td>5.18.2, 5.18.4, 5.22.2, 5.24.0, 5.24.1, 5.26.2, 5.32.1</td>
</tr>
<tr>
<td>Compilers and build tools</td>
<td><a href="https://perl6.org/">perl6</a></td>
<td>perl6</td>
<td>rakudo-star-2017.04, rakudo-star-2019.03</td>
</tr>
<tr>
<td>Compilers and build tools</td>
<td>perl_modules</td>
<td>perl_modules</td>
<td>5.18.4, 5.24.1, 5.26.2, 5.32.1</td>
</tr>
<tr>
<td>Compilers and build tools</td>
<td><a href="https://python.org/">python</a></td>
<td>None</td>
<td>2.5, 2.6, 2.6.1, 2.6.5, 2.6.6, 2.7, 2.7.1, 2.7.2, 2.7.4, 2.7.6, 2.7.9, 2.7.11, 2.7.15, 2.7i, 3.1, 3.1.2, 3.1.3, 3.2, 3.2.4, 3.3, 3.3.1, 3.4.3, 3.5.0, 3.6.0, 3.6.8, 3.7.2, 3.8.7, 3.9.5, 3.10.8, 3.11.8, 3.12.1, 3.12.7, 3.13.11, python-rpath.tcl, python-set-LD_LIBRARY_PATH.tcl, python-set-LD_LIBRARY_PATH_new.tcl</td>
</tr>
<tr>
<td>Compilers and build tools</td>
<td><a href="https://python.org/">python3</a></td>
<td>None</td>
<td>3.6.0, 3.6.8, 3.7.2, 3.8.7, 3.9.5, 3.11.4, 3.11.8, 3.12.1, 3.12.7, 3.13.11, python3-rpath.tcl</td>
</tr>
<tr>
<td>Compilers and build tools</td>
<td><a href="https://www.ruby-lang.org/en/">ruby</a></td>
<td>ruby</td>
<td>2.4.1, 2.5.0, 2.6.2</td>
</tr>
<tr>
<td>Compilers and build tools</td>
<td><a href="https://subversion.apache.org/">subversion</a></td>
<td>None</td>
<td>1.9.3, 1.10.6</td>
</tr>
<tr>
<td>Engineering</td>
<td>matlab</td>
<td>None</td>
<td>7.4, 7.8, 7.10, 7.13, 8.0, 8.1, .matlab, .matlab_ny, .matlab_ny~, R2014a, R2015a, R2015b, R2016a, R2017a, R2018a, R2018b, R2019a, R2020b</td>
</tr>
<tr>
<td>Geospatial</td>
<td><a href="https://code.mpimet.mpg.de/projects/cdo">CDO</a></td>
<td>CDO</td>
<td>1.9.5, 1.9.7.1, 1.9.7.1-intel18.3</td>
</tr>
<tr>
<td>Geospatial</td>
<td><a href="https://confluence.ecmwf.int/display/ECC/ecCodes+Home">ecCodes</a></td>
<td>ecCodes</td>
<td>2.13.1</td>
</tr>
<tr>
<td>Geospatial</td>
<td><a href="https://github.com/kartverket/fyba">FYBA</a></td>
<td>FYBA</td>
<td>4.1.1</td>
</tr>
<tr>
<td>Geospatial</td>
<td><a href="https://trac.osgeo.org/geos/">GEOS</a></td>
<td>GEOS</td>
<td>3.5.0, 3.9.1-gcc9.3.0, 3.12.0-gcc12.3.0</td>
</tr>
<tr>
<td>Geospatial</td>
<td><a href="https://gotm.net/">GOTM</a></td>
<td>GOTM</td>
<td>5.3-221-gac7ec88d</td>
</tr>
<tr>
<td>Geospatial</td>
<td><a href="https://github.com/OSGeo/libgeotiff">libgeotiff</a></td>
<td>libgeotiff</td>
<td>1.4.1, 1.4.3, 1.7.1</td>
</tr>
<tr>
<td>Geospatial</td>
<td><a href="https://confluence.ecmwf.int/display/MAGP/Magics">Magics</a></td>
<td>Magics</td>
<td>3.3.1, 3.3.1-intel18.3</td>
</tr>
<tr>
<td>Geospatial</td>
<td><a href="http://nco.sourceforge.net/">NCO</a></td>
<td>NCO</td>
<td>4.8.1, 4.9.2, 4.9.3</td>
</tr>
<tr>
<td>Geospatial</td>
<td><a href="https://proj.org">PROJ</a></td>
<td>PROJ</td>
<td>6.3.2</td>
</tr>
<tr>
<td>Geospatial</td>
<td><a href="https://github.com/OSGeo/proj.4/wiki">PROJ.4</a></td>
<td>PROJ.4</td>
<td>4.9.2, 4.9.3, 4.9.3-intel18.3</td>
</tr>
<tr>
<td>Geospatial</td>
<td><a href="https://qgis.org/en/site/">QGIS</a></td>
<td>QGIS</td>
<td>3.4.12, 3.32.3</td>
</tr>
<tr>
<td>Geospatial</td>
<td><a href="http://shapelib.maptools.org/">SHAPELIB</a></td>
<td>SHAPELIB</td>
<td>1.5.0</td>
</tr>
<tr>
<td>Libraries</td>
<td><a href="http://math-atlas.sourceforge.net/">ATLAS</a></td>
<td>ATLAS</td>
<td>3.10.3</td>
</tr>
<tr>
<td>Libraries</td>
<td><a href="http://www.netlib.org/blas/">blas</a></td>
<td>blas</td>
<td>3.6.0</td>
</tr>
<tr>
<td>Libraries</td>
<td><a href="https://www.boost.org/">boost</a></td>
<td>boost</td>
<td>1.41.0_gcc9.3.0, 1.44.0, 1.45.0, 1.55.0, 1.55.0_gcc4.8.3, 1.58.0-gcc8.3.0, 1.59.0_gcc4.9.2, 1.59.0_intel15.3, 1.60.0_gcc5.3.0, 1.61.0_gcc5.3.0, 1.63.0_gcc6.3.0, 1.63.0_gcc6.3.0_mpi2.0.2, 1.63.0_intel17.1, 1.66.0, 1.66.0-gcc8.3.0, 1.70.0_gcc9.1.0, 1.70.0_gcc9.1.0_mpi3.1.3, 1.70.0_gcc9.3.0, 1.70.0_gcc9.3.0_mpi3.1.5, 1.70.0_intel18.3, 1.70.0_intel18.3_intelmpi18.3, 1.70.0_intel18.3_mpi3.1.3, 1.75.0-gcc9.3.0, 1.78.0_gcc11.2.0, 1.78.0_gcc11.2.0_mpi4.1.2, 1.79.0_gcc11.2.0_mpi4.1.2, 1.81.0-gcc10.3.0, 1.83.0-gcc12.3.0, 1.83.0-gcc12.3.0-mpi4.1.5</td>
</tr>
<tr>
<td>Libraries</td>
<td><a href="https://sourceware.org/bzip2">bzip2</a></td>
<td>bzip2</td>
<td>1.0.6, 1.0.8</td>
</tr>
<tr>
<td>Libraries</td>
<td><a href="https://www.cairographics.org/">cairo</a></td>
<td>cairo</td>
<td>1.14.8, 1.14.12, 1.17.2, 1.17.4</td>
</tr>
<tr>
<td>Libraries</td>
<td><a href="https://www.dealii.org/">deal.II</a></td>
<td>deal.II</td>
<td>9.1.1-gcc, 9.1.1-intel</td>
</tr>
<tr>
<td>Libraries</td>
<td><a href="http://www.fftw.org">fftw</a></td>
<td>fftw</td>
<td>3.3.8</td>
</tr>
<tr>
<td>Libraries</td>
<td><a href="https://freetype.org/">freetype</a></td>
<td>freetype</td>
<td>2.6, 2.7.1, 2.10.1, 2.12.1</td>
</tr>
<tr>
<td>Libraries</td>
<td><a href="https://sourceforge.net/projects/giflib/">giflib</a></td>
<td>giflib</td>
<td>5.1.4</td>
</tr>
<tr>
<td>Libraries</td>
<td><a href="https://www.gnu.org/software/glpk/">glpk</a></td>
<td>glpk</td>
<td>4.63, 4.65</td>
</tr>
<tr>
<td>Libraries</td>
<td><a href="https://www.gnu.org/software/gsl/">gsl</a></td>
<td>gsl</td>
<td>1.16, 2.1, 2.3, 2.5, 2.6, 2.7</td>
</tr>
<tr>
<td>Libraries</td>
<td><a href="https://support.hdfgroup.org/products/hdf4/">hdf4</a></td>
<td>hdf4</td>
<td>4.2.11_gcc4.9.2, 4.2.14-gcc6.3.0</td>
</tr>
<tr>
<td>Libraries</td>
<td><a href="https://www.hdfgroup.org/solutions/hdf5/">hdf5</a></td>
<td>hdf5</td>
<td>1.8.16_gcc4.9.2, 1.8.16_gcc5.3.0, 1.8.18, 1.8.18_gcc6.3.0, 1.10.1, 1.10.5, 1.10.5-threadsafe-intel18.3, 1.10.9, 1.14.0</td>
</tr>
<tr>
<td>Libraries</td>
<td><a href="http://jemalloc.net/">jemalloc</a></td>
<td>jemalloc</td>
<td>3.6.0, 5.0.1, 5.3.0</td>
</tr>
<tr>
<td>Libraries</td>
<td><a href="https://curl.se/">libcurl</a></td>
<td>libcurl</td>
<td>7.45.0, 8.4.0</td>
</tr>
<tr>
<td>Libraries</td>
<td><a href="http://libharu.org/">libharu</a></td>
<td>libharu</td>
<td>2.3.0</td>
</tr>
<tr>
<td>Libraries</td>
<td><a href="https://github.com/webmproject/libwebp">libwebp</a></td>
<td>libwebp</td>
<td>1.0.3, 1.2.0, 1.3.0</td>
</tr>
<tr>
<td>Libraries</td>
<td><a href="https://sourceforge.net/projects/lpsolve/">lpsolve</a></td>
<td>lpsolve</td>
<td>5.5.2.9</td>
</tr>
<tr>
<td>Libraries</td>
<td><a href="http://www.unidata.ucar.edu/software/netcdf/">netcdf</a></td>
<td>netcdf</td>
<td>4.7.1, 4.7.1-intel18.3</td>
</tr>
<tr>
<td>Libraries</td>
<td><a href="https://github.com/stevengj/nlopt">NLopt</a></td>
<td>NLopt</td>
<td>2.6.1</td>
</tr>
<tr>
<td>Libraries</td>
<td><a href="https://github.com/xianyi/OpenBLAS">openblas</a></td>
<td>openblas</td>
<td>0.2.14a, 0.2.19, 0.2.19-singlethread, 0.2.20, 0.2.20-openmp, 0.2.20-singlethread, 0.3.21, 0.3.26, 0.3.29</td>
</tr>
<tr>
<td>Libraries</td>
<td><a href="http://www.pcre.org/">pcre</a></td>
<td>pcre</td>
<td>8.40</td>
</tr>
<tr>
<td>Libraries</td>
<td><a href="http://plplot.sourceforge.net/">PLplot</a></td>
<td>PLplot</td>
<td>5.15.0, 5.15.0-old_wxWidgets</td>
</tr>
<tr>
<td>Libraries</td>
<td><a href="https://poppler.freedesktop.org">Poppler</a></td>
<td>Poppler</td>
<td>0.43.0, 0.54.0, 0.75.0, 23.02.0, 23.09.0</td>
</tr>
<tr>
<td>Libraries</td>
<td><a href="https://github.com/protocolbuffers/protobuf">protobuf</a></td>
<td>protobuf</td>
<td>3.11.4, 24.3-gcc12.3.0</td>
</tr>
<tr>
<td>Libraries</td>
<td><a href="http://pslib.sourceforge.net">pslib</a></td>
<td>pslib</td>
<td>0.4.6</td>
</tr>
<tr>
<td>Libraries</td>
<td><a href="https://github.com/natefoo/slurm-drmaa">slurm-drmaa</a></td>
<td>slurm-drmaa</td>
<td>1.1.2-slurm19.05.8, 1.1.4-slurm23.02.5</td>
</tr>
<tr>
<td>Libraries</td>
<td><a href="https://github.com/sparsehash/sparsehash">sparsehash</a></td>
<td>None</td>
<td>2.0.2, 2.0.3, 2.0.4</td>
</tr>
<tr>
<td>Libraries</td>
<td><a href="https://www.hdfgroup.org/doc_resource/SZIP/">szip</a></td>
<td>None</td>
<td>2.1.1, 2.1_gcc4.9.2, 2.1_gcc5.3.0</td>
</tr>
<tr>
<td>Libraries</td>
<td><a href="https://threadedbuildingblocks.org">tbb</a></td>
<td>tbb</td>
<td>4.4u1_gcc4.9.2, 4.4u1_intel15.3, 2020.3</td>
</tr>
<tr>
<td>Libraries</td>
<td><a href="https://www.unidata.ucar.edu/software/udunits/">UDUNITS</a></td>
<td>UDUNITS</td>
<td>2.2.26</td>
</tr>
<tr>
<td>Libraries</td>
<td><a href="https://www.wxwidgets.org/">wxWidgets</a></td>
<td>wxWidgets</td>
<td>3.1.3-gtk2, 3.1.3-gtk3, 3.1.3-qt5</td>
</tr>
<tr>
<td>Libraries</td>
<td><a href="http://www.yeppp.info/">Yeppp</a></td>
<td>Yeppp</td>
<td>1.0.0</td>
</tr>
<tr>
<td>Libraries</td>
<td><a href="https://zlib.net/">zlib</a></td>
<td>zlib</td>
<td>1.2.8, 1.2.11, 1.2.13, 1.3, 1.3.1</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/getzlab/ABSOLUTE">ABSOLUTE</a></td>
<td>ABSOLUTE</td>
<td>2023-6c98496</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/MikkelSchubert/adapterremoval">AdapterRemoval</a></td>
<td>AdapterRemoval</td>
<td>2.3.1, 2.3.4</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://www.opensourceimaging.org/project/afni/">AFNI</a></td>
<td>AFNI</td>
<td>24.3.08</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/NBISweden/AGAT">AGAT</a></td>
<td>AGAT</td>
<td>1.0.0, 1.3.2</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://www.gear-genomics.com/alfred/">alfred</a></td>
<td>alfred</td>
<td>0.3.2</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/cancerit/alleleCount">alleleCount</a></td>
<td>alleleCount</td>
<td>4.2.1</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/ANGSD/angsd">ANGSD</a></td>
<td>ANGSD</td>
<td>0.700, 0.902, 0.915, 0.940-stable</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://anvio.org/">Anvio</a></td>
<td>Anvio</td>
<td>9</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/tseemann/any2fasta">any2fasta</a></td>
<td>any2fasta</td>
<td>0.4.2</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="http://cmpg.unibe.ch/software/arlequin35/">Arlequin</a></td>
<td>Arlequin</td>
<td>3.5.2.2</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://acg-team.github.io/bpp-arpip/">ARPIP</a></td>
<td>ARPIP</td>
<td>2023.10.02-ee32c10</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://sites.google.com/view/ashs-dox/home">ASHS</a></td>
<td>ASHS</td>
<td>2.0.0</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/pezmaster31/bamtools">bamtools</a></td>
<td>bamtools</td>
<td>2.5.2</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/statgen/bamUtil">bamUtil</a></td>
<td>bamUtil</td>
<td>1.0.15</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="http://www.htslib.org/doc/bcftools.html">bcftools</a></td>
<td>bcftools</td>
<td>1.19, 1.20</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://faculty.washington.edu/browning/beagle/beagle.html">Beagle</a></td>
<td>Beagle</td>
<td>4.1</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://enkre.net/cgi-bin/code/bgen/dir?ci=trunk">bgen</a></td>
<td>bgen</td>
<td>1.1.6</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/biobakery">BioBakery</a></td>
<td>BioBakery</td>
<td>3.0, 3.1, 3.8</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/JLSteenwyk">BioKIT</a></td>
<td>BioKIT</td>
<td>0.0.9</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://bioperl.org/">BioPerl</a></td>
<td>None</td>
<td>1.6.1, 1.6.1_PERL5.10.1, 1.6.1_PERL5.12.3, 1.6.922, 1.6.923_Perl5.18.4, 1.7.8-perl5.32.1, .BioPerl</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://busco.ezlab.org/">BUSCO</a></td>
<td>BUSCO</td>
<td>4.1.4, 5.3.1, 5.5.0, 5.7.1</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/single-cell-genetics/cellsnp-lite">Cellsnp-lite</a></td>
<td>Cellsnp-lite</td>
<td>1.2.2</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/comprna/CHEUI">CHEUI</a></td>
<td>CHEUI</td>
<td>20230518-c1c9ab6</td>
</tr>
<tr>
<td>Misc</td>
<td>Chromium-cellranger-ATAC</td>
<td>cellranger-ATAC</td>
<td>1.2.0, 2.0.0, 2.1.0</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/what-is-space-ranger">Chromium-spaceranger</a></td>
<td>spaceranger</td>
<td>1.0.0, 4.0.1</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://people.maths.bris.ac.uk/~madjl/finestructure-old/index.html">ChromoPainter</a></td>
<td>ChromoPainter</td>
<td>0.0.4</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/amcpherson/citup">Citup</a></td>
<td>Citup</td>
<td>0.1.0</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/gjeunen/reference_database_creator">CRABS</a></td>
<td>CRABS</td>
<td>0.1.2</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/pinellolab/CRISPResso2">CRISPResso</a></td>
<td>CRISPResso</td>
<td>1.0.7, 2.3.1</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://damageprofiler.readthedocs.io/en/latest/#">DamageProfiler</a></td>
<td>DamageProfiler</td>
<td>1.1</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://www.gnu.org/software/datamash">datamash</a></td>
<td>datamash</td>
<td>1.8</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/MoorjaniLab">DATES</a></td>
<td>DATES</td>
<td>4010</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://pypi.org/project/dds-cli/">dds-cli</a></td>
<td>dds-cli</td>
<td>latest</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/deeptools/deepTools">deepTools</a></td>
<td>deepTools</td>
<td>3.5.5, 3.5.6</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/tony-kuo/eagle">EAGLE</a></td>
<td>EAGLE</td>
<td>1.1.3</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://vaquerizaslab.github.io/fanc/index.html">FAN-C</a></td>
<td>FAN-C</td>
<td>0.9.26</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/ParBLiSS/FastANI">FastANI</a></td>
<td>FastANI</td>
<td>1.33, 1.34</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/thegenemyers/FASTK">fastK</a></td>
<td>fastK</td>
<td>1.1.0</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/OpenGene/fastp">fastp</a></td>
<td>fastp</td>
<td>0.23.4</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="http://fulcrumgenomics.github.io/fgbio/">fgbio</a></td>
<td>fgbio</td>
<td>2.2.1-0</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://www.milan-malinsky.org/fineradstructure">fineRADstructure</a></td>
<td>fineRADstructure</td>
<td>0.3.1</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/LuyiTian/FLAMES">FLAMES</a></td>
<td>FLAMES</td>
<td>20221109-774e16a</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/dstreett/FLASH2">FLASH</a></td>
<td>FLASH</td>
<td>2.2.00</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/ekg/freebayes">freebayes</a></td>
<td>freebayes</td>
<td>1.3.8</td>
</tr>
<tr>
<td>Misc</td>
<td>fsl</td>
<td>fsl</td>
<td>6.0, 6.0.3</td>
</tr>
<tr>
<td>Misc</td>
<td>gdc-client</td>
<td>gdc-client</td>
<td>2.3</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/gpertea/gffread">gffread</a></td>
<td>gffread</td>
<td>0.12.7</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/ryanlayer/giggle">giggle</a></td>
<td>giggle</td>
<td>0.6.3</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://odelaneau.github.io/GLIMPSE/">GLIMPSE</a></td>
<td>GLIMPSE</td>
<td>1.1.1, 2.0.0</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/lczech/grenedalf">grenedalf</a></td>
<td>grenedalf</td>
<td>0.3.0, 0.6.0</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/Ecogenomics/GTDBTk">GTDB-Tk</a></td>
<td>GTDB-Tk</td>
<td>0.3.2, 1.5.0, 2.3.2, 2.4.0</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://nanoporetech.com/nanopore-sequencing-data-analysis">Guppy</a></td>
<td>Guppy</td>
<td>5.0.16-cpu, 5.0.16-gpu, 6.0.6-cpu, 6.0.6-gpu, 6.3.7-cpu, 6.3.7-gpu, 6.4.2-cpu, 6.4.2-gpu, 6.5.7-cpu, 6.5.7-gpu</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://huttenhower.sph.harvard.edu/halla">halla</a></td>
<td>halla</td>
<td>0.8.20</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://hicexplorer.readthedocs.io/en/latest/index.html">HiCExplorer</a></td>
<td>HiCExplorer</td>
<td>2.2-beta, 3.7.3</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/samtools/htslib">htslib</a></td>
<td>htslib</td>
<td>1.19, 1.20</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/biobakery/humann3">HUMAnN</a></td>
<td>HUMAnN</td>
<td>3.6, 3.8</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/LarsNauheimer/HybPhaser">HybPhaser</a></td>
<td>HybPhaser</td>
<td>2.1.0</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="http://software.broadinstitute.org/software/igv/home">IGVtools</a></td>
<td>IGVtools</td>
<td>2.8.13, 2.16.0</td>
</tr>
<tr>
<td>Misc</td>
<td>IMPUTE5</td>
<td>IMPUTE5</td>
<td>1.2.0</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://www.itksnap.org/pmwiki/pmwiki.php">ITK-SNAP</a></td>
<td>ITK-SNAP</td>
<td>4.4.0</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/aidenlab/Juicebox/wiki/Download">Juicebox</a></td>
<td>Juicebox</td>
<td>1.11.08</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://kingrelatedness.com/">KING</a></td>
<td>KING</td>
<td>2.3.2</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/wwood/kingfisher-download">kingfisher</a></td>
<td>kingfisher</td>
<td>0.3.0</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/refresh-bio/KMC">KMC</a></td>
<td>KMC</td>
<td>3.2.2</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/biobakery/kneaddata">KneadData</a></td>
<td>KneadData</td>
<td>0.12.0</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/DerrickWood/kraken2">Kraken2</a></td>
<td>Kraken2</td>
<td>2.1.3-20231102-acc2248</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/fbreitwieser/KrakenUniq">KrakenUniq</a></td>
<td>KrakenUniq</td>
<td>0.6, 1.0.0, 1.0.1</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/marbl/Krona">Krona</a></td>
<td>Krona</td>
<td>2.7.1, 2.8.1-20211222-d1479b3</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://pypi.org/project/ldsc/">ldsc</a></td>
<td>ldsc</td>
<td>1.0.0, 1.0.1, 2.0.1</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://macs3-project.github.io/MACS/">MACS</a></td>
<td>MACS</td>
<td>1.4.1, 1.4.2, 3.0.0b1</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/caleblareau/maegatk">maegatk</a></td>
<td>maegatk</td>
<td>0.2.0, 0.2.1</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/davidliwei/mageck2">MAGeCK2</a></td>
<td>MAGeCK2</td>
<td>20211209-435eacd</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="http://ginolhac.github.io/mapDamage/">mapDamage</a></td>
<td>mapDamage</td>
<td>2.2.3</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://mash.readthedocs.io/en/latest/">mash</a></td>
<td>mash</td>
<td>2.0, 2.3, 2.3-20210519-41ddc61</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/marbl/meryl">meryl</a></td>
<td>meryl</td>
<td>1.4.1</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/statgen/METAL">METAL</a></td>
<td>METAL</td>
<td>2020-05-05</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/hakyimlab/MetaXcan">MetaXcan</a></td>
<td>MetaXcan</td>
<td>20210925-cfc9e36</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="http://legacy.bioinf.uni-leipzig.de/Software/metilene/">metilene</a></td>
<td>metilene</td>
<td>0.2-8</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/caleblareau/mgatk">mgatk</a></td>
<td>mgatk</td>
<td>0.7.0</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/statgen/Minimac4">Minimac4</a></td>
<td>Minimac4</td>
<td>4.1.6</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/rajewsky-lab/mirdeep2">mirdeep2</a></td>
<td>mirdeep2</td>
<td>2.0.1.3-20220221-c6440e2</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://sourceforge.net/projects/mirdp2/">miRDP2</a></td>
<td>miRDP2</td>
<td>1.1.4</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://mixcr.com">MiXCR</a></td>
<td>MiXCR</td>
<td>4.7.0</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/nanoporetech/modkit">modkit</a></td>
<td>modkit</td>
<td>0.2.5-rc2, 0.3.1, 0.3.3, 0.4.1, 0.5.1-rc1</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/brentp/mosdepth">mosdepth</a></td>
<td>mosdepth</td>
<td>0.3.3</td>
</tr>
<tr>
<td>Misc</td>
<td>Mothur</td>
<td>Mothur</td>
<td>1.25.1, 1.30.1, 1.33.3, 1.36.1, 1.38.1, 1.48.0</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://www.nitrc.org/projects/mricrogl/">MRIcroGL</a></td>
<td>MRIcroGL</td>
<td>1.2.20220720</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/xjtu-omics/msisensor-pro">msisensor-pro</a></td>
<td>msisensor-pro</td>
<td>1.2.0</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://multiqc.info">MultiQC</a></td>
<td>MultiQC</td>
<td>1.22.2</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://www.ncbi.nlm.nih.gov/datasets/docs/v2/download-and-install/">NCBI-datasets</a></td>
<td>NCBI-datasets</td>
<td>15.29.0, 16.35.0</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/bbglab/oncodrive3d">oncodrive3d</a></td>
<td>oncodrive3d</td>
<td>1.0.7</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/nanoporetech/ont_h5_validator">ont_h5_validator</a></td>
<td>ont_h5_validator</td>
<td>2.0.1</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/richarddurbin/pbwt">pbwt</a></td>
<td>pbwt</td>
<td>2.1</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://pharmcat.clinpgx.org/">PharmCAT</a></td>
<td>PharmCAT</td>
<td>3.1.1</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://phyluce.readthedocs.io/en/latest/">phyluce</a></td>
<td>phyluce</td>
<td>1.7.3</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://broadinstitute.github.io/picard/">picard</a></td>
<td>picard</td>
<td>3.1.1</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/Illumina/Pisces/releases/tag/v5.3.0.0">Pisces</a></td>
<td>Pisces</td>
<td>5.3.0.0</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://www.cog-genomics.org/plink/2.0/">plink2</a></td>
<td>plink2</td>
<td>2.00-alpha-3.7-20221024, 2.00-alpha-5-20230923, 2.00-alpha-6.20-20250707</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/smithlabcode/preseq">preseq</a></td>
<td>preseq</td>
<td>3.2</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://proj.org">PROJ</a></td>
<td>PROJ</td>
<td>8.1.0, 9.1.1</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/Roth-Lab/pyclone-vi">PyClone-VI</a></td>
<td>PyClone-VI</td>
<td>20210623-6607ea1</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/EGA-archive/ega-download-client">pyega3</a></td>
<td>pyega3</td>
<td>5.1.0</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/statgen/raremetal">Raremetal</a></td>
<td>Raremetal</td>
<td>4.15.1</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://refgenie.databio.org/en/latest/">refgenie</a></td>
<td>refgenie</td>
<td>0.12.1</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://rgcgithub.github.io/regenie">regenie</a></td>
<td>regenie</td>
<td>3.4.1</td>
</tr>
<tr>
<td>Misc</td>
<td>removethis</td>
<td>removethis</td>
<td>1</td>
</tr>
<tr>
<td>Misc</td>
<td>RepeatModeler</td>
<td>RepeatModeler</td>
<td>2.0.4</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/CostaLab/reg-gen">RGT</a></td>
<td>RGT</td>
<td>1.0.2</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/RealTimeGenomics/rtg-core">rtgcore</a></td>
<td>rtgcore</td>
<td>3.12.1</td>
</tr>
<tr>
<td>Misc</td>
<td>rtgtools</td>
<td>rtgtools</td>
<td>3.12.1</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/samtools/samtools">samtools</a></td>
<td>samtools</td>
<td>1.7, 1.19, 1.20</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://scanpy.readthedocs.io/en/stable/index.html">scanpy</a></td>
<td>scanpy</td>
<td>1.11.0</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/grenaud/schmutzi">schmutzi</a></td>
<td>schmutzi</td>
<td>1.5.7</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://scikit-allel.readthedocs.io/en/stable/">scikit-allel</a></td>
<td>scikit-allel</td>
<td>1.3.5</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/scverse/scvi-tools">scvi-tools</a></td>
<td>scvi-tools</td>
<td>1.0.4</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://bioinf.shenwei.me/seqkit/">SeqKit</a></td>
<td>SeqKit</td>
<td>2.4.0</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/walaj/SeqLib">SeqLib</a></td>
<td>SeqLib</td>
<td>1.2.0</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://guertinlab.github.io/seqOutBias">seqOutBias</a></td>
<td>seqOutBias</td>
<td>1.4.0</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/clwgg/seqstats">seqstats</a></td>
<td>seqstats</td>
<td>20170404-e6f482f</td>
</tr>
<tr>
<td>Misc</td>
<td>SHAPEIT</td>
<td>SHAPEIT</td>
<td>v4.2.2, v5.1.1</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/relipmoc/skewer">skewer</a></td>
<td>skewer</td>
<td>0.2.2</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/brentp/slivar">slivar</a></td>
<td>slivar</td>
<td>0.2.7</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/popgenmethods/smcpp">SMC++</a></td>
<td>SMC++</td>
<td>1.15.4, 1.15.5.dev12+g8bdecdf</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/KamilSJaron/smudgeplot">smudgeplot</a></td>
<td>smudgeplot</td>
<td>0.3, 0.4.0</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://bioinformaticshome.com/tools/gwas/descriptions/SNPTEST.html">SNPTEST</a></td>
<td>SNPTEST</td>
<td>2.5.6</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/YaolingYang/SparsePainter">SparsePainter</a></td>
<td>SparsePainter</td>
<td>1.2.1</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://squidpy.readthedocs.io/en/stable/index.html">squidpy</a></td>
<td>squidpy</td>
<td>1.6.5</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/lanl/sracat">sracat</a></td>
<td>sracat</td>
<td>20210916-b896745</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/ncbi/sra-tools">sratools</a></td>
<td>sratools</td>
<td>3.0.7</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="http://catchenlab.life.illinois.edu/stacks/">Stacks</a></td>
<td>Stacks</td>
<td>2.66</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/J35P312/SVDB">SVDB</a></td>
<td>SVDB</td>
<td>2.8.1</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/ACEnglish/truvari">truvari</a></td>
<td>truvari</td>
<td>4.3.1</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/vcflib/vcflib">vcflib</a></td>
<td>vcflib</td>
<td>1.0.9, 1.0.14</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/pwwang/vcfstats">vcfstats</a></td>
<td>vcfstats</td>
<td>0.4.2</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/Ensembl/ensembl-vep">vep</a></td>
<td>vep</td>
<td>110.1, 111.0, 113.0</td>
</tr>
<tr>
<td>Misc</td>
<td><a href="https://github.com/hardingnj/xpclr">XP-CLR</a></td>
<td>XP-CLR</td>
<td>1.1.2</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="http://www.tgries.de/agrep/">agrep</a></td>
<td>agrep</td>
<td>3.41.5</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="https://aws.amazon.com/cli">awscli</a></td>
<td>awscli</td>
<td>1.11.140, 1.16.225, 1.29.52</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="http://circos.ca">circos</a></td>
<td>circos</td>
<td>0.69-9</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="https://cmake.org/">cmake</a></td>
<td>cmake</td>
<td>3.5.1, 3.7.2, 3.13.2, 3.17.3</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="https://github.com/schacon/cowsay">cowsay</a></td>
<td>cowsay</td>
<td>3.03</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="https://github.com/broadinstitute/cromwell">Cromwell</a></td>
<td>Cromwell</td>
<td>71, 86</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="http://www.stack.nl/%7Edimitri/doxygen/">doxygen</a></td>
<td>doxygen</td>
<td>1.8.11</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="https://www.gnu.org/savannah-checkouts/gnu/emacs/emacs.html">emacs</a></td>
<td>emacs</td>
<td>25.1, 25.2, 27.2, 28.2</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="https://www.gnu.org/software/gawk/">gawk</a></td>
<td>gawk</td>
<td>4.1.4</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="https://github.com/gnudatalanguage/gdl">gdl</a></td>
<td>gdl</td>
<td>1.0.0-rc.1</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="https://ghostscript.com">GhostPDL</a></td>
<td>GhostPDL</td>
<td>9.53.3</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="http://www.gnuplot.info/">gnuplot</a></td>
<td>None</td>
<td>4.4.3, 4.6.5, 5.0.7, 5.2.7, .gnuplot</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="https://graphviz.org/">Graphviz</a></td>
<td>Graphviz</td>
<td>2.40.1, 9.0.0</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="https://www.gnu.org/software/groff/">groff</a></td>
<td>groff</td>
<td>1.22.3</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="http://ab-initio.mit.edu/wiki/index.php/H5utils">h5utils</a></td>
<td>h5utils</td>
<td>1.12.1</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="https://docs.haskellstack.org/en/stable/README/">haskell-stack</a></td>
<td>None</td>
<td>1.0.4.3, 1.4.0, 1.7.1</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="https://stedolan.github.io/jq/">jq</a></td>
<td>jq</td>
<td>1.6</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="https://mariadb.org/">MariaDB</a></td>
<td>MariaDB</td>
<td>10.2.11</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="http://www.maier-komor.de/mbuffer.html">mbuffer</a></td>
<td>mbuffer</td>
<td>20151002</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="https://micans.org/mcl/">mcl</a></td>
<td>mcl</td>
<td>14-137, 22-282</td>
</tr>
<tr>
<td>Misc applications</td>
<td>metaWRAP</td>
<td>metaWRAP</td>
<td>1.3.2</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="https://github.com/openbabel/openbabel">openbabel</a></td>
<td>openbabel</td>
<td>3.1.1-gcc9.3.0, 3.1.1-gcc12.3.0</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="http://www.openbugs.net">OpenBUGS</a></td>
<td>OpenBUGS</td>
<td>3.2.3</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="https://sourceforge.net/projects/p7zip/">p7zip</a></td>
<td>p7zip</td>
<td>16.02</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="https://pandoc.org/">pandoc</a></td>
<td>pandoc</td>
<td>1.16.0.2, 2.2.3.2, 2.10.1</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="https://www.postgresql.org/">PostgreSQL</a></td>
<td>PostgreSQL</td>
<td>10.3</td>
</tr>
<tr>
<td>Misc applications</td>
<td>povray</td>
<td>povray</td>
<td>3.7</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="https://root.cern.ch/">ROOT</a></td>
<td>ROOT</td>
<td>6.04.08, 6.06.08, 6.20.04</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="https://github.com/saigegit/SAIGE">SAIGE</a></td>
<td>SAIGE</td>
<td>0.42.1, 1.4.1, 1.5.0</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="http://scons.org">SCons</a></td>
<td>None</td>
<td>2.5.0, .scons</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="https://www.sqlite.org/">sqlite</a></td>
<td>sqlite</td>
<td>3.8.5, 3.11.1, 3.16.2, 3.24.0, 3.34.0, 3.45.0</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="https://www.swig.org/index.html">swig</a></td>
<td>swig</td>
<td>3.0.7, 3.0.12, 4.1.1</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="https://www.gnu.org/software/texinfo/">texinfo</a></td>
<td>texinfo</td>
<td>6.0, 6.5, 6.6, 6.8, 7.1</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="https://www.tug.org/texlive/">texlive</a></td>
<td>texlive</td>
<td>2015, 2016, 2018, 2019, 2021, 2023-08-14, 2024-04-24</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="https://github.com/douglasgscofield/tinyutils">tinyutils</a></td>
<td>None</td>
<td>1.1, 1.2, 1.3, 1.4, .tinyutils-1.1</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="https://github.com/tmux/tmux/wiki">tmux</a></td>
<td>tmux</td>
<td>2.5, 3.1b, 3.3a</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="https://www.vim.org">vim</a></td>
<td>None</td>
<td>8.0-1360, 8.1-1053, 8.2.3701</td>
</tr>
<tr>
<td>Misc applications</td>
<td><a href="https://tukaani.org/xz">xz</a></td>
<td>xz</td>
<td>5.2.2, 5.2.6, 5.4.5</td>
</tr>
<tr>
<td>Parallel</td>
<td><a href="https://savannah.gnu.org/projects/parallel">gnuparallel</a></td>
<td>gnuparallel</td>
<td>20140222, 20150522, 20170122, 20180822, 20230422</td>
</tr>
<tr>
<td>Phylogeny</td>
<td><a href="https://github.com/millanek/Dsuite">Dsuite</a></td>
<td>Dsuite</td>
<td>0.5-r57</td>
</tr>
<tr>
<td>Phylogeny</td>
<td><a href="https://gite.lirmm.fr/atgc/FastME">FastME</a></td>
<td>FastME</td>
<td>2.1.6.4</td>
</tr>
<tr>
<td>Phylogeny</td>
<td><a href="http://www.microbesonline.org/fasttree/">FastTree</a></td>
<td>FastTree</td>
<td>2.1.11</td>
</tr>
<tr>
<td>Phylogeny</td>
<td><a href="https://github.com/AI-sandbox/gnomix">G-Nomix</a></td>
<td>G-Nomix</td>
<td>2022-09-18-de952a2</td>
</tr>
<tr>
<td>Phylogeny</td>
<td><a href="http://www.hyphy.org">HyPhy</a></td>
<td>HyPhy</td>
<td>2.5.51-mpi</td>
</tr>
<tr>
<td>Phylogeny</td>
<td><a href="http://www.iqtree.org">iqtree</a></td>
<td>iqtree</td>
<td>2.2.2.6-omp-mpi</td>
</tr>
<tr>
<td>Phylogeny</td>
<td><a href="https://github.com/DivyaratanPopli/Kinship_Inference">KIN</a></td>
<td>KIN</td>
<td>3.1.3, 3.1.3-20230612-76dc469</td>
</tr>
<tr>
<td>Phylogeny</td>
<td><a href="https://huttenhower.sph.harvard.edu/metaphlan/">MetaPhlAn4</a></td>
<td>MetaPhlAn4</td>
<td>4.0</td>
</tr>
<tr>
<td>Phylogeny</td>
<td><a href="http://abacus.gene.ucl.ac.uk/software/paml.html">paml</a></td>
<td>paml</td>
<td>4.10.7</td>
</tr>
<tr>
<td>Phylogeny</td>
<td><a href="https://github.com/ruidlpm/pathPhynder">pathPhynder</a></td>
<td>None</td>
<td>1.a-20221011-a407a97, 2020-12-19-b8532c0</td>
</tr>
<tr>
<td>Phylogeny</td>
<td><a href="https://phylipweb.github.io/phylip/">phylip</a></td>
<td>phylip</td>
<td>3.697</td>
</tr>
<tr>
<td>Phylogeny</td>
<td><a href="https://github.com/biobakery/phylophlan/wiki">PhyloPhlAn</a></td>
<td>PhyloPhlAn</td>
<td>3.0.3</td>
</tr>
<tr>
<td>Phylogeny</td>
<td><a href="https://github.com/FePhyFoFum/phyx">phyx</a></td>
<td>phyx</td>
<td>1.3</td>
</tr>
<tr>
<td>Phylogeny</td>
<td><a href="https://github.com/DessimozLab/read2tree">read2tree</a></td>
<td>read2tree</td>
<td>0.1.5-20240117-ff2d167</td>
</tr>
<tr>
<td>Phylogeny</td>
<td><a href="https://github.com/thlee/SNPhylo/tree/20180901">snphylo</a></td>
<td>snphylo</td>
<td>20180901</td>
</tr>
<tr>
<td>Pipelines</td>
<td><a href="https://github.com/aidenlab/3d-dna">3D-DNA</a></td>
<td>3D-DNA</td>
<td>20190801-529ccf4</td>
</tr>
<tr>
<td>Pipelines</td>
<td><a href="https://biomodal.com">biomodal</a></td>
<td>biomodal</td>
<td>1.0.2</td>
</tr>
<tr>
<td>Pipelines</td>
<td><a href="https://github.com/jsalignon/cactus">cactus_atac</a></td>
<td>cactus_atac</td>
<td>1.0.0</td>
</tr>
<tr>
<td>Pipelines</td>
<td><a href="https://cutadapt.readthedocs.io/en/stable/index.html">cutadapt</a></td>
<td>cutadapt</td>
<td>4.5, 4.8, 5.0</td>
</tr>
<tr>
<td>Pipelines</td>
<td><a href="https://github.com/opain/GenoPred/tree/master/GenoPredPipe">GenoPredPipe</a></td>
<td>GenoPredPipe</td>
<td>20211104-02777ce, 20221121-e3caf6b</td>
</tr>
<tr>
<td>Pipelines</td>
<td><a href="https://github.com/Nellaker-group/happy">happy</a></td>
<td>happy</td>
<td>1.0.0</td>
</tr>
<tr>
<td>Pipelines</td>
<td><a href="https://github.com/aidenlab/juicer">Juicer</a></td>
<td>Juicer</td>
<td>1.6, 2.0</td>
</tr>
<tr>
<td>Pipelines</td>
<td><a href="https://github.com/aidenlab/juicer">Juicer_tools</a></td>
<td>Juicer_tools</td>
<td>1.6, 1.22.01</td>
</tr>
<tr>
<td>Pipelines</td>
<td><a href="https://github.com/tewhey-lab/MPRA_oligo_barcode_pipeline">MPRASuite</a></td>
<td>MPRASuite</td>
<td>1.0.3</td>
</tr>
<tr>
<td>Pipelines</td>
<td>nextNEOpi</td>
<td>nextNEOpi</td>
<td>1.4.0</td>
</tr>
<tr>
<td>Pipelines</td>
<td><a href="https://github.com/nf-core/tools">nf-core</a></td>
<td>nf-core</td>
<td>2.4.1, 2.6, latest</td>
</tr>
<tr>
<td>Pipelines</td>
<td>Panaroo</td>
<td>Panaroo</td>
<td>1.2.10, 1.3.2</td>
</tr>
<tr>
<td>Pipelines</td>
<td><a href="https://www.parsebiosciences.com/">ParseBiosciences-Pipeline</a></td>
<td>ParseBiosciences-Pipeline</td>
<td>1.4.0, 1.6.4</td>
</tr>
<tr>
<td>Pipelines</td>
<td><a href="https://pepatac.databio.org/en/latest/">PEPATAC</a></td>
<td>PEPATAC</td>
<td>0.12.3</td>
</tr>
<tr>
<td>Pipelines</td>
<td><a href="https://qiime2.org/">qiime2</a></td>
<td>qiime2</td>
<td>2024.2, 2024.10</td>
</tr>
<tr>
<td>Pipelines</td>
<td><a href="https://github.com/marbl/SALSA">SALSA</a></td>
<td>SALSA</td>
<td>20220408-ed76685</td>
</tr>
<tr>
<td>Pipelines</td>
<td><a href="https://cran.r-project.org/web/packages/Seurat/index.html">Seurat</a></td>
<td>Seurat</td>
<td>5.0.2, 5.1.0, 5.2.1, 5.3.0</td>
</tr>
<tr>
<td>Pipelines</td>
<td><a href="https://www.pacb.com/support/software-downloads">SMRT</a></td>
<td>SMRT</td>
<td>13.0.0.207600</td>
</tr>
<tr>
<td>Pipelines</td>
<td><a href="https://github.com/snakemake/snakemake">snakemake</a></td>
<td>snakemake</td>
<td>5.32.2, 8.20.1</td>
</tr>
<tr>
<td>Pipelines</td>
<td><a href="https://github.com/STAR-Fusion/STAR-Fusion/wiki">star-fusion</a></td>
<td>star-fusion</td>
<td>1.9.1, 1.10.1</td>
</tr>
<tr>
<td>Pipelines</td>
<td><a href="https://github.com/AnantharamanLab/ViWrap">ViWrap</a></td>
<td>ViWrap</td>
<td>1.3.0</td>
</tr>
<tr>
<td>Statistics</td>
<td><a href="https://www.iconplc.com/innovation/nonmem/">nonmem</a></td>
<td>nonmem</td>
<td>7.3.0, 7.4.3, 7.4.4, 7.5.0, 7.6.0</td>
</tr>
<tr>
<td>Statistics</td>
<td><a href="https://uupharmacometrics.github.io/PsN/">PsN</a></td>
<td>PsN</td>
<td>5.0.0</td>
</tr>
<tr>
<td>Statistics</td>
<td><a href="https://www.r-project.org">R</a></td>
<td>None</td>
<td>2.8, 2.8.1, 2.10, 2.10.1, 2.11.1, 2.12.1, 2.12.2, 2.13.0, 2.14.0, 2.15.0, 2.15.1, 2.15.2, 3.0.1, 3.0.2, 3.1.0, 3.2.2, 3.2.3, 3.3.0, 3.3.1, 3.3.2, 3.3.2_rackham, 3.4.0, 3.4.3, 3.5.0, 3.5.2, 3.6.0, 3.6.1, 4.0.0, 4.0.4, 4.1.1, 4.2.1, 4.3.1, 4.3.2, 4.4.2, 4.5.1, .R, R.tcl</td>
</tr>
<tr>
<td>Statistics</td>
<td>R_packages</td>
<td>R_packages</td>
<td>3.3.0, 3.3.1, 3.3.2, 3.4.0, 3.4.3, 3.5.0, 3.5.2, 3.6.0, 3.6.1, 4.0.0, 4.0.4, 4.1.1, 4.3.1, 4.5.1</td>
</tr>
<tr>
<td>Statistics</td>
<td><a href="https://posit.co/download/rstudio-desktop/">RStudio</a></td>
<td>RStudio</td>
<td>1.0.136, 1.0.143, 1.0.153, 1.1.423, 1.1.463, 1.4.1106, 2022.02.0-443, 2022.02.3-492, 2022.07.1-554, 2023.06.0-421, 2023.06.2-561, 2023.12.1-402, 2024.04.2-764</td>
</tr>
<tr>
<td>Sw_collections</td>
<td><a href="https://jgi.doe.gov/data-and-tools/software-tools/bbtools/">bbmap</a></td>
<td>bbmap</td>
<td>39.06, 39.08</td>
</tr>
<tr>
<td>Sw_collections</td>
<td><a href="https://bedops.readthedocs.io">BEDOPS</a></td>
<td>BEDOPS</td>
<td>2.4.41</td>
</tr>
<tr>
<td>Sw_collections</td>
<td><a href="https://bedtools.readthedocs.io">BEDTools</a></td>
<td>BEDTools</td>
<td>2.20.1, 2.31.1</td>
</tr>
<tr>
<td>Sw_collections</td>
<td><a href="https://cellbender.readthedocs.io/">cellbender</a></td>
<td>cellbender</td>
<td>0.3.0</td>
</tr>
<tr>
<td>Sw_collections</td>
<td><a href="https://www.10xgenomics.com/support/software/cell-ranger/downloads">Chromium-cellranger</a></td>
<td>cellranger</td>
<td>4.0.0, 5.0.1, 6.0.2, 6.1.2, 7.0.0, 7.0.1, 7.1.0, 8.0.1, 9.0.1</td>
</tr>
<tr>
<td>Sw_collections</td>
<td>Chromium-cellranger-ARC</td>
<td>cellranger-ARC</td>
<td>1.0.0, 2.0.2</td>
</tr>
<tr>
<td>Sw_collections</td>
<td><a href="https://support.illumina.com/downloads/illumina-dragen-bio-it-platform-v4-1.html">DRAGEN</a></td>
<td>DRAGEN</td>
<td>4.1.5</td>
</tr>
<tr>
<td>Sw_collections</td>
<td>GATK</td>
<td>GATK</td>
<td>1.0.4105, 1.0.5365, 1.0.5909, 1.2.12, 1.4.5, 1.4.21, 1.5.11, 1.5.21, 2.1.13, 2.3.6, 2.5.2, 2.7.2, 2.8.1, 3.1.1, 3.2.0, 3.2.2, 3.3.0, 3.4-46, 3.4.0, 3.4.46, 4.0.8.0, 4.1.4.1, 4.3.0.0, 4.6.2.0</td>
</tr>
<tr>
<td>Sw_collections</td>
<td><a href="http://%20%20%20%20meme-suite.org">MEMEsuite</a></td>
<td>MEMEsuite</td>
<td>4.11.1, 4.11.2_1, 5.1.1, 5.5.1, 5.5.8</td>
</tr>
<tr>
<td>Sw_collections</td>
<td><a href="https://github.com/patrickbryant1/SpeedPPI">SpeedPPI</a></td>
<td>SpeedPPI</td>
<td>2023.07.11-37d0a03</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/google-deepmind/alphafold3">alphafold</a></td>
<td>alphafold</td>
<td>2.3.1, 3.0.1</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/google-deepmind/alphafold3">alphafold_dataset</a></td>
<td>alphafold_dataset</td>
<td>2.0.0, 2.1.1, 2.3.1, 3.0.1</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>Amber</td>
<td>None</td>
<td>16-GPU, 18-GPU</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>amber_previous</td>
<td>None</td>
<td>amber.tcl, ambertools18_intel_python2</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>AmpliconNoise</td>
<td>None</td>
<td>1.27</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>AnnotSV</td>
<td>None</td>
<td>1.1.1</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>annovar_data</td>
<td>None</td>
<td>2019.10.21, 2021.05.18</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://www.ansys.com/">Ansys</a></td>
<td>Ansys</td>
<td>19.1, 19.5, 2020R1, 2023R2</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/apache/apr">apr</a></td>
<td>None</td>
<td>1.7.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://apr.apache.org/">apr-util</a></td>
<td>None</td>
<td>1.6.1</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/archspec/archspec">archspec</a></td>
<td>archspec</td>
<td>0.2.4</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/aria2/aria2">aria2</a></td>
<td>aria2</td>
<td>1.36</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://sourceforge.net/projects/arma/">Armadillo</a></td>
<td>None</td>
<td>7.900.0, 9.700.2</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://www.ibm.com/docs/en/ahts/4.4?topic=linux-ascp-transferring-from-command-line">ascp</a></td>
<td>ascp</td>
<td>4.2.0.183804, 4.4.3.891, 4.4.4.1308</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>athena</td>
<td>None</td>
<td>1.1</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/linnarsson-lab/BackSPIN">BackSPIN</a></td>
<td>None</td>
<td>20171211_2fbcf5d</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>BAM-matcher</td>
<td>None</td>
<td>20160611, .bammatcher, .bammatcher~</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>Bandage</td>
<td>None</td>
<td>0.8.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://bazel.build/">bazel</a></td>
<td>bazel</td>
<td>7.0.0-pre.20230917.3</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/freeseek/mocha">bcftools-MoCha</a></td>
<td>None</td>
<td>1.9-20191129, 1.11-20210315, 2019-11-29</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>BclConverter</td>
<td>None</td>
<td>1.7.1</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/biobakery">BioBakery_data</a></td>
<td>BioBakery_data</td>
<td>3.0-20210423, 3.1-20231102</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>BisSNP</td>
<td>None</td>
<td>0.82.2, .bissnp</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://www.blake2.net/">BLAKE2</a></td>
<td>BLAKE2</td>
<td>20230212-ed1974e</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://ccb.jhu.edu/software/bracken">Bracken</a></td>
<td>None</td>
<td>2.5</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/containers/bubblewrap">bubblewrap</a></td>
<td>bubblewrap</td>
<td>0.6.2</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://busco.ezlab.org/">BUSCO_data</a></td>
<td>BUSCO_data</td>
<td>latest</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://sourceforge.net/projects/c3d/">c3d</a></td>
<td>c3d</td>
<td>1.4.4</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/algo-cancer/CAMMiQ">CAMMiQ</a></td>
<td>None</td>
<td>20211015-6142150</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>CAP3</td>
<td>None</td>
<td>08-06-13</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://capnproto.org/">capnproto</a></td>
<td>capnproto</td>
<td>1.0.2</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/ComparativeGenomicsToolkit/Comparative-Annotation-Toolkit">CAT</a></td>
<td>None</td>
<td>20190926-e25443</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>cellranger-ARC-data</td>
<td>None</td>
<td>2020-A</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>cellranger-ATAC-data</td>
<td>None</td>
<td>1.2.0, 2.0.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://www.10xgenomics.com/support/software/cell-ranger/downloads">cellranger-data</a></td>
<td>cellranger-data</td>
<td>1.1.0, 1.2.0, 3.0.0, 2020-A, 2024-A</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>cellranger-DNA-data</td>
<td>None</td>
<td>1.0.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://www.10xgenomics.com/support/software/cell-ranger/downloads">cellranger-VDJ-data</a></td>
<td>cellranger-VDJ-data</td>
<td>4.0.0, 5.0.0, 7.1.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>chain_files</td>
<td>chain_files</td>
<td>20230825</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://www.ebi.ac.uk/chembl/">ChEMBL</a></td>
<td>ChEMBL</td>
<td>22.1, 33</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>ChimeraSlayer</td>
<td>None</td>
<td>20110519</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://support.10xgenomics.com/single-cell-dna/software/downloads/latest">Chromium-cellranger-DNA</a></td>
<td>None</td>
<td>1.1.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://support.10xgenomics.com/genome-exome/software/pipelines/latest/what-is-long-ranger">Chromium-longranger</a></td>
<td>None</td>
<td>2.2.2</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://sanger-pathogens.github.io/circlator/">circlator</a></td>
<td>None</td>
<td>1.5.5</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://www.netlib.org/clapack/">clapack</a></td>
<td>clapack</td>
<td>3.2.1</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>clearcut</td>
<td>None</td>
<td>1.0.9</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>ClonalFrameML</td>
<td>None</td>
<td>1.11-4f13f23</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://docs.conda.io/en/latest/miniconda.html">conda</a></td>
<td>None</td>
<td>latest</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>coreutils</td>
<td>None</td>
<td>8.25, 8.27</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://www.ibm.com/se-en/analytics/cplex-optimizer">CPLEXOptimizationStudio</a></td>
<td>None</td>
<td>12.9.0, 20.1</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/tingchenlab/CROP">CROP</a></td>
<td>None</td>
<td>1.33</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://spin.niddk.nih.gov/bax/software/CSROSETTA/#Top">CS-Rosetta</a></td>
<td>None</td>
<td>1.01_Rosetta_3.7</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://www.technia.se/software/simulia/simulia-cst-studio-suite">CST_Studio</a></td>
<td>CST_Studio</td>
<td>2023.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>CTAT_RESOURCE_LIB</td>
<td>None</td>
<td>2017-11, 2018-02, 2019-08</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://developer.nvidia.com/cuda-toolkit">cuda</a></td>
<td>cuda</td>
<td>11.2.1, 11.7.0, 11.8.0, 12.0.0, 12.1.0, 12.1.1, 12.2.0, 12.2.2</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://developer.nvidia.com/cudnn">cuDNN</a></td>
<td>cuDNN</td>
<td>8.1.0.77, 8.4.1.50, 8.6.0.163, 8.7.0.84, 8.8.0.121, 8.9.2.26</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/fl-yu/CUT-RUNTools-2.0">CUT-RUNTools</a></td>
<td>None</td>
<td>2</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/cyrusimap/cyrus-sasl">cyrus-sasl</a></td>
<td>None</td>
<td>2.1.27</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/UPPMAX/darsync">darsync</a></td>
<td>darsync</td>
<td>20240208-7ff09d9</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://bcb.unl.edu/dbCAN2/">dbCAN</a></td>
<td>dbCAN</td>
<td>11</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/datacharmer/dbdeployer">DBdeployer</a></td>
<td>DBdeployer</td>
<td>latest</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>ddt-perf</td>
<td>None</td>
<td>7.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>desmond</td>
<td>desmond</td>
<td>2022-2</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://dfam.org/home">Dfam</a></td>
<td>Dfam</td>
<td>3.7, 3.8</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/Dfam-consortium/TETools">Dfam-TE-Tools</a></td>
<td>None</td>
<td>1.4</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://dftbplus.org/">DFTB+</a></td>
<td>None</td>
<td>19.1</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>dlib</td>
<td>None</td>
<td>19.15</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>dssp</td>
<td>None</td>
<td>2.0.4</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>edlib</td>
<td>edlib</td>
<td>1.2.7</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>Eigen</td>
<td>None</td>
<td>3.3.4</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>eLSA</td>
<td>None</td>
<td>20160907-febe2d7a57c8</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://espressomd.org">ESPResSo</a></td>
<td>None</td>
<td>4.1.4</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/benjann/estout">estout</a></td>
<td>None</td>
<td>20200417-d392e71</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://stevenwingett.github.io/FastQ-Screen/">fastq_screen_data</a></td>
<td>fastq_screen_data</td>
<td>20220330</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://surfer.nmr.mgh.harvard.edu/fswiki/FreeSurferWiki">freesurfer</a></td>
<td>freesurfer</td>
<td>6.0.0, 7.4.1, 8.1.0-1</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/nextgenusfs/funannotate">funannotate_data</a></td>
<td>funannotate_data</td>
<td>1.8.17</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/NBISweden/GAAS">GAAS</a></td>
<td>None</td>
<td>1.2.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>gamess</td>
<td>None</td>
<td>20070324R1, 20070324R1-kalkyl, 20101001R1-kalkyl-intel-12.1-mvapich2, 20110818R1-kalkyl-intel12.0-mvapich2, 20170930</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>gaussian</td>
<td>None</td>
<td>.g09, .gaussian, .gv, .gv~, g03, g03.d02, g03.e01, g09, g09.a02, g09.c01, g09.d01</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>gaussview</td>
<td>None</td>
<td>5.0.8, .gv, .gv~</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>gcc</td>
<td>None</td>
<td>4.2.3, 4.3.0, 4.3.2, 4.3.2-test, 4.3.2t, 4.4, 4.4.2, 4.4.3, 4.4.4, 4.5.0, 4.6.2, gcc, gcc4.3, gcc4.4, gcc4.5, gcc4.6.tcl</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://ftp.gnu.org/gnu/gdb/">gdb</a></td>
<td>None</td>
<td>11.2</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>gdc</td>
<td>None</td>
<td>2.064.2_4.8, .gdc</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/schatzlab/genomescope">genomescope</a></td>
<td>None</td>
<td>1.0.0_d2aefdd</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/tbenavi1/genomescope2.0">genomescope2.0</a></td>
<td>None</td>
<td>1.0.0_5034ed4</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/Kinggerm/GetOrganelleDB">GetOrganelleDB</a></td>
<td>GetOrganelleDB</td>
<td>0.0.1, 0.0.2</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://cli.github.com/">github-cli</a></td>
<td>github-cli</td>
<td>2.63.2</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://docs.gtk.org/glib/">glib</a></td>
<td>glib</td>
<td>2.72.1</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://ccb.jhu.edu/software/glimmerhmm/">GlimmerHMM</a></td>
<td>None</td>
<td>3.0.4</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://gmplib.org/">GMP</a></td>
<td>GMP</td>
<td>6.3.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://gnomAD.broadinstitute.org/">gnomAD_data</a></td>
<td>gnomAD_data</td>
<td>4.1</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://go.dev/">go</a></td>
<td>go</td>
<td>1.11.5, 1.20.3</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://cloud.google.com/sdk">GoogleCloudSDK</a></td>
<td>GoogleCloudSDK</td>
<td>447.0.0, 455.0.0, 528.0.0, 534.0.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/grocsvs/grocsvs">grocsvs</a></td>
<td>None</td>
<td>0.2.5</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="http://www.gromacs.org/">gromacs-plumed</a></td>
<td>None</td>
<td>2019.4.th</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>gromacs-saxs</td>
<td>None</td>
<td>2020.04-9edbdbc</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>gstreamer</td>
<td>None</td>
<td>0.10.31</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://ecogenomics.github.io/GTDBTk/index.html">GTDB</a></td>
<td>GTDB</td>
<td>R04-RS89, R202, R214.1, R220, R226</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://www.gurobi.com/downloads/gurobi-software">Gurobi</a></td>
<td>Gurobi</td>
<td>9.5.1, 10.0.2, 11.0.3, 12.0.3</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://www.gnu.org/software/gzip/">gzip</a></td>
<td>gzip</td>
<td>1.12, 1.13</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>HG002_Q100_T2T_assembly</td>
<td>HG002_Q100_T2T_assembly</td>
<td>0.7</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>HHsuite</td>
<td>None</td>
<td>2.0.16, 2.0.16~</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/DiltheyLab/HLA-LA">HLA-LA</a></td>
<td>None</td>
<td>1.0.1-20201001-f636b62</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/It4innovations/hyperqueue">hyperqueue</a></td>
<td>hyperqueue</td>
<td>0.18.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/ewels/AWS-iGenomes">iGenomes</a></td>
<td>None</td>
<td>latest</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://imagej.net">ImageJ</a></td>
<td>ImageJ</td>
<td>1.52j, 1.54g</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/jkimlab/IMAP">IMAP</a></td>
<td>None</td>
<td>1.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://immcantation.readthedocs.io/en/stable/index.html">Immcantation</a></td>
<td>None</td>
<td>4.0.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>imputor</td>
<td>None</td>
<td>20180829</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>InterOp</td>
<td>None</td>
<td>1.1.4</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://www.jamovi.org/">jamovi</a></td>
<td>jamovi</td>
<td>2.3.21</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/akheron/jansson">jansson</a></td>
<td>None</td>
<td>2.12</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>JasPer</td>
<td>None</td>
<td>2.0.14</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/open-source-parsers/jsoncpp">JsonCpp</a></td>
<td>JsonCpp</td>
<td>1.9.5</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/rikuu/kermit">kermit</a></td>
<td>None</td>
<td>1.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>Kraken_data</td>
<td>None</td>
<td>latest</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/fbreitwieser/KrakenUniq">KrakenUniq_data</a></td>
<td>KrakenUniq_data</td>
<td>latest</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>KyotoTycoon</td>
<td>None</td>
<td>stable-20170410</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://labelstud.io">Label-Studio</a></td>
<td>Label-Studio</td>
<td>1.10.1dev</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://lammps.sandia.gov/">lammps</a></td>
<td>None</td>
<td>29Oct2020_gcc_openmpi, 30Jul2016_intel_mkl, 31Mar2017_gcc_openmpi</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>libarchive</td>
<td>libarchive</td>
<td>3.6.2</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>libb2</td>
<td>libb2</td>
<td>0.98.1</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/ebiggers/libdeflate">libdeflate</a></td>
<td>libdeflate</td>
<td>1.19</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://centos.pkgs.org/7/centos-x86_64/libicu-50.2-4.el7_7.x86_64.rpm.html">libicu</a></td>
<td>None</td>
<td>5.2-4</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>liblzma</td>
<td>None</td>
<td>5.2.2</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://librdf.org/">librdf</a></td>
<td>librdf</td>
<td>1.0.17</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://gitlab.gnome.org/GNOME/librsvg">librsvg</a></td>
<td>None</td>
<td>2.48.4</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://sbml.org/software/libsbml/">libSBML</a></td>
<td>libSBML</td>
<td>5.20.2</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://www.libtiff.org">libtiff</a></td>
<td>libtiff</td>
<td>4.5.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/libvips/libvips">libvips</a></td>
<td>libvips</td>
<td>8.15.2</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>lighter</td>
<td>None</td>
<td>1.1.1</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>LLVM</td>
<td>None</td>
<td>13.0.1</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>LoFreq</td>
<td>None</td>
<td>2.1.2, .lofreq</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>longranger-data</td>
<td>None</td>
<td>2.0.0, 2.1.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/yjx1217/LRSDAY">LRSDAY</a></td>
<td>None</td>
<td>1.5.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/oushujun/LTR_retriever">LTR_retriever</a></td>
<td>None</td>
<td>2.9.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/lz4/lz4">lz4</a></td>
<td>None</td>
<td>1.9.2</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="http://www.oberhumer.com/opensource/lzo/">lzo</a></td>
<td>None</td>
<td>2.1</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>m4</td>
<td>None</td>
<td>1.4.17</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>mathematica</td>
<td>None</td>
<td>11.2</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>mauve</td>
<td>None</td>
<td>2015-02-13</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="http://mesos.apache.org/">mesos</a></td>
<td>None</td>
<td>1.9.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/tjcreedy/metamate">metaMATE</a></td>
<td>None</td>
<td>20220327-3cdacd7</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/metasnv-tool/metaSNV">metaSNV</a></td>
<td>None</td>
<td>1.0.3, 2.0.1</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>miniconda2</td>
<td>None</td>
<td>4.5.4</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>miniconda3</td>
<td>None</td>
<td>4.5.4</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/conda-forge/miniforge">miniforge</a></td>
<td>miniforge</td>
<td>latest</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>mlst</td>
<td>None</td>
<td>2.12</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/soedinglab/mmseqs2/wiki#downloading-databases">MMseqs2_data</a></td>
<td>MMseqs2_data</td>
<td>20230125, 20230816, 20240202</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://www2.statmt.org/moses/">Moses</a></td>
<td>Moses</td>
<td>4.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>mp-tools</td>
<td>mp-tools</td>
<td>latest</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://www.mpfr.org/">MPFR</a></td>
<td>MPFR</td>
<td>4.2.1</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>mpjexpress</td>
<td>None</td>
<td>.mpjexpress, v0.38</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://www.multibugs.org/">MultiBUGS</a></td>
<td>None</td>
<td>2.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://graal.ens-lyon.fr/MUMPS/">MUMPS</a></td>
<td>MUMPS</td>
<td>5.5.0, 5.5.0-hybrid</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/jts/nanopolish">nanopolish</a></td>
<td>None</td>
<td>0.12.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://www.ncbi.nlm.nih.gov/guide/taxonomy/">ncbi_taxonomy</a></td>
<td>None</td>
<td>latest</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>ncftp</td>
<td>None</td>
<td>3.2.6</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="http://meteora.ucsd.edu/~pierce/ncview_home_page.html">ncview</a></td>
<td>None</td>
<td>2.1.7, 2.1.7-intel-2019b</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>ngsF-HMM</td>
<td>None</td>
<td>20160614</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/TravisWheelerLab/NINJA">NINJA</a></td>
<td>None</td>
<td>0.97-cluster_only</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://www.ibbr.umd.edu/nmrpipe/">NMRPipe</a></td>
<td>None</td>
<td>10.4</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>nullarbor</td>
<td>None</td>
<td>2.0.20180819</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>octave</td>
<td>None</td>
<td>5.1.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://opam.ocaml.org/">opam</a></td>
<td>opam</td>
<td>2.1.2</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/opencv/opencv">OpenCV</a></td>
<td>OpenCV</td>
<td>4.12.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://www.openfoam.com/">OpenFOAM</a></td>
<td>None</td>
<td>6, 7, v1912</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://www.openjpeg.org/">OpenJPEG</a></td>
<td>OpenJPEG</td>
<td>2.3.0, 2.5.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/openslide/openslide">openslide</a></td>
<td>openslide</td>
<td>3.4.1, 4.0.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://orcaforum.kofo.mpg.de/">ORCA</a></td>
<td>None</td>
<td>4.0.1.2, 4.2.1, 5.0.4, 6.0.0, 6.0.1, 6.1.0, .4.0.1.2</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="http://pantherdb.org/">panther</a></td>
<td>panther</td>
<td>14.1, 15.0, 17.0, 18.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://www.paraview.org/">paraview</a></td>
<td>None</td>
<td>5.6, 5.9.1, .paraview_noQT, .paraview_noQT~</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>PCAP</td>
<td>None</td>
<td>PCAP</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/PCRE2Project/pcre2">pcre2</a></td>
<td>pcre2</td>
<td>10.42</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/ncbi/pgap">PGAP</a></td>
<td>None</td>
<td>2022-02-10.build5872</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://www.php.net">PHP</a></td>
<td>PHP</td>
<td>7.4.32</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/broadinstitute/PhylogicNDT">PhylogicNDT</a></td>
<td>None</td>
<td>2020</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>phylophlan</td>
<td>None</td>
<td>0.99</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://zlib.net/pigz/">pigz</a></td>
<td>pigz</td>
<td>2.4, 2.8</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://www.plumed.org">plumed</a></td>
<td>None</td>
<td>2.6.0, 2.7.1-gcc</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>pm-tools</td>
<td>pm-tools</td>
<td>latest</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://pyinstaller.org/en/stable/index.html">PyInstaller</a></td>
<td>PyInstaller</td>
<td>6.12.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>python_GIS_packages</td>
<td>python_GIS_packages</td>
<td>3.10.8</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://pypi.org">python_ML_packages</a></td>
<td>python_ML_packages</td>
<td>3.9.5-cpu, 3.9.5-gpu, 3.11.8-cpu, 3.11.8-gpu</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://pytorch.org/">PyTorch</a></td>
<td>PyTorch</td>
<td>1.12.0-cpu</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://www.well.ox.ac.uk/~gav/qctool_v2/">qctool</a></td>
<td>None</td>
<td>2-5559450, 2-beta</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/hartleys/QoRTs">QoRTs</a></td>
<td>None</td>
<td>1.3.6</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://qualcoder.wordpress.com/">QualCoder</a></td>
<td>QualCoder</td>
<td>3.7</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="http://www.quantum-espresso.org">QuantumESPRESSO</a></td>
<td>None</td>
<td>6.7MaX-Release</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="http://rawcopy.org">rawcopy</a></td>
<td>None</td>
<td>1.1</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/rdkit/rdkit">RDKit</a></td>
<td>RDKit</td>
<td>20220301</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://tiswww.case.edu/php/chet/readline/rltop.html">readline</a></td>
<td>None</td>
<td>6.2-11</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>rfmix</td>
<td>None</td>
<td>v2</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://www.rosettacommons.org/">Rosetta</a></td>
<td>None</td>
<td>3.7, 2019.4</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://www.physics.uu.se/research/materials-theory/ongoing-research/code-development/rspt-main/">RSPt</a></td>
<td>None</td>
<td>2019-08-30, 2020-06-10</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>rspt</td>
<td>None</td>
<td>1951_intel_mkl, 2003_intel_mkl, 2018-03-31_intel_mkl, rspt.tcl</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://www.rust-lang.org/">rust</a></td>
<td>rust</td>
<td>1.43.1, 1.67.0, 1.77.0, 1.85.1</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>sbcl</td>
<td>None</td>
<td>1.0.12, .sbcl</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/scikit-learn/scikit-learn">scikit-learn</a></td>
<td>None</td>
<td>0.22.1</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/NBISweden/sda-cli">sda-cli</a></td>
<td>sda-cli</td>
<td>0.1.0, 0.1.3, 0.3.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/simongog/sdsl-lite">SDSL</a></td>
<td>None</td>
<td>2.1.1</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://gitlab.com/siesta-project/siesta">Siesta</a></td>
<td>None</td>
<td>4.1-b4, 4.1-MaX-1.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>siesta</td>
<td>None</td>
<td>3.2-pl4_intel_mkl, 4.1-b3_intel_mkl, siesta-2018.tcl, siesta.tcl</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>silva</td>
<td>None</td>
<td>128, 132, 138.1</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/SimpleITK/SimpleITK">simpleitk</a></td>
<td>None</td>
<td>1.2.4</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://www.singular.uni-kl.de">Singular</a></td>
<td>None</td>
<td>4.1.2</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/brentp/smoove">smoove</a></td>
<td>None</td>
<td>0.2.5</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>snp-sites</td>
<td>None</td>
<td>2.4.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://pcingola.github.io/SnpEff">snpEff_data</a></td>
<td>snpEff_data</td>
<td>5.1, 5.2</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>SourceTracker</td>
<td>None</td>
<td>0.9.5</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://www.10xgenomics.com/support/software/space-ranger/downloads">spaceranger-data</a></td>
<td>spaceranger-data</td>
<td>1.0.0, 2020-A, 2024-A</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>spm</td>
<td>None</td>
<td>12</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://www.ebert.cup.uni-muenchen.de/index.php/en/software-en/13-sprkkr">SPRKKR</a></td>
<td>None</td>
<td>7.7.1, 7.7.3</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/wyang17/SQuIRE">SQuIRE</a></td>
<td>None</td>
<td>0.9.9.9a-beta, 885bf4d-20190301</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/ncbi/SRPRISM">SRPRISM</a></td>
<td>None</td>
<td>3.2.7</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="http://suitesparse.com">SuiteSparse</a></td>
<td>SuiteSparse</td>
<td>5.8.1, 7.7.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>sumaclust</td>
<td>None</td>
<td>1.0.00</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>sunstudio</td>
<td>None</td>
<td>12.1, 12.1u1, .sunstudio</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/walaj/svaba">SvABA</a></td>
<td>None</td>
<td>1.1.3-20201112-0f60e36</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/svviz/svviz">svviz</a></td>
<td>None</td>
<td>1.6.4</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>swarm</td>
<td>None</td>
<td>1.2.19</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/symengine/symengine/releases/download/v0.7.0/symengine-0.7.0.tar.gz">SymEngine</a></td>
<td>None</td>
<td>0.7.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://www.taguette.org/">taguette</a></td>
<td>taguette</td>
<td>1.4.1</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://missionbio.com/products/platform/">tapestri</a></td>
<td>None</td>
<td>2.0.1</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>tcl</td>
<td>None</td>
<td>8.6.8</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>TelomereHunter</td>
<td>None</td>
<td>1.0.4</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://tesseract-ocr.github.io">tesseract</a></td>
<td>None</td>
<td>4.1.3</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/mhammell-laboratory/TEtranscripts">TEtranscripts</a></td>
<td>None</td>
<td>2.2.1</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>totalview</td>
<td>None</td>
<td>8.8.0-2, 8.9.2-0, 8.9.2-2, 8.10.0-1, 8.15.0-15, totalview</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>turbo</td>
<td>None</td>
<td>6.00, 6.01, 6.1, 6.02, .turbo</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>turbomole</td>
<td>None</td>
<td>7.3</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>umap</td>
<td>None</td>
<td>0.3.7</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/JuliaStrings/utf8proc">utf8proc</a></td>
<td>None</td>
<td>2.5.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://vampire.york.ac.uk">Vampire</a></td>
<td>Vampire</td>
<td>6.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>vasp</td>
<td>None</td>
<td>4.6.34_intel_mkl, 4.6_intel_mkl, 5.2.8_intel_mkl, 5.2.11_intel_mkl, 5.2.12_intel_mkl, 5.2_intel_mkl, 5.3.3_intel_mkl, 5.4.1_intel_mkl, 5.4.1_openmpi_intelmkl, 5.4.4_intel_mkl, .vasp_acml, vasp, vasp.save, vasp.tcl, vasp.tcl_2013, vasp.tcl_2014, vasp.tcl_2017, vasp.tcl_2017_intelmpi, vasp.tcl_2017_openmpi, vasp_new</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>vasp_acml</td>
<td>None</td>
<td>4.6.28, 4.6.34, 4.6.36, .vasp_acml</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>vasp_goto</td>
<td>None</td>
<td>4.6.28, 4.6.34, 4.6.36, .vasp_goto</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://www.ks.uiuc.edu/Research/vmd/">VMD</a></td>
<td>VMD</td>
<td>1.9.4-alpha-57</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://vscodium.com/">VSCodium</a></td>
<td>VSCodium</td>
<td>latest</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>vtk</td>
<td>None</td>
<td>6.1.0, .vtk</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/openai/whisper">Whisper</a></td>
<td>Whisper</td>
<td>0.5.1, 20240930</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/UPPMAX/Whisper-sens">Whisper-gui</a></td>
<td>Whisper-gui</td>
<td>0.1, 0.2, 0.3, 1.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>Whispercpp</td>
<td>Whispercpp</td>
<td>1.7.3</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/m-bain/whisperX">WhisperX</a></td>
<td>WhisperX</td>
<td>3.1.1</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://www.winehq.org/">wine</a></td>
<td>wine</td>
<td>7.0</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="http://www.wrf-model.org">WPS</a></td>
<td>None</td>
<td>4.1</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://www2.mmm.ucar.edu/wrf/users/download/get_sources_wps_geog.html">WPS-geog</a></td>
<td>None</td>
<td>4</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="http://www.wrf-model.org">WRF</a></td>
<td>None</td>
<td>4.1.3, 4.1.3-dmpar</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://wrfg.net/">WRFg</a></td>
<td>None</td>
<td>3.8.1</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="http://www.xcrysden.org">XCrySDen</a></td>
<td>None</td>
<td>1.5.60, 1.6.2</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://siglang.org">zig</a></td>
<td>zig</td>
<td>0.9.1, 0.11.0, 0.14.1, 0.15.2</td>
</tr>
<tr>
<td>Uncategorized</td>
<td>zsh</td>
<td>None</td>
<td>5.7.1</td>
</tr>
<tr>
<td>Uncategorized</td>
<td><a href="https://github.com/facebook/zstd">zstd</a></td>
<td>zstd</td>
<td>1.5.2</td>
</tr>
</tbody>
</table>
</div></div>












                
              </article>
            </div>
          
          
  <script>var tabs=__md_get("__tabs");if(Array.isArray(tabs))e:for(var set of document.querySelectorAll(".tabbed-set")){var labels=set.querySelector(".tabbed-labels");for(var tab of tabs)for(var label of labels.getElementsByTagName("label"))if(label.innerText.trim()===tab){var input=document.getElementById(label.htmlFor);input.checked=!0;continue e}}</script>

<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
      </main>
      
        <!-- <footer class="md-footer"> -->
<!--   <div class="md-footer-meta md-typeset"> -->
<!--     <div class="md-footer-meta__inner md-grid"> -->
<!--       <div > -->
<!--         <h2>Contact UPPMAX</h2> -->
<!--         <div> -->
<!--           <ul> -->
<!--             <li>Support: <a href="mailto:support@uppmax.uu.se">support@uppmax.uu.se</a></li> -->
<!--             <li>Director: <a href="http://katalog.uu.se/profile/?id=N94-2143">Elisabeth Larsson</a></li> -->
<!--             <li>Economy: <a href="http://katalog.uu.se/empinfo/?id=N18-298">Wiola &Ouml;hlund</a></li> -->
<!--             <li>Post: P.O. Box 337, 751 05 Uppsala</li> -->
<!--           </ul> -->
<!--         </div> -->
<!--       </div> -->
<!--       <div > -->
<!--         <h2>Visit UPPMAX</h2> -->
<!--         <div> -->
<!--           <ul> -->
<!--             <li>Floor 3, House 10, &Aring;ngstr&ouml;m Laboratory</li> -->
<!--             <li>L&auml;gerhyddsv&auml;gen 1</li> -->
<!--             <li>752 37 Uppsala</li> -->
<!--           </ul> -->
<!--         </div> -->
<!--       </div> -->
<!--       <div> -->
<!--         <h2>Shortcuts</h2> -->
<!--         <div> -->
<!--           <p style="text-align:start"><a href="/support/getting-started/">Apply for a project&nbsp;or user account</a> -->
<!--           </p> -->
<!--           <p style="text-align:start"><a href="/support/faq/">FAQ</a></p> -->
<!--           <p style="text-align:start"><a href="/support/user-guides/">User Guides</a></p> -->
<!--           <p style="text-align:start"><a href="/resources/software/installed-software/">List of Installed Software</a> -->
<!--           </p> -->
<!--         </div> -->
<!--       </div> -->
<!--     </div> -->
<!--   </div> -->
<!---->
<!-- </footer> -->
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    
      
      
      <script id="__config" type="application/json">{"annotate": null, "base": "../..", "features": ["toc.follow", "navigation.tabs", "navigation.tabs.sticky", "navigation.sections", "navigation.path", "navigation.footer", "search.suggest", "content.code.annotate", "content.code.copy", "content.tabs.link", "content.action.edit", "content.action.view", "content.code.copy"], "search": "../../assets/javascripts/workers/search.2c215733.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
    
    
      <script src="../../assets/javascripts/bundle.79ae519e.min.js"></script>
      
        <script src="../../js/kapa-widget.js"></script>
      
        <script src="../../js/jquery-3.7.0.min.js"></script>
      
        <script src="../../js/jquery.dataTables.min.js"></script>
      
        <script src="../../js/sortable_searchable_tables.js"></script>
      
        <script src="../../js/open_in_new_tab.js"></script>
      
    
  </body>
</html>